{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "ec590399",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import sklearn\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "import os\n",
    "%matplotlib inline\n",
    "\n",
    "# some code to control figure aeshetics\n",
    "sns.set_style(\"ticks\")\n",
    "sns.set_context(\"notebook\", font_scale=1.5, rc={\"lines.linewidth\": 2.5})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "916df7cf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of data points: 3321\n",
      "Number of features: 4\n",
      "Features: ['ID' 'Gene' 'Variation' 'Class']\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Class</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>FAM58A</td>\n",
       "      <td>Truncating Mutations</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>CBL</td>\n",
       "      <td>W802*</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>CBL</td>\n",
       "      <td>Q249E</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>CBL</td>\n",
       "      <td>N454D</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>CBL</td>\n",
       "      <td>L399V</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   ID    Gene             Variation  Class\n",
       "0   0  FAM58A  Truncating Mutations      1\n",
       "1   1     CBL                 W802*      2\n",
       "2   2     CBL                 Q249E      2\n",
       "3   3     CBL                 N454D      3\n",
       "4   4     CBL                 L399V      4"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "basedir = os.getcwd()\n",
    "datadir = basedir + '/data'\n",
    "\n",
    "# reading the gene variants\n",
    "data = pd.read_csv(datadir + '/training_variants.zip')\n",
    "print('Number of data points:', data.shape[0])\n",
    "print('Number of features:', data.shape[1])\n",
    "print('Features:', data.columns.values)\n",
    "data.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "eb039953",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pandas/util/_decorators.py:311: ParserWarning: Falling back to the 'python' engine because the 'c' engine does not support regex separators (separators > 1 char and different from '\\s+' are interpreted as regex); you can avoid this warning by specifying engine='python'.\n",
      "  return func(*args, **kwargs)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of data points: 3321\n",
      "Number of features: 2\n",
      "Features: ['ID' 'Text']\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>Cyclin-dependent kinases (CDKs) regulate a var...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>Abstract Background  Non-small cell lung canc...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>Abstract Background  Non-small cell lung canc...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>Recent evidence has demonstrated that acquired...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>Oncogenic mutations in the monomeric Casitas B...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   ID                                               Text\n",
       "0   0  Cyclin-dependent kinases (CDKs) regulate a var...\n",
       "1   1   Abstract Background  Non-small cell lung canc...\n",
       "2   2   Abstract Background  Non-small cell lung canc...\n",
       "3   3  Recent evidence has demonstrated that acquired...\n",
       "4   4  Oncogenic mutations in the monomeric Casitas B..."
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# reading the training text\n",
    "data_text = pd.read_csv(datadir + '/training_text.zip', sep = '\\|\\|', names = [\"ID\",\"Text\"],skiprows = 1)\n",
    "print('Number of data points:', data_text.shape[0])\n",
    "print('Number of features:', data_text.shape[1])\n",
    "print('Features:', data_text.columns.values)\n",
    "data_text.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "0bbd4821",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"Signaling by transforming growth factor-β (TGF-β) superfamily ligands to the nucleus is mediated by type I and type II receptors and the intracellular signal transducers, the Smads. Alteration of some of the components of these pathways has been observed in human tumors. These alterations can be deletions or mutations, or downregulation of components that act positively in the pathway, or alternatively, amplification or overexpression of inhibitors of the pathways. The selection of these alterations during tumor progression and their correlation with clinical outcomes, such as survival, risk of recurrence after tumor resection or tendency for metastatic spread, suggest that many are involved in tumor progression. Here, we review the genetic alterations and epigenetic modifications that occur in different components of the TGF-β superfamily signaling pathways in human tumors and we discuss their correlation with clinical outcome. The evidence suggests that not all alterations of the TGF-β superfamily signaling pathway components in human cancer have an equivalent effect on tumor progression and we discuss what implications this has for our understanding of the role of TGF-β signaling in human cancer.  Abbreviations BAT-RII, big adenine tract in receptor type II; EMT, epithelial-mesenchymal transition; FAP, familial adenomatous polyposis; HCC, hepatocellular carcinoma; HCV, Hepatitis C Virus; HNPCC, hereditary non-polyposis colorectal cancer; HNSCC, head and neck squamous cell carcinoma; JP, Juvenile Polyposis; LN, lymph node; LOH, loss of heterozygosity; MSI−, without microsatellite instability; MSI+, with microsatellite instability; MSS, microsatellite stable; NSCLC, non-small cell lung carcinoma; SCC, squamous cell carcinoma; TGF-β, transforming growth factor-β Keywords Cancer; Receptors; Smad; TGF-β; Tumor suppressor; Tumor promoter 1. Introduction Members of the TGF-β superfamily ligands, such as TGF-β, Activin and BMP, transduce their signals through heterotetrameric complexes comprising two types of serine-threonine kinase receptor, the type I and type II. Upon ligand binding, the type II receptor phosphorylates and activates the type I receptor, which initiates downstream signaling by phosphorylating the receptor-regulated Smads (R-Smads). Each ligand signals through a specific combination of type II, type I and R-Smads (reviewed in [1]). TGF-β binds to the TGF-β type II receptor (TβRII) and the TGF-β type I receptor (TβRI also called Alk5 for Activin receptor-like kinase 5), although in endothelial cells it can also bind a complex comprising TβRII, Alk1 and Alk5 [2]. Activin and BMPs can both bind to the same type II receptors (ActR-IIA/ACVR2 and ActR-IIB/ACVR2b), but in combination with distinct type I receptors, which are Alk4 for Activin and Alks 2, 3 and 6 for BMPs. In addition, BMPs can signal through the BMP type II receptor (BMPR-II) in conjuction with Alks 2, 3 and 6. The type I receptor dictates the specificity for the R-Smad activation; Alks 4, 5 and 7 phosphorylate the R-Smads Smad2 and Smad3, whilst Alks 1, 2, 3 and 6 phosphorylate the R-Smads Smad1, Smad5 and Smad8. Once phosphorylated, these R-Smads transduce the signal to the nucleus in cooperation with the common mediator Smad, Smad4, to transcriptionally activate or repress different targets genes [1]. The TGF-β superfamily pathways are also negatively regulated. The inhibitory Smads, Smad6 and Smad7, bind the active receptor complexes and also recruit E3 ubiquitin ligases Smurf1/2 to the receptor complexes to degrade them [3]; [4] ;  [5]. Smad7 has also been shown to participate in a complex that dephosphorylates the active TGF-β receptors [6]. In addition, the transcriptional repressor, Ski, and its close relative, SnoN, bind to active Smad complexes and inhibit their transcriptional activity [7].  The TGF-β superfamily signaling pathways are involved in many different biological processes during embryonic development, and play a role in adult organisms in tissue homeostasis [8]. They are also strongly implicated in cancer, since alterations of some specific and some common components of these different pathways have been identified in human tumors. Fig. 1 shows the particular pathways that are affected in human cancer.  Alerations of the different components of the TGF-β superfamilly pathway in ... Fig. 1.  Alerations of the different components of the TGF-β superfamilly pathway in human cancer. Pathways for which components have been found to be altered in human cancers are represented. Inactivating alterations such as mutations, deletions or downregulation, are shown in green, whilst activating alterations, such as amplification or overexpression, are shown in red. Components of these pathways that have not been shown to be affected in human tumors are shown in yellow. For details, see text. (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.) Figure options Cancer is a multistep process resulting in the evolution of a clonal cell population that has escaped from the control of regulatory circuits that normally govern cell proliferation and homeostasis. This can happen through two distinct types of genetic alteration: gain-of-function mutations in oncogenes that usually result in growth factor-independent cell proliferation, or recessive loss-of-function mutations in tumor suppressors that allow evasion of growth inhibitory signals. The well-characterized growth inhibitory response of TGF-β [9], combined with the fact that up to 74% of colon cancer cell lines, and 85% of lung cancer cell lines have become resistant to this antiproliferative effect [10] ;  [11], led to the search for evidence of inactivation of components of the TGF-β pathway in human cancer. Inactivation of a tumor suppressor is characterized by alterations that inactivate both alleles, which is achieved either by homozygous deletion or mutation, or more commonly, by mutation of one allele accompanied by the loss of a chromosomal region containing the other allele, termed loss of heterozygosity (LOH). Thus, identification of high frequency LOH at a particular locus in different types of cancers, in conjunction with the mapping of genes that have undergone mutation or homozygous deletion at the same locus in the same cancers, has been a common way to identify new tumor suppressor genes. In fact, the gene encoding Smad4, MADH4/DPC4 (hereafter SMAD4) was identified as a tumor suppressor gene in pancreatic cancer in this way [12]. Germline mutations in SMAD4 have also been identified in some individuals with the familial syndrome, Juvenile Polyposis (JP), which leads to predisposition to hamartomatous polyps and gastrointestinal cancer [13]. Furthermore, other JP patients who do not have germline mutations in SMAD4, have mutations in BMPR1A (which encodes the BMP type I receptor, Alk3), indicating that this gene too is a tumor suppressor [14]. There is also evidence that both the TGF-β type I and type II receptors act as tumor suppressors, since they are both inactivated by a variety of different mechanisms in human tumors [15]. The alternative way of inactivating the tumor suppressor function of the TGF-β pathway is through overexpression or amplification of its inhibitors such as SMAD7, SMURF2 or the transcriptional repressors, SKI/SNON, and indeed, these events have been identified in human tumors [7]; [16] ;  [17].  In contrast to the tumor suppressive effects of the TGF-β signaling pathways, is the striking finding that TGF-β is widely overexpressed in many human cancers and this alteration in tumors is associated with poor prognosis, tumor vascularization and metastasis [18]. This observation suggests that TGF-β can also act as a tumor promoter.  In this review, we first describe in detail how the TGF-β superfamily signaling pathways are altered in human cancer. We then go on to assess, where possible, the clinical consequences of modulating the activity or expression levels of different components of the TGF-β signaling pathway in human tumors.  2. Alterations in components of the TGF-β signaling pathways 2.1. Receptor alterations in human tumors  2.1.1. TGF-β type II receptor (TGFBR2/TβRII)  The TGF-β type II receptor (TβRII) gene, TGFBR2, has been mapped to chromosome 3p, a chromosome for which LOH is frequently observed in both small cell lung carcinoma (SCLC) and non-small cell lung carcinoma (NSCLC) and in gastric cancer [19] ;  [20]. In these cancers, though, LOH was not found to be associated with TGFBR2 mutations, suggesting that TGFBR2 is not the tumor suppressor gene at this locus that is targeted in these cases.  However, there is other evidence that TGFBR2 is indeed a tumor suppressor, which has come from its specific alteration in cancers associated with microsatellite instability (MSI). MSI arises from inactivation of the mutation mismatch repair system (MMR) [21]. Inactivation of the complex of proteins that repairs basepair mismatches during DNA replication predominantly affects mono, di- and trinucleotide tracts and short sequence repeats by accumulating replication errors at these repeats. Thus, genes that contain such “microsatellite-like” repeats in their coding regions are observed to be mutated in MSI-associated cancer. The TGFBR2 gene contains such repeats in exon 3, consisting of a 10-base pair (bp) poly-adenine repeat called big adenine tract in receptor type II (BAT-RII) [22]. This repeat has been shown to be frequently mutated in MSI-associated cancers. BAT-RII mutations are characterized by an insertion/deletion of one or two adenines that results in the introduction of a non-sense mutation leading to a truncated protein that lacks the transmembrane domain and the intracellular serine–threonine kinase domain [22]. BAT-RII mutations have been identified at very high frequency in hereditary non-polyposis colorectal cancer (HNPCC), which is mainly characterized by inherited genetic instability [23]. In sporadic cancers associated with the MSI phenotype, BAT-RII mutations have been identified in a high percentage of colon cancers, gastric cancers, NSCLC, gliomas and biliary tract cancers (Table 1) [22]; [23]; [24]; [25]; [26] ;  [27]. In most cases, the BAT-RII mutations were present on both alleles; in a few other cases, point mutations in the serine–threonine kinase domain of TβRII were observed on the other allele [23]. The fact that the TGFBR2 mutations are biallelic strongly suggests a tumor suppressor function for TβRII. Importantly, although all poly-adenine tracts are unstable in MSI tumors, the BAT-RII tract is more unstable when compared to other poly-adenine tracts of identical size in the same tumors, which suggests that there is a selective advantage for TGFBR2 inactivation during MSI-associated tumorigenesis. However, since some MSI-associated cancers, such as endometrial carcinomas and hepatocellular carcinomas (HCC) do not exhibit any BAT-RII mutations [24] ;  [28], it is likely that the selection for BAT-RII mutations is also dependent on another event.  Table 1. Frequency of the alterations of the TGF-β superfamilly components in human cancer and their correlation with clinical outcome Protein (Locus; gene)\\tTumor type\\tNa\\tAlteration type\\tFb (%)\\tCorrelation with\\tRef.c LIGAND   TGF-β1 (19q13.1; TGFB1)\\tBreast\\t81\\tOverexpression\\t17\\tMetastasis\\t[112] 28\\tOverexpression\\t68\\tMetastasis\\t[113] Colon\\t34\\tOverexpression\\tnd\\tDisease progression to metastasis\\t[118] 22\\tHigh level in serum\\tnd\\tTumor stage of Duckes’ classification\\t[116] 121\\tHigh level in serum\\tnd\\tTumor stage, depth of tumor invasion and metastasis\\t[117] Esophageal (SCC)\\t80\\tOverexpression\\t36.6\\tDepth of invasion only\\t[37] Gastric\\t101\\tOverexpression\\t22.8\\tDepth of invasion, tumor stage, VEGF expression and prognosis\\t[120] HCC\\t17\\tHigh level in serum\\tnd\\tTumor vascularity\\t[123] 26\\tHigh level in serum\\tnd\\tnd\\t[124] Lung\\t124\\tOverexpression\\t48\\tBetter prognosis\\t[114] Lung (NSCLC)\\t53\\tOverexpression\\tnd\\tTumor grade, LN metastasis, microvessel density\\t[115] Pancreatic\\t60\\tOverexpression\\t47\\tLack of TGF-β correlates with increased postoperative survival time\\t[121] Prostate\\t73\\tOverexpression\\tnd\\tTumor grade, high vascular counts and metastasis\\t[122]  TYPE II RECEPTORS   TβRII (3p22; TGFBR2)\\tBiliary tract  MSI−\\t7\\tPoly(A)10 tract mutation\\t0\\tnd\\t[27]  MSI+\\t16\\tPoly(A)10 tract mutation\\t50\\tnd\\t[27] Bladder\\t59\\tLoss of expression\\t44\\tTumor grade\\t[53] Breast\\t34\\tIntragenic mutation\\t12\\tnd\\t[30] nd\\tLoss of expression\\tnd\\tTumor grade\\t[39] Colon sporadic  MSI−\\t14\\tIntragenic mutation\\t15\\tnd\\t[11]  MSI+\\t62\\tPoly(A)10 tract mutation\\t61\\tBetter 5-year survival rate\\t[134]  MSI−\\t54\\tPoly(A)10 tract mutation\\t9\\tnd\\t[22]  MSI+\\t14\\tPoly(A)10 tract mutation\\t86\\tnd\\t[22]  nd\\t100\\tIntragenic mutation\\t11\\tnd\\t[76] Colon sporadic and HNPCC  MSI−\\t40\\tPoly(A)10 tract mutation\\t0\\tnd\\t[23]  MSI+\\t111\\tPoly(A)10 tract mutation\\t90\\tnd\\t[23] Esophageal (SCC)\\t21\\tDownregulation\\t29\\tnd\\t[36] 80\\tLoss of expression\\t28.8\\tDepth of invasion, metastasis, pathological stage, poor prognosis\\t[37] Gastric  MSI−\\t15\\tPoly(A)10 tract mutation\\t0\\tnd\\t[56]  MSI+\\t25\\tPoly(A)10 tract mutation\\t68\\tnd\\t[56]  MSI−\\t35\\tnd\\tnd\\tnd\\t[24]  MSI+\\t15\\tPoly(A)10 tract mutation\\t∼70\\tnd\\t[24] Glioma  MSI−\\t33\\tPoly(A)10 tract mutation\\t3\\tnd\\t[26]  MSI+\\t7\\tPoly(A)10 tract mutation\\t71\\tnd\\t[26] HCC (HCV+)\\t73\\tPoly(A)10 tract mutation\\t25\\tnd\\t[29] HNSCC\\t28\\tIntragenic mutation\\t21\\tnd\\t[31] Lung (NSCLC)  MSI+\\t8\\tPoly(A)10 tract mutation\\t75\\tnd\\t[25]  nd\\t20\\tPoly(A)10 tract mutation\\t5\\tPoly(A) tract mutation associated with MSI\\t[19]  nd\\t28\\tIntragenic mutation\\t21\\tnd\\t[31]  nd\\t43\\tDownregulation\\t44\\tPromoter methylation\\t[38] Ovarian\\t24\\tIntragenic mutation\\t25\\tnd\\t[32] 22\\tDecrease or loss of expression\\t23\\tClinical stage\\t[32] Pancreatic\\t97\\tHomozygous deletion and poly(A)10 tract mutation\\t4.1\\tPoly(A) tract mutations associated with MSI\\t[47] Prostate\\t32\\tLoss of expression\\t12.5\\tTumor grade\\t[41] 52\\tLoss of expression\\tnd\\tNot correlated with poor prognosis\\t[40]   ACVR2 (2q22; ACVR2)\\tColon  MSI−\\t20\\tPoly(A)8 tract mutation\\t0\\tnd\\t[44]  MSI+\\t54\\tPoly(A)8 tract mutation\\t83\\tnd\\t[44]  MSI+\\t46\\tPoly(A)8 tract mutation\\t58.1\\tnd\\t[43]  MSI+\\t24\\tPoly(A)8 tract mutation\\t90\\tnd\\t[46] Gastric  MSI+\\t18\\tPoly(A)8 tract mutation\\t44.4\\tnd\\t[45] Pancreatic  MSI−\\t7\\tIntragenic mutation\\t14\\tnd\\t[46]  MSI+\\t3\\tPoly(A)8 tract mutation\\t100\\tnd\\t[46]  TYPE I RECEPTORS   TβRI/Alk5 (9p22; TGFBR1)\\tBiliary tract\\t12\\tHomozygous deletion\\t8.5\\tnd\\t[47] Bladder\\t59\\tLoss of expression\\t30.5\\tTumor grade and poor prognosis\\t[53] Breast  Carcinoma\\t31\\tIntragenic mutation\\t10\\tLN metastasis\\t[51]  LN metastasis\\t12\\tIntragenic mutation\\t42\\tLN metastasis\\t[51] Gastric  MSI−\\t15\\tPromoter methylation\\t26.7\\tnd\\t[56]  MSI+\\t25\\tPromoter methylation\\t64\\tnd\\t[56]  nd\\t17\\tDownregulation\\t82\\tnd\\t[54]  nd\\t40\\tPromoter methylation\\t12.5\\tHypermethylation of CpG island\\t[55] HNSCC  Metastasis\\t23\\tIntragenic mutation\\t17\\tnd\\t[48]  Esophageal\\t80\\tLoss of expression\\t53\\tDepth of invasion, metastasis, pathological stage, poor prognosis\\t[37] Ovarian\\t32\\tIntragenic mutation\\t31.3\\tnd\\t[49] 30\\tIntragenic mutation\\t33\\tnd\\t[50] Pancreatic\\t97\\tHomozygous deletion\\t1\\tnd\\t[47] Prostate\\t32\\tLoss of expression\\t25\\tTumor grade\\t[41] 52\\tLoss of expression\\tnd\\tTumor grade and poor prognosis\\t[40]   Alk3 (10q22-q23; BMPR1A)\\tColon JP\\t25\\tGermline mutation\\t40\\tnd\\t[60] 54\\tGermline mutation\\t24\\tHigher frequency of gastrointestinal cancer\\t[57] 29\\tGermline mutation\\t17\\tnd\\t[58] 77\\tGermline mutation\\t20.8\\tnd\\t[59]   Alk4 (12q13; ACVR1B)\\tPancreatic\\t95\\tIntragenic mutation\\t2\\tnd\\t[61]  SMADS   Smad4 (18q21; SMAD4)\\tBiliary tract\\t32\\tIntragenic mutation\\t16\\tnd\\t[65] Bladder\\t8\\tHomozygous deletion\\t12\\tnd\\t[67] Breast\\t8\\tHomozygous deletion\\t12\\tnd\\t[67] 456\\tLoss of expression\\t2\\tInversely correlates with the presence of LN metastases\\t[148] Cervical\\t25\\tIntragenic mutation\\t4\\tnd\\t[66] Colon  JP\\t54\\tGermline mutation\\t17\\tHigher frequency of gastrointestinal cancer\\t[57] 9\\tGermline mutation\\t56\\tnd\\t[13] 29\\tGermline mutation\\t24\\tPrevalence of massive gastric polyposis\\t[58] 77\\tGermline mutation\\t18\\tnd\\t[59]  FAP\\t52\\tIntragenic mutation\\t11\\tnd\\t[73]  HNPCC\\t9\\tIntragenic mutation\\t11\\tnd\\t[73]  Sporadic\\t30\\tIntragenic mutation\\t13\\tnd\\t[70] 64\\tIntragenic mutation\\t11\\tnd\\t[71] 31\\tIntragenic mutation\\t16\\tnd\\t[72] 111\\tIntragenic mutation\\t15\\tAdvanced stage and metastasis\\t[73] 29\\tIntragenic mutation\\t21\\tLiver metastases\\t[74] 9\\tIntragenic mutation\\t11\\tnd\\t[75] 100\\tIntragenic mutation\\t5\\tnd\\t[76] 83\\tLoss of expression\\t8\\tLate tumor stage + borderline correlation with distant metastases\\t[141]  MSI−\\t32\\tIntragenic mutation\\t56\\tSMAD4 mutations mutually exclusive of MSI\\t[77]  MSI+\\t12\\tIntragenic mutation\\t0\\tSMAD4 mutations mutually exclusive of MSI\\t[77] Esophageal (SCC)\\t170\\tLoss of Smad4 staining\\t22\\tMetastatic spreading\\t[147] HCC\\t35\\tIntragenic mutation\\t6\\tnd\\t[68] Lung (NSCLC)\\t42\\tIntragenic mutation\\t7\\tnd\\t[69] Ovarian\\t8\\tIntragenic mutation\\t12\\tnd\\t[67] Pancreatic\\t41\\tHomozygous deletion and intragenic mutation\\t49\\tnd\\t[12] Small intestine\\t17\\tIntragenic mutation\\t24\\tnd\\t[168]   Smad2 (18q21; SMAD2)\\tCervical\\t25\\tIntragenic mutation\\t8\\tnd\\t[66] Colon\\t36\\tHomozygous deletion and intragenic mutation\\t8\\tnd\\t[88] 66\\tIntragenic mutation\\t6\\tnd\\t[87] 29\\tIntragenic mutation\\t10.3\\tNo correlation with clinicopathological finding\\t[74] HCC\\t35\\tIntragenic mutation\\t3\\tnd\\t[68] Lung (NSCLC)\\t57\\tIntragenic mutation\\t2\\tnd\\t[86]   Smad3 (15q21-22; SMAD3)\\tGastric\\t30\\tDownregulation\\t38\\tnd\\t[92]  ANTAGONISTS   Smad7 (18q21; SMAD7)\\tColon\\t233\\tDeletion\\t48\\tnd\\t[97] 233\\tAmplification\\t10\\tnd\\t[97] 178\\tDeletion\\t43\\tBetter prognosis\\t[17] 178\\tAmplification\\t15\\tPoor prognosis\\t[17]   Smurf2 (17q22-23; SMURF2)\\tEsophageal (SCC)\\t80\\tHigh expression level\\t56.3\\tDepth of invasion, LN metastasis and poor survival\\t[16]   Ski (1p36.3; SKI)\\tColon\\t179\\tPartial or complete allelic loss\\t41\\tNo correlation with prognosis\\t[101] 179\\tAmplification\\t10.1\\tReduced relapse-free and overall survival\\t[101] Melanoma\\t44\\tNuclear staining\\tnd\\tPre-invasive melanoma\\t[107] 44\\tNuclear and cytoplasmic staining\\tnd\\tPrimary invasive melanoma and metastasis\\t[107]   SnoN (3q36; SNON)\\tBreast\\t1122\\tStrong nuclear staining\\t70\\tFavorable outcome\\t[106] 1122\\tStrong cytoplasmic staining\\t20\\tWorse prognosis\\t[106] Colon\\t179\\tPartial or complete allelic loss\\t55.2\\tNo correlation\\t[101] 179\\tAmplification\\t15.1\\tNo correlation\\t[101] Esophageal (SCC)\\t80\\tHigh expression level\\tnd\\tDepth of invasion and pathological stage\\t[110] 44\\tAmplification\\t11.4\\tIncreased SnoN expression\\t[102] nd: not determined.  a Number of tumor samples.  b Frequency.  c References.  Table options BAT-RII mutations are specifically associated with MSI and such mutations are very infrequent or non-existent in tumors without MSI (also called MSS for microsatellite stable) with the exception of HCC associated with Hepatitis C Virus (HCV) [29]. However, non-BAT-RII mutations that have been shown to occur occasionally on the second allele of TGFBR2 in MSI tumors, are much more frequently observed in tumors without MSI ( Table 1). Indeed, somatic point mutations in TGFBR2 have been identified in 15% of non-MSI colon cancers [11], which taken together with the BAT-RII mutations, means that 28% of colon cancers have TGFBR2 mutations. In addition, TGFBR2 mutations are found in 25% of ovarian cancers, 21% of HNSCC and 12% of breast cancers [30]; [31] ;  [32], but are very rare in pancreatic cancer. Most of the somatic point mutations in TGFBR2 are missense mutations that occur on both alleles in the highly conserved serine-threonine kinase domain or the transmembrane domain. Some have been proposed to induce a conformational change in the protein that could affect transphosphorylation of TβRI [33]. Others, that have been found to be heterozygous in lymphoma (D404G) or esophageal carcinomas (E526Q), have been proposed to act as dominant-negative mutations [34] ;  [35]. The D404G mutation inhibits the function of the wild-type receptor by preventing its appearance on the plasma membrane [34].  Beside mutations in the coding region of TGFBR2, loss of expression of TβRII is another highly frequent alteration observed in tumors ( Table 1). It is observed in 44% of NSCLCs, in 44% of bladder cancers, in about 30% of head and neck squamous cell carcinomas (HNSCC) of the esophageus, in 23% of ovarian carcinomas, 12.5% of prostate cancers and also in breast cancers [32]; [36]; [37]; [38]; [39]; [40] ;  [41]. It is either characterized as a loss of TβRII protein or loss of mRNA expression in tumor cells [32], suggesting that downregulation of TβRII expression in tumors is the main cause for this alteration. In NSCLC, downregulation of TβRII is correlated with CpG methylation of the TGFBR2 promoter [38], which could account for loss of TβRII expression. Mutations in the TGFBR2 promoter could also cause loss of TGFBR2 expression and there is evidence for this. An A → G mutation at position −364 of the 5′ untranslated region of the TGFBR2 gene that results in a significantly decreased transcriptional activity of the TGFBR2 promoter [42] has been identified in a squamous carcinoma cell line with reduced levels of TGFBR2 mRNA.  2.1.2. Activin type II receptor (ACVR2)  The Activin type II receptor (encoded by ACVR2), that specifically recognizes Activin and also BMPs as its ligands, is another type II receptor for TGF-β superfamily members that seems to have a tumor suppressor function. Like the TGFBR2 gene, the ACVR2 gene contains poly-adenine tracts in its coding region, which are a target for MSI. Mutations in one of the two 8-bp poly-adenine tracts of ACVR2 have been identified in gastrointestinal tumors associated with MSI with a frequency of 44–90% in colon cancer [43] ;  [44], 44% in gastric carcinomas [45] and also in 3/3 pancreatic tumors investigated [46]. These mutations are similar to those observed in the 10-bp poly-adenine tracts of TGFBR2 and mainly consist of the addition or deletion of one adenine in the tract that generates a frameshift mutation leading to a truncated protein lacking the serine-threonine kinase domain. In most of the cases, these mutations are biallelic, supporting the idea that ACVR2 could have tumor suppressor properties [46]. Moreover, it has been proposed that poly-adenine tracts of less than 10 bp are rarely mutated in MSI tumors. Thus, the prevalence of this mutation in tumors, suggests a positive selection of ACVR2 inactivation during tumor progression.  2.1.3. TGF-β type I receptor (TβRI/Alk5)  No mutations of the TGF-β type I receptor, TβRI, have been associated with MSI in gastrointestinal cancer, presumably because, unlike TGFBR2, it lacks a repetitive nucleotide sequence that could be a target for genomic instability. However, alteration of this gene due to somatic mutation has been reported in tumors, although at very low frequency ( Table 1). Homozygous deletion of TβRI has been identified in 1/97 pancreatic and 1/12 biliary adenocarcinomas [47] and mutations in the TβRI coding sequence have been found in about 30% of ovarian carcinomas, 17% of HNSCC tumors and in an average of 16% of breast carcinomas and metastases [48]; [49]; [50] ;  [51]. Some mutations, such as those found in ovarian tumors and in HNSCC [48] ;  [49], are frameshift mutations in exon 2 or 5 of TβRI, which are presumed to generate truncated proteins that lack both the catalytic and transmembrane domains in the first case, or just the catalytic domain in the second case. Missense mutations, which are clustered in the catalytic domain of the receptor, have also been identified [50]. Other mutations are found between the transmembrane and extracellular domains and one of these, A230T, has been shown to cause rapid protein degradation within the endoplasmic reticulum [48]. Interestingly, the S387Y mutation, which is located in the catalytic core of the TβRI kinase domain, has been specifically associated with metastatic tumors from both breast and HNSCC [48] ;  [51], although it should be noted that another study failed to identify such mutations in metastases of breast, lung and colon cancer [52].  Whilst mutations in TβRI are relatively rare, loss of expression of TβRI is more frequently observed ( Table 1). It has been found in 53% HNSCC, 30.5% bladder carcinomas and 25% prostate cancers [37]; [41] ;  [53]. In gastric cancer, no mutations of TβRI have been found, but downregulation of the TβRI, demonstrated by a reduction in the level of TβRI mRNA levels, is observed in up to 82% gastric carcinomas when compared to the corresponding normal mucosa [54]. This strongly suggests that downregulation of TβRI is important for the progression of gastric cancers. One mechanism by which this downregulation could occur is through methylation of the promoter. Hypermethylation of CpG islands localized in the 5′ region of the TβRI gene has been observed in 12.5% gastric cancers in one study [55] and 50% of gastric cancers in a second study [56]. Treatment of gastric cell lines with such hypermethylation with a demethylating agent results in increased expression of the TβRI gene [55] ;  [56]. Moreover, in gastric cancers, methylation of the TβRI promoter has been detected in 64% of MSI cases compared with only 26.7% of MSS cases [56]. This suggests that, like the BAT-RII mutation, hypermethylation of the TβRI promoter is an alteration that is frequently selected for during gastric tumorigenesis associated with MSI.  2.1.4. BMP type I receptor (BMPR1A/Alk3)  As discussed above, JP is an autosomal dominant gastrointestinal polyposis syndrome in which patients are at risk of developing gastrointestinal cancer. The BMP type I receptor (Alk3) gene, BMPR1A, is located on chromosomal region 10q22-23, which has been mapped as a locus linked to JP [14], and germline mutations of BMPR1A have been identified in between 17 and 40% patients with JP, depending on the study [57]; [58]; [59] ;  [60]. Most of the mutations are microdeletions or non-sense mutations that result in a frameshift, leading to production of a truncated protein. Mutant receptors that only lack the kinase domain are expected to act as dominant-negatives, since BMPs would still be able to bind the receptor, but no signaling would occur. In contrast, mutant receptors lacking the transmembrane domain would be non-functional, and these mutations would be recessive. Some missense mutations have also been identified in JP patients that could result in conformational changes with functional consequences. Hence, the identification of BMPR1A mutations in an average of 20% of JP patients provides strong evidence for its involvement in the development of JP. Strikingly, no mutations of BMPR1A have been identified so far in sporadic carcinomas.  2.1.5. Activin type I receptor (ACVR1B/Alk4)  Two examples of mutations in the ACVR1B gene, which encodes the Activin type I receptor, Alk4, have been reported out of 95 pancreatic cancers studied [61]. One mutation is a homozygous deletion of 657 bp, which occurs at a 4-bp repeat sequence (5′-ggct-3′) and removes the entire exon 8 of ACVR1B. The other is a 5-bp deletion that would cause a frameshift and early termination of translation, and is associated with LOH. Both mutations eliminate part of the kinase domain of ACVR1B. These are the only examples so far which argue for a potential tumor suppressor role for ACVR1B.  2.2. SMAD alterations in human tumors  2.2.1. SMAD4 (DPC4/MADH4)  LOH on chromosome 18q is found in about 30% of breast, prostate, neuroblastoma and cervical cancers, and even more frequently in HNSCC (40%), NSCLC (56%), colon cancer (60%), gastric cancer (61%) and in up to 90% of pancreatic tumors [62] ;  [63]. The minimal lost region that has been mapped to 18q21 contains at least three candidate tumor suppressor genes, DCC (deleted in colorectal carcinomas), SMAD4/DPC4 (deleted in pancreatic carcinomas, locus 4, which encodes Smad4) and MADR2/JV18-1 (MAD-related gene 2, which encodes Smad2) [63]. For many years, it was thought that DCC was the targeted tumor suppressor at this locus. However, no mutations have been identified in this gene. In contrast, the SMAD4 gene has been identified as a genetic target of this 18q LOH in about 50% of pancreatic tumors ( Table 1) [12]. SMAD4 is homozygously deleted in about 30% of pancreatic carcinomas and inactivated by intragenic mutation in another 20% pancreatic tumors in conjunction with loss of the other allele. This means that genetic alteration of SMAD4 in pancreatic tumors is almost as common as mutations in K-Ras (80%), p53 (70%) and CDKN2A (p16INK4a) (80%) [64].  By contrast, searches for SMAD4 mutations in other tumor types carrying 18q LOH has led to the identification of mutations at a much lower frequency ( Table 1): 16% biliary tract cancers, 12% breast cancers, bladder cancers and ovarian cancers, 7% lung cancers, 6% HCC and 4% cervical cancers [65]; [66]; [67]; [68] ;  [69]. In colorectal carcinomas, SMAD4 mutations have been identified at an average rate of 13% when many different analyses are considered [70]; [71]; [72]; [73]; [74]; [75] ;  [76], and mutations in SMAD4 were also found in 11% of familial adenomatous polyposis (FAP) carcinomas and in 11% of HNPCC [73]. The frequency of SMAD4 mutation in colon cancer however increases significantly when the analysis is made with respect to the MSI status of the tumor. Whereas no SMAD4 mutations have been detected in MSI-associated colorectal cancers, 56% of non-MSI cancers exhibit SMAD4 alterations [77]. Moreover, germline mutations in SMAD4 were identified in JP families after the discovery that this syndrome showed linkage to markers localized on 18q21 [13]. To date, SMAD4 germline mutations have been detected in more than 26 JP families and account for between 17 and 56% of JP patients. The polyps from patients with a germline SMAD4 mutation have also lost the wild-type allele of the SMAD4 locus at 18q, which supports the idea that SMAD4 acts as a tumor suppressor in JP [78]. This idea is also strengthened by the finding that SMAD4 heterozygous knockout mice start to develop gastric polyps after the age of 1 year that are histologically similar to human juvenile polyps [79].  In most cases, SMAD4 mutations are clustered after codon 200 in the C-terminal, so-called MH2 domain of the protein that is involved in homo- and/or hetero-oligomerization of Smad4 with the R-Smads [80]. The most frequent mutations observed in this region are non-sense or frameshift mutations that generate a truncated protein unable to bind other Smads. Such C-terminal truncations can also affect Smad4 protein stability [81]. Missense mutations in the MH2 or in the N-terminal MH1 domain have also been identified. Depending on the amino-acid substitution, missense mutations have been shown in vitro to disrupt homo-oligomerization of Smad4 or its interaction with activated Smad2 (D351H, R361C, V370D, D537E) [80]. Alternatively, they can generate a dominant-negative auto-inhibited form of Smad4 unable to bind Smad2 (R100T) [82], or an unstable protein that is degraded rapidly in comparison with the wild-type protein (R100T) [83] ;  [84], or they can inhibit DNA binding (L43S, G65V, R100T, P130) [84] or inhibit nuclear translocation (L43S, R100T) [84]. Immunohistochemistry has revealed however, both in pancreatic cancers and in JP polyps/tumors, that almost all of the tumors tested that contain mutated SMAD4 lack Smad4 protein expression, in contrast with the strong expression of SMAD4 in normal tissues or in carcinomas expressing wild-type Smad4 [78] ;  [85]. This suggests that SMAD4 mutations most commonly generate protein instability in tumors and that loss of the protein expression might be the major consequence of SMAD4 alterations in tumors.  2.2.2. SMAD2  The gene that encodes Smad2 maps close to SMAD4 at 18q21 and has also been proposed to be a putative tumor suppressor target for 18q LOH. Mutation of SMAD2 occurs at very low frequency ( Table 1), in 8% cervical cancers, about 8% colorectal cancers, 3% HCC and 2% NSCLC [66]; [68]; [74]; [86]; [87] ;  [88]. Two cases of homozygous deletion have been observed in colon cancer, but most of the mutations are missense mutations either in the MH1 or MH2 domains that have been proposed to affect Smad2 phosphorylation (D450E, P445H) [87] ;  [89], nuclear translocation (P445H) [90] or to increase Smad2 auto-inhibition (R133C) [82] or to decrease protein stability (L440R, Q407R) [87] ;  [91].  2.2.3. SMAD3  Mutation of the SMAD3 gene, located on 15q21-q22, has not been identified in human tumors. However, loss of expression of Smad3 in tumor samples, compared with surrounding mucosa, has been identified in three out of eight gastric cancers and in two out of nine gastric cell lines that have lost some TGF-β responsiveness [92]. Tumor suppressive activity of TGF-β in those two cell lines was restored by reintroducing Smad3. This suggests that SMAD3 might be a target for epigenetic inactivation during gastric tumorigenesis and supports the finding that SMAD3−/− mice develop metastatic colon carcinoma [93]. However, it is likely that the putative tumor suppressor function of Smad3 is dependent on other factors, as mice with different genetic backgrounds did not show any malignancy [94] ;  [95].  2.3. Overexpression of antagonists  As discussed in Section 1, there are several known inhibitors of the TGF-β signaling pathway, for example, the inhibitory Smads, Smad6 and Smad7, the E3 ubiquitin ligases, Smurf1 and 2 and the transcriptional repressors, Ski and SnoN. Overexpression of these inhibitors is known to inhibit tumor suppressive functions of TGF-β by decreasing TGF-β signaling and inhibiting TGF-β-induced growth arrest in tissue culture cell lines. Overexpression and/or amplification of some of these inhibitory components has been detected in some human tumors.  2.3.1. SMAD7  The gene encoding SMAD7 is located in the 18q21 region and has been mapped between the genes encoding SMAD2 and SMAD4 within a 4 Mb cluster [96]. Interestingly, when the frequency of deletion of these 3 genes was compared in 233 DNA samples of colorectal cancers, SMAD4 and SMAD2 showed a higher deletion frequency (66 and 64%, respectively) than SMAD7 (48%). The SMAD7 locus also appears to be more frequently amplified (10%) than the SMAD2 (6%) and SMAD4 (7%) loci [97]. Therefore, retention and even amplification of SMAD7 seems to be a selected event during progression of colorectal tumors. This finding is in agreement with the frequent overexpression of SMAD7 observed in endometrial and thyroid follicular carcinomas [98] ;  [99]. Moreover, ectopic Smad7 overexpression has been shown to increase the tumorigenicity of a pancreatic tumor cell line when it is injected into nude mice [100].  2.3.2. SMURF2  There is evidence for overexpression of the TGF-β antagonist, Smurf2 in human tumors. In one study comprising 80 patients with esophageal squamous cell carcinoma (SCC), high levels of Smurf2 expression were detected by immunohistochemical staining in 56.3% of the surgical specimens [16]. Where staining was seen in tumor tissue, it was particularly high at the tumor front. Since Smurf2 expression in tumors appears to be associated with a loss of phosphorylated Smad2 staining, it is likely that overexpression of Smurf2 inhibits TGF-β signaling in tumors. However, whether overexpression of Smurf2 is sufficient to promote tumorigenesis, remains unclear.  2.3.3. SKI/SNON  As observed for SMAD7, SKI and the closely related gene, SNON, can be both deleted or amplified in human tumors. In 179 colorectal cancer samples, partial or complete allelic loss was found in 41.5 and 55.2% for SKI and SNON, respectively, whereas amplification was found in 10.1 and 15.1%, respectively [101]. The more frequent deletion of SKI and SNON suggests that they might act as tumor suppressor genes, which contradicts the fact that they were initially described as proto-oncogenes. Moreover, chromosome regions 1p36 and 3q36 where SKI and SNON are located, respectively, have been reported to be deleted at high frequency in some cancers [102] ;  [103], and SKI−/+ and SNON−/+ heterozygous mice show an increased number of lymphomas compared to wild-type mice when challenged with carcinogens [104] ;  [105]. However, other evidence suggests that SKI/SNON do indeed act as oncogenes in human cancer, rather than as tumor suppressors. First, no mutations of SKI/SNON have been reported so far, suggesting that the chromosome region where they are located bears another tumor suppressor. Second, the amplification at 3q26 is accompanied by a significant increase in the expression level of SnoN in esophageal SCC, suggesting that SNON is a probable target for gene amplification in this region [102]. Third, increased expression of SKI/SNON has been detected in many different tumor types including breast cancer, melanoma and esophageal SCC [106]; [107]; [108]; [109] ;  [110]. Finally, the amplification observed for SKI/SNON in colorectal cancer is clearly associated with poor outcome, whereas deletions are not. This suggests that SKI/SNON amplification is the selected event during the development of malignancy, whilst deletions are not [101]. Overexpression of Ski/SnoN is able to repress growth arrest induced by TGF-β, which could explain why this event is selected during tumor progression [111].  Another intriguing alteration observed for Ski/SnoN in human tumors is a change in their subcellular localization, which is normally predominantly nuclear. High nuclear levels of SnoN have been detected in 70% of breast carcinomas, whereas 20% show high cytoplasmic staining [106]. Moreover, nuclear and cytoplasmic staining of Ski is frequently observed in advanced stage melanoma compared to normal skin tissue [107]. Cytoplasmic Ski is able to form complexes with Smad3 in the cytoplasm, which prevents its nuclear translocation [107]. Therefore, a combination of the cytoplasmic and nuclear Ski could provide two levels of Smad inhibition in tumors and could lead to a stronger disruption of Smad signaling.  2.4. Increase of TGF-β secretion and activity in tumors  Most of the tumor cell lines that harbor mutations in one of the components of the TGF-β pathway are resistant to TGF-β-induced growth arrest, which is a tumor suppressive response of TGF-β. However, the role of TGF-β in tumorigenesis appears to be more complex than that of a classic tumor suppressor. Unexpectedly for a tumor suppressor, TGF-β is very frequently overexpressed either at the mRNA or protein level in most tumor types (Table 1) including: breast, esophageal, lung, colorectal, gastric, pancreatric, prostate and HCC [37]; [112]; [113]; [114]; [115]; [116]; [117]; [118]; [119]; [120]; [121]; [122]; [123] ;  [124]. Immunostaining indicates that both intracellular TGF-β and extracellular deposition of TGF-β is increased in mammary tumors [113], and moreover, tissue adjacent to pancreatic tumors shows increased immunoreactivity [121]. High levels of TGF-β were also detected in the serum of patients with colorectal carcinomas or HCCs compared to normal patients [116]; [117]; [123] ;  [124] and serum levels were shown to decrease to the normal range following surgical resection of the tumor [117]. Altogether, these observations show that not only is TGF-β production increased in tumor cells, but so is its secretion in adjacent tissues and in the blood. Hence, both autocrine and paracrine effects of TGF-β may promote tumor progression. This tumor promoting role of TGF-β in tumors has been demonstrated in transgenic mice when TGF-β1 expression was targeted in keratinocytes [125]. TGF-β1-expressing mice exhibit a strong resistance to benign tumor formation after treatment with carcinogens, which confirms the “protective” tumor suppressor function of TGF-β. However, once a tumor has developed, the malignant conversion rate was significantly increased in transgenic mice compared to controls, which confirms that TGF-β1 expression in tumors has a tumor promoting effect.  3. Clinical correlations for the different alterations of the TGF-β pathway in human cancer Pathological analysis of resected tumor specimens allows the definition of tumor grade which is essential for the evaluation of the treatment needed by the patient. However, patients with tumors of the same stage can have very different clinical outcomes. The molecular analysis of genetic and epigenetic alterations of signaling pathways that occur in the tumors has recently helped to give clues as to the different ways that tumors can evolve, and it is of great interest to try to establish correlations between the molecular defects detected in tumors and clinical outcomes, such as survival and tendency for metastatic spread. The molecular defects could thus provide additional predictive markers to determine prognosis and plan appropriate treatment of patients. Having detailed the alterations that occur in TGF-β superfamily signaling pathway components, we now review the different clinical correlations that have been observed for these pathways in human cancer. It should be noted that such correlations do not necessarily imply that defects in the TGF-β signaling pathways are the primary cause of the observed clinical features, since these molecular defects often occur in conjunction with other mutations or other mutagenic events such as MSI. Moreover, the number of samples in each study (Table 1) and whether the result has been reproduced independently also have to be taken in account when determining the statistical relevance of these correlations.  3.1. TGF-β  Consistent with the idea that TGF-β can act as a tumor promoter, increased TGF-β1 mRNA or protein expression in tumor cells and/or its plasma levels has been correlated with advanced tumor progression in colorectal cancer, gastric carcinoma, NSCLC and prostate cancer ( Table 1) [115]; [116]; [117]; [120] ;  [122]. Intense TGF-β staining has also been significantly correlated with disease progression to metastasis in breast carcinoma, prostate cancer, and colorectal cancer [112]; [118] ;  [122] and serum levels of TGF-β are significantly higher in patients with NSCLC or colorectal carcinoma that have lymph node metastases [115] ;  [117]. Moreover, TGF-β staining has been shown to be stronger in the invading local lymph node metastases compared to the paired primary tumor site in colorectal and breast cancer [113] ;  [119]. Primary tumors that have metastasized have stronger TGF-β staining than those that have not, and the metastases also exhibit strong TGF-β staining. Thus, there is good evidence to suggest that TGF-β is a potent factor that promotes metastasis in late stage tumors. This correlates with the ability of TGF-β to induce epithelial–mesenchymal transition (EMT), invasion and migration of both untransformed and tumor cell lines in vitro [126]; [127]; [128] ;  [129].  High levels of TGF-β in tumors have also been shown to correlate with angiogenesis. TGF-β expression is associated with high vascular density in prostate cancer [122] and high TGF-β plasma level in patients with HCC correlates with tumor vascularity [123]. Moreover, TGF-β1 expression and VEGF expression are closely correlated in gastric carcinoma [120]. The fact that TGF-β1 knock-down mice exhibit defective vasculogenesis [130] and that TGF-β has been shown to induce VEGF expression [131], strongly suggests that TGF-β might have a pro-angiogenic effect that would promote tumor progression.  Finally, and possibly as a consequence of the pro-angiogenic and pro-metastatic behavior of TGF-β, high expression of TGF-β in tumors is significantly associated with poor prognosis in many cancers [115]; [120] ;  [121]. The 5-year survival rate is better in patients that have TGF-β1-negative gastric tumors than in those that have TGF-β1-positive tumors [120] and lack of expression of TGF-β isoforms in tumors correlates with increased post-operative survival time [121]. Taken together, these observations indicate that TGF-β isoforms might be useful biomarkers in the clinical management of patients with pancreatic, colorectal, gastric, prostate and breast carcinomas.  3.2. The type I and type II TGF-β receptors  In colon cancer, no BAT-RII mutations have been detected in early MSI adenomas which suggests that the acquision of TGFBR2 mutations is a late event that would be selected during adenoma to carcinoma conversion [132]. In gastric cancer, BAT-RII mutations are restricted to advanced carcinomas [56], although hypermethylation of TβRI has been found both in early and in advanced carcinomas. This suggests that inactivation of TβRI is an earlier event than TβRII inactivation during tumor development.  What are the clinical consequences of TGF-β receptor alterations? In cancers from the gastrointestinal tract, the presence of MSI, which is associated with BAT-RII mutations, correlates with a better prognosis for gastric and colon cancers [133]. Moreover, the 5-year survival rate among patients whose colon cancer has high MSI has been shown to be greater where there are TGFBR2 mutations in the tumors compared to patients whose tumors do not have TGFBR2 mutations [134] (Table 1). In breast cancer-negative for the oestrogen receptor, loss of expression of TβRII is correlated with a longer overall survival [135]. Taken together, this suggests that TβRII alterations in cancer are associated with a better prognosis.  In contrast, when reduced expression of both TβRII and TβRI is considered, a correlation with tumor grade in prostate, breast and bladder cancers has been observed [39]; [41] ;  [53]. Reduced expression of TβRII and TβRI in esophageal SCC appears also to be correlated with depth of invasion, lymph node metastasis, tumor stage and poor prognosis [37]. Interestingly, loss of expression of TβRI seems to exhibit a more pronounced correlation with aggressive tumor phenotype than do TβRII alterations. For example, in bladder cancer, only a decrease in TβRI expression was associated with an increased probability of tumor progression and a decreased probability of survival [53]. Moreover, in breast cancer, lymph node status was not significantly associated with loss of TβRII expression [39], but mutation of TβRI appears to be significantly more frequent in lymph node metastasis [51]. Finally, in prostate cancer, only the loss of TβRI expression showed a statistically significant association with the tumor stage, 4-year survival rate and serological recurrence rate after radical prostatectomy [40]. Therefore, it appears that alterations of TβRI and TβRII do not have the same consequences for tumor progression. It is not yet clear why this is the case, although it might suggest that these receptors do not always function as obligate partners. Alternatively, mutation of the type II receptor may abolish all TGF-β signaling, which is particularly the case for the BAT-RII mutations as these lead to a non-functional protein. In contrast, alteration of TβRI by reduced expression or point mutation may only affect some aspects of TGF-β signaling, as discussed in Section 4. As yet, no consistent correlations with clinical outcome have been reported for the ACVR2 and Alk4 receptors.  3.3. SMAD4 and BMPR1A in Juvenile Polyposis  Mutations in SMAD4 or BMPR1A account for an average of about 50% of all JP cases reported so far [57]; [58]; [59] ;  [60]. Patients with germline SMAD4 or BMPR1A mutations (MUT+) have a more prominent JP phenotype than those without (MUT−), although the mean age for symptoms and diagnosis is higher for MUT+ patients than for MUT− cases. This suggests that MUT− patients may carry an as yet undiscovered mutated gene, which predisposes patients to an earlier age of onset for the disease [57]. When compared, patients with SMAD4 mutations show a higher predisposition to develop upper gastrointestinal polyposis than those with BMPR1A mutations, or without any identified mutations [57]. A more striking correlation was also reported in another study, where a massive gastric polyposis was more commonly observed in patients with SMAD4 mutations, than in patients with BMPR1A mutations or without identified mutations [58]. Taken together, these data suggest that SMAD4 mutations have a stronger phenotype that predispose patients to polyposis, than do BMPR1A mutations.  3.4. SMADs  Loss at 18q is associated with unfavorable outcome in patients with colorectal cancer stage II, gastric cancer, HNSCC and pancreatic adenocarcinoma [136]; [137]; [138] ;  [139], suggesting that mutation of the putative tumor suppressor on this chromosome arm may play significant roles in disease progression. In colorectal carcinoma, 18q LOH is not an early event [140] that would initiate tumorigenesis but rather, promotes progression of adenoma to invasive carcinoma, as do SMAD4 mutations and loss of SMAD4 expression [73] ;  [141]. These alterations were observed in between 22 and 46% of invasive carcinomas, whereas none were observed in adenomas or stage I adenocarcinomas [73]; [140] ;  [141]. SMAD4 mutations are associated with loss of the other allele in 95% invasive and metastasized carcinomas, suggesting that the SMAD4 gene is one of true targets of 18q LOH, and that its inactivation is involved in more advanced stages of human colorectal carcinogenesis, such as development of distant metastases [73].  Alteration of SMAD4 has also been shown to occur at later stages of pancreatic ductal tumorigenesis, based on the fact that immunodetection of Smad4 is lost in 31% of the advanced pancreatic duct lesion (PanIn-3 lesions), but not in those characterized as earlier duct lesions (PanIn-1A, PanIn-1B and PanIn-2) [142]. Another study of intraductal papillary–mucinous tumor (IPMT), thought to correspond to one of the premalignant neoplasms of the pancreas, has also suggested that inactivation of SMAD4 is a late event, since no loss of SMAD4 expression has been detected in these tumors [143]. Curiously, 18q LOH has been described as an early event in pancreatic tumorigenesis [144] and LOH at the SMAD4 locus is frequently detected in IPMT [143]. This suggests that loss of 18q could occur as a genetic change with no pathological significance in the early stages of tumorigenesis. Alternatively, there may be an unknown tumor suppressor gene at 18q, in addition to SMAD4, that is involved in pancreatic ductal carcinogenesis [143]. Although this needs to be clarified, it is noticeable that SMAD4 alterations only account for about 50–60% of the 18q21 allelic loss observed in colorectal and pancreatic cancers, suggesting the existence of another tumor suppressor at this locus [73].  Thus, SMAD4 inactivation appears to be an intermediate event in both colorectal and pancreatic tumorigenesis and might therefore play a role in the acquisition of a more advanced tumor phenotype. Indeed, there is evidence for this from mouse studies since heterozygous SMAD4−/+/APCΔ716 mice develop colon cancer, whereas the APCΔ716 mouse only develops intestinal adenomas [145], confirming that SMAD4 inactivation promotes adenoma to carcinoma progression. It has been proposed that SMAD4 mutations occur during the transition of adenoma to carcinoma after the acquisition of sequential mutations of APC and K-Ras in human colorectal cancer [21] or after K-Ras, CDKN2A (p16INK4a) and p53 mutations during pancreatic cancer progression [146]. Interestingly, significant correlation of SMAD4 alterations with metastasis has been detected in both HNSCC and sporadic colon carcinomas [73]; [74] ;  [147], although the opposite was found in breast cancer [148]. However, it is not yet clear whether alteration of SMAD4 could be a prognostic marker that correlates with patient survival and clinical outcome. In contrast to SMAD4, SMAD2, which is also localized on 18q21, is very infrequently mutated and no clinical correlations with SMAD2 alterations have been observed in any tumors.  3.5. TGF-β signaling pathway inhibitory components  Overexpression/amplification of inhibitors of the TGF-β pathway has been linked to a more aggressive tumor phenotype (Table 1). SMAD7, which also maps to 18q21, has been proposed to be a prognostic marker in patients with colorectal cancer [17]. Patients with deletion of SMAD7 have a favorable outcome. Conversely, amplification of SMAD7 is associated with a significantly worse prognosis, which suggests a graded effect on patient outcome, depending on SMAD7 gene copy number. Similarly, SMURF2 overexpression in esophageal SCC has been correlated with the depth of invasion, lymph node metastasis and a poor survival rate [16].  Amplification of the negative regulator, SKI, in colorectal cancer correlates with reduced relapse-free and overall survival, whereas deletions do not correlate with prognosis. No correlation, however, was observed in the same study for the amplification of the related gene, SNON [101], although immunostaining analysis has shown that reduced expression of SNON in oestrogen receptor-positive breast tumors is significantly associated with a favorable clinical outcome [106]. As mentioned above, changes in subcellular localization of Ski/SnoN are also observed in tumors. Ski is predominantly nuclear in pre-invasive melanomas, but is both in the nucleus and cytoplasm in invasive and metastatic melanomas [107], although the mechanism underlying this remains to be defined. It appears that cytoplasmic localization of Ski/SnoN is associated with a worse prognosis. High levels of cytoplasmic SnoN in breast cancer are associated with adverse prognostic features, whereas high levels of nuclear SnoN were associated with favorable prognostic features [106]. Although Ski and SnoN seem to share some similarities, it is still not clear whether they act through the same mechanism during tumor progression. Ski appears to be an independent prognosis factor in early stage colorectal cancer [101], whereas SnoN is a novel prognostic marker for ER-positive breast cancers [106].  4. Are all TGF-β pathway alterations in tumors equivalent? 4.1. Mutations that affect different TGF-β superfamily receptors might have different consequences in human cancer  Intuitively, it seems likely that mutations in human tumors of different receptors for the different TGF-β superfamily ligands might have different outcomes, since the Activin, BMP and TGF-β pathways are known to activate distinct sets of target genes and hence have different effects on cellular processes (Fig. 1; for a review see [1]). Thus, mutations of TβRI or TGFBR2 in tumors would specifically affect the TGF-β pathway, whilst mutations of BMPR1A (Alk3), which are only observed in JP patients would only affect the BMP pathway. Similarly, mutations of the ACVR1B (Alk4) receptor should specifically affect the Activin pathway, whilst mutations in ACVR2 could affect both BMP and Activin signaling. However, whilst the role of TGF-β in tumor progression and its overexpression in cancer has been well documented, it is still unclear to what extent Activin and BMP signaling are involved in cancer. As discussed in the section above though, alterations in some of the different receptors do have different clinical outcomes. Importantly, most of the colorectal and pancreatic tumors that have poly(A)8 tract mutations of the Activin and BMP type II receptor, ACVR2, have also poly(A)10 tract mutations of the TGF-β type II receptor TGFBR2 [46]. This suggests that alteration of both the BMP/Activin and the TGF-β pathways might be required for colorectal and pancreatic tumorigenesis, and also indicates that pertubation of the BMP/Activin pathway might not be sufficient to trigger tumorigenesis alone.  4.2. Two independent mutations in different pathway components sometimes occur in the same tumor  More intriguing and unexpected is the observation that hypermethylation of TβRI is frequently associated with BAT-II mutations of TGFBR2 in MSI sporadic gastric carcinomas [56]. This is surprising as these alterations should both affect the TGF-β pathway. Mutations are acquired under selective pressure and therefore usually, if signaling through a pathway has been altered due to mutation in one component, there is no selective pressure to acquire another mutation in the same pathway. Therefore, in the case of the TGF-β type I and type II receptors, it is not clear whether the combination of mutations in both components strengthens a common effect or whether alteration of the two receptors has two distinct effects. It is possible that TβRI might in some circumstances function independently of TβRII, and have some additional effects, perhaps in conjuction with another type II receptor. This is suggested by the fact that mutation or downregulation of TβRI has a worse prognosis than mutation of TβRII. However, the alternative explanation for the difference in prognosis is that loss of TβRII may simply abolish all TGF-β signaling, whilst alterations of TβRI may still allow some signal transduction (see below).  Mutations in SMAD4 have also been found in conjunction with mutations in one of the TGF-β superfamily receptors, either ACVR1B [61] or TGFBR2 [11] in the same tumor samples or cell lines. Again, this is unexpected. However, given that Smad4 is a common component of the TGF-β, BMP and Activin pathways, this might reflect the fact that signaling through one pathway is first abolished at early stages of tumorigenesis by the acquisition of a receptor mutation, and then signaling through the other TGF-β superfamily pathways is compromised by the SMAD4 mutation acquired later in tumorigenesis [61]. Another possibility is that SMAD4 may play an additional role to its role as a signal transducer in the Activin and TGF-β pathways. Indeed, it has been suggested that Smad4 could act with SnoN as a transcriptional repressor in unstimulated cells [149]. Loss of Smad4 might therefore lead to upregulation of target genes independently of TGF-β superfamily signaling, which could contribute to tumor development or progression.  4.3. Some mutations in components of the TGF-β pathway might differentially affect TGF-β responses  As mentioned above, of the TGF-β superfamily signaling pathways that are altered in human cancer, the TGF-β pathway itself is by far the best understood. However, it is clear that even for this one pathway, mutations in different components of the pathway often do not have the same effects on tumor development, even when only positively acting components are considered (Fig. 1). In addition, in some contexts, alteration of one component might have a certain clinical outcome, yet in another type of cancer, alteration of the same component might have very different consequences. A good example of this is the differential effects that alteration of the type II receptor can have on tumor development in different contexts. In MSI cancers of the gastrointestinal tract, the presence of BAT-RII mutations correlates with a better prognosis [133]. However, in prostate cancer, loss of tumor TβRII expression has been found to be frequently associated with TGF-β1 overproduction and patients with both defects show particularly short survival when compared to patients with normal TβRII immunoreactivity [122]. One explanation for this may lie in the dual role that TGF-β plays in tumorigenesis. It is commonly accepted that the TGF-β signaling pathway plays a tumor suppressor role and a tumor promoting role [150]; the tumor suppressive role is thought to be associated with the growth inhibitory and apoptotic responses to TGF-β, and the tumor promoting role thought to reflect the positive effects of TGF-β on tumor cell invasion, EMT and metastasis and also on angiogenesis and immune suppression [151]. When some components of the TGF-β pathway are altered, it may be possible to lose the tumor suppressive effects of TGF-β, but to retain the tumor promoting effects. Thus, in the example above, loss of TβRII expression as detected by immunohistochemistry in the prostate cancer samples may reflect only a reduction in TβRII levels that may be sufficient for the tumor cells to become resistant to the tumor suppressive effects of TGF-β, but still allow the tumor promoting effects. In fact, as detailed below, there is increasing evidence from different studies to indicate that some alterations of TβRII, TβRI, Smad3 or Smad4 observed in human tumors might differentially affect the TGF-β target genes and functions.  4.3.1. TGF-β receptor mutations  Many studies have shown that TβRII mutations in human tumors abolish TGF-β-induced growth inhibition. However, it has been proposed that some point mutations of either TβRI or TβRII might generate receptors unable to induce TGF-β-mediated growth arrest, but still able to affect extracellular matrix remodeling that can be construed as a tumor promoting effect [18]. Deletion or point mutation of the juxtamembrane region of TβRI impaired TGF-β-induced growth arrest, but not induction of PAI-1 and fibronectin [152]. Similar findings were observed for the TβRII germline point mutation T315M identified in a kindred of HNPCC without MSI [153]. When compared to the introduction of wild-type TβRII in the DR26 cell line deficient for TβRII, the T315M mutant failed to restore TGF-β-induced growth arrest and transcriptional upregulation of p15INK4b, but was still able to induce PAI-1, which has been implicated in TGF-β-induced migration [154] ;  [155]. These observations may be related to the finding that a mutant form of TGF-β with a reduced ability to activate the type I and type II TGF-β receptors, is 20% as potent as native TGF-β1 in the induction of PAI-1, but less than 1% as efficient as native TGF-β1 in inhibition of cell growth in the same cell line [156]. A threshold model might explain this uncoupling, where the full activities of both the type II and type I receptors are required for the antiproliferative responses to TGF-β, but not for other TGF-β responses, such as the induction of PAI-1. Weak activation of the pathway could both reduce the levels of active Smads in the nucleus [157], and also their time of residence there [158]. This would affect how strongly TGF-β target genes are induced or repressed, and also the duration of their induction/repression. It is thus tempting to speculate that lowering the activation of the TβRI/TβRII receptor complex in tumors either by point mutation or by decreasing (but not abolishing) receptor levels could impair specifically the tumor suppressor function of TGF-β, but not other pro-oncogenic TGF-β functions. However, whereas the loss of TGF-β-induced growth arrest has been shown many times in tumor cell lines [10] ;  [11], the retention of tumor promoting activity in these instances remains to be demonstrated.  4.3.2. SMAD mutations  In the case of SMAD3, although mutations have not been found, levels of Smad3 have been found to be lower in some human tumor cells, and loss of Smad3 has been shown to accompany EMT in the canine cell line, MDCK [92] ;  [159]. It is interesting to speculate that the lower levels of Smad3 may be sufficient to abolish the growth suppressive effects of TGF-β [94] ;  [160], but not the EMT response and migration that has been associated with TGF-β's tumor promoting effects [159] ;  [160].  Loss of Smad4 was originally proposed to result in loss of all TGF-β signaling, as it was thought to be a critical component of the pathway [161] ;  [162]. However, genetic studies, as well as studies carried out in tissue culture cells, suggest that Smad4 might not be required for all responses by TGF-β family members [163]; [164]; [165]; [166] ;  [167]. We have recently shown, using cell lines expressing a Tet-inducible Smad4 siRNA, that loss of Smad4 leads to loss of only a subset of TGF-β target genes and to a specific loss of TGF-β-mediated growth arrest and migration, but not of the TGF-β-induced cell spreading and EMT [167]. This is in agreement with the proposal that TGF-β-induced EMT might be a Smad-independent process, utilizing the RhoA pathway and its target p160ROCK[126]. These observations suggest that tumor cells that have lost Smad4 might not be sensitive to the tumor suppressor function of TGF-β but still sensitive to TGF-β induced pro-oncogenic functions, such as EMT. More work is required to confirm this idea in human tumors.  5. Unanswered questions Much is now known about the role of TGF-β superfamily signaling pathways in human cancer, in terms of what sort of alterations are observed in components of these pathways in human tumors, and it is beginning to be understood how these alterations correlate with clinical outcome. However, there are still many questions that remain to be answered. What is the relevance of the Activin and BMP pathways to human cancer? Why are only a subset of the receptors for these family members found to be mutated in human tumors? What is the basis for the tissue specificity seen for the alterations in different components of the TGF-β superfamily pathways? For instance, why is SMAD4 predominantly mutated or deleted in colorectal and pancreatic cancers, but not in cancers derived from other organs? What triggers the switch between TGF-β acting as a tumor suppressor, to it acting as a tumor promoter? What role does the tumor stroma play in this process? What is the mechanism of TGF-β's tumor promoting activity? Does the secreted TGF-β have a direct effect on the tumor cells themselves, promoting their invasive and metastatic activities or does it reflect its activity as a promoter of angiogenesis and as an immune suppressor? Finally, with our current knowledge of TGF-β signaling pathways in human cancer, will it be possible to develop therapeutics that will specifically inhibit TGF-β's tumor promoting effects, without inhibiting its tumor suppressive effects? Given the intense activity in this field these questions should soon be answered.  A subset of colorectal carcinomas exhibit a molecular phenotype commonly referred to as high microsatellite instability (MSI-H) [1]. The microsatellite instability (MSI) pathway begins with the inactivation of one of a group of genes responsible for DNA nucleotide mismatch repair (MMR), which leads to extensive mutations in both repetitive and non-repetitive DNA sequences [2–4]. The mechanism of tumorigenesis in MSI-H tumors is thought to involve frameshift mutations of microsatellite repeats within the coding regions of affected genes, and the inactivation of these genes is believed to contribute directly to tumor development and progression [5,6]. The frameshift mutations observed in the affected genes are expected to generate previously undescribed amino acid sequences in the C-terminal part of the respective proteins (Figure S1). If abnormal mRNAs and proteins are generated from the frameshift-mutated genes, tumor-specific antigen may be generated. High peritumoral lymphocytic infiltration and a relatively good prognosis have been reported in MSI-H tumors [7,8].One of the important consequences of frameshift mutations is the formation of premature termination codons (PTCs). In mammalian cells, mRNAs containing a PTC due to a nonsense mutation or a frameshift mutation are recognized and degraded by nonsense-mediated mRNA decay (NMD), thus eliminating the production of the potentially deleterious truncated proteins [9,10]. NMD of mRNAs carrying PTCs is mediated through the recognition of the PTC by its position relative to the 3′-most last exon–exon junction. As a general rule, mammalian transcripts that contain a PTC more than 50–55 nucleotides (nt) upstream of the last exon–exon junction will be subjected to NMD [11,12].Although PTC formation in frameshift mutation-derived mRNAs and their subsequent degradation through NMD is widely accepted, PTCs located within 50–55 nt or downstream of the last exon–exon junction are not recognized by NMD (NMD-irrelevant), and some mRNAs with PTCs more than 50–55 nt upstream of their last exon–exon junction are not degraded by NMD (NMD-escape) [13,14]. In MSI-H tumors, several NMD-sensitive or NMD-escape PTC-containing mRNAs have been reported. A previous study compared the total mRNAs of affected genes from various cell lines [15]. However, this study did not differentiate the proportion of wild-type and mutant mRNAs, and did not confirm the mutant mRNAs through sequencing. This study also did not consider that the amount of mRNA from the affected genes might vary between cell lines. Moreover, the expression statuses and biological effects of the NMD-escape PTC-containing mRNAs are essentially unknown.In order to clarify the protein expression status of affected genes with frameshift mutations and the role of NMD in these mutated genes, we selected MSI-H cancers as a model system because these cancers have accumulated genes with frameshift mutations, and the mRNAs expected from these mutated genes contain PTCs. We analyzed the expression of 20 mutant mRNAs from 12 genes and evaluated their regulation along with the regulation of associated proteins. We demonstrate that some PTC-containing mRNAs escaped from NMD, but did not generate truncated proteins, indicating that PTC-containing transcripts can be regulated either by NMD or translational repression.ResultsGenes with Frameshift Mutations Form PTCs in Their mRNAsTo examine the effect of NMD on the affected genes with frameshift mutations in MSI-H tumors, we selected 12 genes from MSI-H tumors based on the reported frameshift mutation frequencies greater than 30% (ABCF1, ACVR2, hMSH3, hMSH6, hRad50, MARCKS, PRKWNK1, RFC3, SEC63, TAF1B, TCF-4, and TGFBR2). We used genome sequencing of these 12 genes to identify frameshift mutation status. In these 12 genes, we identified 20 frameshift mutations that fell into three categories: 12 mutations were single base pair (bp) deletions, six were 2-bp deletions, and two were single bp insertions in coding mononucleotide repeats (cMNR) (Table S1). All 20 frameshift mutations of the 12 genes resulted in mRNAs containing a PTC (Table 1).thumbnail\\tDownload:PPT PowerPoint slidePNG larger imageTIFF original imageTable 1.Expressions of Frameshift Mutant mRNAs in MMR-Deficient Colorectal Cancer Cell Lineshttps://doi.org/10.1371/journal.pbio.0050109.t001Identification of NMD-Escape PTC-Containing mRNAsWe analyzed mRNA expression of the 12 genes by reverse transcriptase PCR (RT-PCR) in seven MMR-deficient (LS174T, HCT-8, SNU C2A, SNU C4, DLD-1, HCT116, and LOVO) and three MMR-proficient (NCI-H508, SW480, and HT29) colorectal cancer cell lines. Primers were designed to contain at least one exon–exon junction region and to amplify the coding repeat sequences (Table S2). Of the 20 frameshift mutations in the genomic DNA, mutation-derived transcripts were detected from ten alleles representing six genes (hMSH3, TAF1B, TGFBR2, ACVR2, MARCKS, and TCF-4), whereas ten alleles representing six genes (ABCF1, hMSH6, hRad50, PRKWNK1, RFC3, and SEC63) did not generate frameshift mutation-derived transcripts. No differences in expression of frameshift mutation-derived mRNA were observed between cell lines. Of the ten transcripts with frameshift mutations, five transcripts (representing three genes: hMSH3, TAF1B, and TGFBR2) had PTCs more than 50–55 nt upstream of the last exon–exon junction and were therefore expected to be degraded by NMD but instead escaped from NMD (NMD-escape). On the other hand, the five remaining transcripts (representing three genes: ACVR2, MARCKS, and TCF-4) had PTCs within 50–55 nt upstream of the last exon–exon junction and were therefore expected to be irrelevant to NMD (NMD-irrelevant). Accordingly, the 20 transcripts from 12 genes were classified as NMD-sensitive, NMD-escape, and NMD-irrelevant (Table 1).In order to confirm the effect of NMD on the NMD-sensitive and NMD-escape PTC-containing mRNAs, we used RT-PCR and a ribonuclease protection assay (RPA) to analyze the expression of the target gene mRNAs after treatment with puromycin, a translation inhibitor. In the five NMD-escape alleles that generated detectable frameshift mutation-derived mRNAs, no expression differences were found after puromycin treatment. In the ten NMD-sensitive alleles that produced no detectable frameshift mutation-derived mRNAs, mutant transcripts were detected after puromycin treatment (Figure S2). We analyzed the amount of two degraded NMD-sensitive transcripts, hRad50 and hMSH6, by RPA and found a total loss of mutant transcripts, as evidenced by a 2-fold increase in hRad50 and hMSH6 products after puromycin treatment. In contrast, there was no loss of TGFBR2 mutant mRNA, an NMD-escape transcript, because the amount of product was unchanged after puromycin treatment (Figure 1).thumbnail\\tDownload:PPT PowerPoint slidePNG larger imageTIFF original imageFigure 1. Measurement of Degraded Premature Termination Codon-Containing mRNAs by RPA AnalysisNo loss of TGFBR2 mutant transcripts was noted after puromycin treatment. In contrast, the amount of hRad50 and hMSH6 mRNA doubled after puromycin treatment. GAPDH was used for standardization. The asterisk indicates mutation status of each gene: (−1) denotes a 1-bp deletion in the cMNR, (w) denotes no mutation in the cMNR, (+1) denotes a 1-bp insertion in the cMNR, and (−2) denotes a 2-bp deletion in the cMNR.https://doi.org/10.1371/journal.pbio.0050109.g001Next, we evaluated the effect of down-regulating UPF1 or UPF2, which are key NMD factors, on the stability of the frameshift mutation-derived mRNAs, hRad50 and hMSH6, using specific small interfering RNA (siRNA). Upon the treatment of luciferase siRNA, expression of the mutation-derived hRad50 and hMSH6 mRNAs were not detected in the cell lines with hRad50 and hMSH6 mutations. In contrast, down-regulating UPF1 or UPF2 abundantly increased the frameshift mutation-derived mRNAs, as confirmed by RT-PCR, and sequence analysis. These findings indicate that frameshift mutation-derived mRNAs of hRad50 and hMSH6 are recognized and degraded by the NMD system (Figure S3).Truncated Proteins from NMD-Escape PTC-Containing mRNAs of hMSH3 Are Not Detected by Western BlottingIn order to determine if the truncated protein products from PTC-containing mRNAs can be detected, we first performed Western blotting analyses using antibodies directed against the N-terminus of hRad50, hMSH6, and hMSH3. Truncated proteins were not detected for the NMD-sensitive (hRad50 and hMSH6) genes, whereas wild-type proteins were detected in the cell lines containing the wild-type allele. These results support the previous finding that NMD-sensitive PTC-containing mRNAs are degraded by the NMD system. In the NMD-escape hMSH3 gene, we detected full-length hMSH3 proteins in cell lines with no mutations or with monoallelic mutations in these genes; however, we could not detect truncated hMSH3 proteins in any of the cell lines carrying frameshift mutations (Figure 2). The hMSH3 antibody detected the truncated proteins from the cell lines transfected with mutant hMSH3 cDNA, indicating that the antibodies used in our experiments specifically react with the N-terminal region of hMSH3 protein (Figure S4). Furthermore, we could not detect the truncated proteins of hMSH3 genes after treatment with the proteasome inhibitors MG132 or E64 (unpublished data), which excludes the possibility of rapid degradation of mutated proteins.thumbnail\\tDownload:PPT PowerPoint slidePNG larger imageTIFF original imageFigure 2. Western Blotting Analysis of NMD-Sensitive and NMD-Escape ProteinsTruncated proteins were not detected in NMD-sensitive (hRad50 and hMSH6) genes, whereas wild-type proteins were detected in the cell lines containing the wild-type allele. In the NMD-escape hMSH3 gene, wild-type hMSH3 proteins were detected in the cell lines containing the wild-type allele. However, no truncated proteins of hMSH3 were detected in the cell lines with frameshift mutation. GAPDH was used as a loading control. The single asterisk indicates mutation status of each gene: (−1) denotes a 1-bp deletion in the cMNR, (w) denotes no mutation in the cMNR, and (+1) denotes 1-bp insertion in the cMNR. The double asterisk indicates frameshift mutations of the other coding regions present in both alleles of DLD-1 [38]. The dagger indicates that the expected size of the truncated protein is indicated by the arrow.https://doi.org/10.1371/journal.pbio.0050109.g002Selective Translational Repression of Mutant TGFBR2 following Normal SplicingWe interpreted the failure to detect truncated protein from the NMD-escape PTC-containing mRNAs as follows: (1) truncated proteins were generated, but at an amount not sufficient for detection by Western blotting, (2) truncated proteins were generated, but then rapidly degraded, or (3) truncated proteins were not generated from the mutant mRNA.To rule out an insufficient amount of endogenous truncated proteins, we constructed expression plasmids with NMD-escape PTC-containing genomic DNA or cDNA of TGFBR2: (1) wild-type cDNA of TGFBR2 (K: TGFBR2(WT)-cDNA), (2) PTC-containing mutant cDNA of TGFBR2 without downstream exons and introns (L: TGFBR2(−1)-cDNA), (3) wild-type TGFBR2 genomic DNA (M: TGFBR2(WT)-splicing), (4) mutant TGFBR2 genomic DNA with a 1-bp deletion (N: TGFBR2(−1)-splicing), and (5) mutant TGFBR2 genomic DNA with a 1-bp deletion and a PTC artificially located in the last exon (O: TGFBR2(−1)-irrelevant) (Figure 3A). Among three NMD-escape PTC-containing mutated genes that we found in MSI-H tumors, we selected TGFBR2. TAF1B and hMSH3 were excluded because of their large size and number of exons, which result in the failure or inefficient transfection of genomic DNA.thumbnail\\tDownload:PPT PowerPoint slidePNG larger imageTIFF original imageFigure 3. Transfection Assay of the TGFBR2 Vector(A) Schematic diagram of construct K (TGFBR2 wild-type cDNA), construct L (TGFBR2(−1)-deleted cDNA), construct M (TGFBR2 wild-type genomic DNA), construct N (TGFBR2(−1)-deleted genomic DNA), and construct O (TGFBR2(−1)-deleted genomic DNA with a PTC artificially located in the last exon).(B) Analysis of TGFBR2 mRNA by Northern blotting. The abundant expression of TGFBR2 mRNAs from the transfected constructs is shown. β-Actin was used as a RNA loading control. Con. denotes the control vector pSecTag2B.(C) Western blotting using anti-FLAG. Protein expression of transfected TGFBR2 constructs. Cell lines transfected with the constructs K and M expressed wild-type TGFBR2. Cell lines transfected with the constructs L and O expressed truncated TGFBR2. However, cell lines transfected with construct N did not express truncated TGFBR2, indicating translational repression after normal splicing. Con. denotes the control vector pSecTag2B. Enhanced green fluorescent protein (EGFP) was used as a transfection control.(D) Sucrose gradient fractionation of cytoplasmic extracts from cells expressing TGFBR2(WT)-splicing mRNA.(E) Sucrose gradient fractionation of cytoplasmic extracts from cells expressing TGFBR2(−1)-splicing mRNA.(F) TGFBR2(WT)-splicing transfected cells were treated with puromycin prior to lysis and fractionation.(G) TGFBR2(−1)-splicing transfected cells were treated with puromycin prior to lysis and fractionation.In (D, E, F, and G), RNA extracted from each fraction was subjected to RT-PCR. Endogenous GAPDH mRNA was used as a control. TGFBR2(WT) mRNA was found in the polysome-containing fractions similar to endogenous GAPDH mRNA. However, a greater percentage of PTC-containing TGFBR2(−1) mRNA was found in the fractions containing ribosomal subunits and monosomes. The plots denote quantitative representation of TGFBR2 mRNA distribution in polysome gradients. Relative mRNA levels in each fraction were calculated as a percentage of the total. Results from two independently performed experiments did not vary, and the plot represents the average of the two independent experiments.https://doi.org/10.1371/journal.pbio.0050109.g003In all of the constructs described above, the nucleotide sequences encoding FLAG peptide was introduced immediately downstream of the initiation codon, which allows for detection of the encoded proteins by Western blotting. These vectors were designed to differentiate the effect of spliced wild-type mRNA, spliced mutant NMD-escape mRNA, and spliced mutant NMD-irrelevant mRNA in terms of truncated protein expression. We observed abundant expression of PTC-containing TGFBR2 mRNA in cell lines transfected with TGFBR2(−1)-cDNA, TGFBR2(−1)-splicing, and TGFBR2(−1)-irrelevant (Figure 3B). Cell lines transfected with TGFBR2(WT)-splicing, TGFBR2(−1)-splicing, and TGFBR2(−1)-irrelevant showed accurate splicing, and all normal and mutant mRNA products were confirmed by sequence analysis (unpublished data). A semi-quantitative RT-PCR analysis designed to detect exogenous TGFBR2 mRNA showed similar and abundant amounts of TGFBR2 mRNA expression in all of the cell lines transfected with the five different constructs (unpublished data). In this analysis of protein expression using the anti-FLAG antibody, we demonstrated the expression of wild-type TGFBR2 protein in cell lines transfected with TGFBR2(WT)-cDNA and TGFBR2(WT)-splicing. We also demonstrated the expression of truncated TGFBR2 protein in cell lines transfected with TGFBR2(−1)-cDNA and TGFBR2(−1)-irrelevant. Intriguingly, we could not detect any TGFBR2 protein in cell lines transfected with TGFBR2(−1)-splicing, indicating a selective translational repression of NMD-escape mutant mRNA (Figure 3C).PTC-Containing TGFBR2 mRNA Does Not Associate with PolysomesIn order to confirm that translational repression is responsible for the failure to detect truncated protein from PTC-containing TGFBR2 mRNA, we examined the mRNA distribution of TGFBR2(WT)-splicing and TGFBR2(−1)-splicing by polysome analysis.In the cell line with the TGFBR2(WT)-splicing vector, TGFBR2(WT)-splicing mRNA was found in the polysome-containing fractions similar to endogenous GAPDH mRNA (Figure 3D). However, in the cell line with the TGFBR2(−1)-splicing vector, a greater percentage of TGFBR2(−1)-splicing mRNA was found in the fractions that contained ribosomal subunits and monosomes, whereas endogenous GAPDH mRNA co-sedimented with polysomes (Figure 3E). Furthermore, upon the treatment of puromycin, a greater percentage of TGFBR2(WT)-splicing mRNA and endogenous GAPDH mRNA were shifted into ribosomal subunits and monosome-containing fractions (Figure 3F). In order to rule out the possibility that the weak polysome association of TGFBR2(−1)-splicing mRNA is due to its shorter open reading frame as compared to the TGFBR2(WT)-splicing mRNA, we repeated the same experiment using the TGFBR2(−1)-splicing vector with puromycin treatment. The results show that there is no significant difference in the cell line transfected with TGFBR2(−1)-splicing after puromycin treatment (Figure 3E and 3G). These results indicate that (1) the sedimentation of TGFBR2(WT)-splicing mRNA in heavy fractions was due to polysome association, and (2) the shift of TGFBR2(−1)-splicing mRNA into ribosomal subunits and monosome-containing fractions is due to translational repression, similar to TGFBR2(WT)-splicing mRNA and TGFBR2(−1)-splicing mRNA treated with puromycin (Figure 3E–3G). This novel mechanism, whereby PTC recognition itself triggers translational repression, is referred to as nonsense-mediated translational repression (NMTR).Factors Involved in Translational Repression of NMD-Escape Mutant mRNAWe demonstrated the selective translational repression of the NMD-escape mutant TGFBR2(−1)-splicing mRNA after normal splicing, and this repression was not found in the NMD-irrelevant mutant TGFBR2 mRNA, which lacks a downstream sequence of the termination codon. Therefore, we examined other possible factors influencing the expression of the truncated protein by: (1) changing the 3′ UTR length (the length between the PTC and poly(A) tail) to check the possible effect of 3′ UTR length on translational repression [16], (2) treating with a proteasome inhibitor (MG132) in the cell lines transfected with TGFBR2(−1)-splicing and TGFBR2(−1)-irrelevant to rule out that the truncated proteins are generated but rapidly degraded by the proteasome, and (3) down-regulating key NMD factors, UPF1 and UPF2, to evaluate whether NMD factors are involved in the translational repression of NMD-escape PTC-containing spliced TGFBR2 mutant mRNA.In order to change the 3′ UTR length, we constructed another TGFBR2(−1)-irrelevant with a full-length cDNA sequence spanning from the PTC to the 3′ end of TGFBR2 (P: TGFBR2(−1)-irrelevant-F) (Figure 4A). When the genomic DNAs of TGFBR2(−1)-splicing, TGFBR2(−1)-irrelevant, and TGFBR2(−1)-irrelevant-F were transfected, normal splicing and a large amount of mutant mRNAs were present in all three cell lines transfected with the different genomic DNAs (Figure 4C). However, no proteins were expressed in the cell lines transfected with TGFBR2(−1)-splicing, whereas a large amount of truncated proteins were expressed in the cell lines transfected with TGFBR2(−1)-irrelevant. In the cell lines transfected with TGFBR2(−1)-irrelevant-F, the amount of truncated proteins was reduced to about 35% of that of the cell lines transfected with TGFBR2(−1)-irrelevant. These findings indicate that the 3′ UTR length itself or specific cis-acting element(s) within the 3′ UTR seem to contribute to the translational inhibition of TGFBR2(−1) mRNA. However, more importantly, a splicing event downstream of the PTC may be involved in NMTR because truncated proteins are expressed in cells transfected with TGFBR2(−1)-irrelevant-F, but not in cells transfected with TGFBR2(−1)-splicing (Figure 4D).thumbnail\\tDownload:PPT PowerPoint slidePNG larger imageTIFF original imageFigure 4. Both 3′ UTR and Splicing Are Important for Translational Suppression of PTC-Containing TGFBR2 mRNA(A) Schematic diagram of constructs P (NMD-irrelevant-type TGFBR2(−1)-deleted construct containing a full coding sequence), O, and N.(B) Western blotting using anti-UPF1, anti-UPF2, or as a loading control, anti-GAPDH.(C) Analysis of TGFBR2 mRNA by Northern blotting. The abundant expression of TGFBR2 mRNAs from the transfected constructs is shown. β-Actin was used as a RNA loading control.(D) Protein expression of transfected TGFBR2 constructs. Truncated proteins are not expressed in the cell lines transfected with TGFBR2(−1)-splicing, whereas, truncated proteins are expressed by cell lines transfected with TGFBR2(−1)-irrelevant and TGFBR2(−1)-irrelevant-F. The amount of truncated protein from the TGFBR2(−1)-irrelevant-F is approximately 35% of TGFBR2(−1)-irrelevant, indicating that the 3′ UTR length or cis-element(s) affect translational efficiency. Truncated proteins are not detected from cell lines transfected with TGFBR2(−1)-splicing when treated with MG132, a proteasome inhibitor. Truncated proteins are not generated from cell lines transfected with TGFBR2(−1)-splicing when UPF1 and UPF2 are knocked down by siRNA, indicating that UPF1 and UPF2 do not have an important role in the translational repression of TGFBR2(−1)-splicing mRNA. Enhanced green fluorescent protein (EGFP) was used for transfection control. Con. denotes the control vector pSecTag2B. Luci denotes luciferase siRNA.https://doi.org/10.1371/journal.pbio.0050109.g004We excluded the possibility that mutated proteins are generated, but then rapidly degraded by the proteasome, because cell lines transfected with TGFBR2(−1)-splicing and treated with MG132, a proteasome inhibitor, demonstrated no truncated proteins. In contrast, the cell lines transfected with TGFBR2(−1)-irrelevant-F and treated with MG132 demonstrated similar amounts of truncated proteins compared to cell lines only transfected with TGFBR2(−1)-irrelevant-F (Figure 4D).Finally, we evaluated whether key NMD factors are involved in NMTR. We expected that the most significant difference between PTC-containing TGFBR2 mRNA and PTC-containing TGFBR2 mRNA, which lacks an intron downstream of the PTC, would be the presence of exon junction complexes (EJCs) behind the PTC. An EJC recruits the NMD factors, UPF1 and UPF2, which play a key role in mRNA quality control. We treated cells with UPF1 and UPF2 siRNA in order to elucidate whether these two NMD factors are involved in NMTR. The level of UPF1 was down-regulated to about 20% of normal, where normal is defined as the level in the presence of the nonspecific control, luciferase siRNA, whereas the level of UPF2 was down-regulated to about 10% of normal (Figure 4B). Treatment of any of the siRNAs failed to produce truncated proteins in the cell lines transfected with TGFBR2(−1)-splicing, indicating that at least these two NMD factors do not play an important role in the NMTR of NMD-escape mutant TGFBR2 mRNA (Figure 4D).NMD-Irrelevant Mutant mRNA of MARCKS Generates Truncated ProteinsWe then examined whether NMD-irrelevant PTC-containing mRNA can generate truncated protein. We selected one NMD-irrelevant mRNA, mutant MARCKS, and performed a transfection assay using wild-type MARCKS genomic DNA (MARCKS(WT)-splicing) and mutant MARCKS genomic DNA with a 2-bp deletion (MARCKS(−2)-splicing) (Figure 5A).thumbnail\\tDownload:PPT PowerPoint slidePNG larger imageTIFF original imageFigure 5. Truncated Protein Expression from Mutant NMD-Irrelevant MARCKS mRNA(A) Schematic diagram of construct Q (MARCKS wild-type genomic DNA) and construct R (MARCKS(−2) deleted genomic DNA).(B) Western blotting (WB) analysis of total protein (30 μg) isolated from transiently transfected cell lines with constructs Q or R. An antibody against FLAG was used as the primary antibody.(C) Western blotting (WB) analysis of total protein (30 μg) isolated from transiently transfected cell lines with the constructs Q or R was performed using an antibody against MARCKS. Enhanced green fluorescent protein (EGFP) was used as a control for transfection efficiency. Comparable results were obtained in at least two independent experiments. Asterisks (*) denote endogenous MARCKS reacted with anti-MARCKS antibody; Con. denotes the control vector pcDNA3.1(+). The dagger indicates uncharacterized protein that did not interfere with experimental interpretations.https://doi.org/10.1371/journal.pbio.0050109.g005We found expression of wild-type and truncated protein in cell lines transfected with MARCKS(WT)-splicing and MARCKS(−2)-splicing, respectively, by Western blotting with an anti-FLAG antibody (Figure 5B). To verify that these protein products were identical to MARCKS, we performed Western blotting with the anti-MARCKS antibody and confirmed the expression of wild-type and truncated proteins (Figure 5C). We also found that the truncated MARCKS protein is subject to active proteasome-mediated degradation; the amount of truncated MARCKS protein increased with time when cells were treated with MG132, a proteasome inhibitor (Figure 5B and 5C).DiscussionIn this study, we found that some PTC-containing mRNAs are not degraded by the NMD system, and their protein translations are repressed. We therefore suggest that PTC-containing mRNAs resulting from frameshift mutations can be classified into three groups: NMD-sensitive mRNAs, which are degraded by the NMD system; NMD-escape mRNAs, which are not degraded by the NMD system, but do experience repression of protein expression; and NMD-irrelevant mRNAs, which are not recognized by the NMD system, and generate truncated proteins. Our findings indicate that both NMD and NMTR, an additional surveillance mechanism for translational control, are involved in the recognition of PTC and suppression of truncated protein from PTC-containing genes that can be deleterious to cell function (Figure 6).thumbnail\\tDownload:PPT PowerPoint slidePNG larger imageTIFF original imageFigure 6. Schematic Model for the Functional Consequences of the Three Classes of PTC-Containing mRNAsNMD-sensitive mRNAs are degraded by the NMD system. NMD-escape mRNAs are not degraded by the NMD system, but experience repression of protein expression. NMD-irrelevant mRNAs are not recognized by the NMD system and generate truncated proteins.https://doi.org/10.1371/journal.pbio.0050109.g006NMD is a quality control-based surveillance mechanism that protects cells from the potentially dominant negative effects of truncated mutant proteins. The primary role of the NMD pathway is to eliminate nonsense transcripts that result from faulty transcription, alternative splicing, or somatic mutation [17,18]. This pathway selectively degrades mRNAs that prematurely terminate translation due to a frameshift or nonsense mutation. Although NMD is a quality control-based surveillance mechanism, avoidance of NMD by PTC-containing mRNAs has been reported for the mutated genes of many diseases. Moreover, about one third of the alternative transcripts in cells are expected to contain PTCs due to splicing errors and regulated unproductive splicing and translation (RUST). Some of these PTC-containing mRNAs belong to the NMD-escape variety [14,15]. If translated, these NMD-escape mRNAs could produce truncated proteins that may critically interfere with cell viability. Among the PTC-containing mRNAs, some PTCs, which are called NMD-irrelevant mRNAs, are located within 50–55 nt or downstream of the last exon–exon junction and are not detected by NMD. Proteins generated from these types of PTC-containing mRNAs and their causal relationship to specific diseases have been well documented [19–21]. However, there are no reports describing translated proteins from mutation-derived NMD-escape mRNAs, although many NMD-escape PTC-containing mRNAs have been reported [22–24].In this study, we demonstrated that NMD-escape TGFBR2 mRNA is subject to NMTR. Our transfection study of TGFBR2 constructs demonstrated that PTC-containing mRNAs from mutant TGFBR2 were abundant after transfection of mutant cDNA and mutant TGFBR2 genomic DNAs with a 1-bp deletion. However, truncated proteins were only detected in the cell lines transfected with mutant TGFBR2 cDNA, and no truncated proteins were detected in the cell lines transfected with mutant TGFBR2 genomic DNAs with a 1-bp deletion. In contrast, strong expression of TGFBR2 protein was observed in the cell lines transfected with wild-type TGFBR2 genomic DNA. Next, we confirmed using polysome analysis that the lack of truncated protein translated from PTC-containing TGFBR2 mRNA is due to translational repression, not instability of TGFBR2 mRNA.The major expected differences between the two PTC-containing TGFBR2 mutant mRNAs and mRNA from the cDNA of TGFBR2 or TGFBR2 genomic DNA are the deposition of the EJC and the possible recruitment of NMD factors to the mRNA during translation termination. We also confirmed the NMTR by demonstrating that mutant mRNA and truncated proteins were efficiently expressed in the cell line transfected with mutant TGFBR2 genomic DNA containing a PTC in the last exon without downstream introns (Figures 3C and 4D). We therefore suspected that EJC proteins and/or NMD factors might play an important role in the NMTR. It is well known that NMD recognizes PTC and downstream splicing events that deposit an EJC at an exon–exon junction. The EJC is composed of proteins involved in splicing and the subsequent steps of mRNA transport and translation. EIF4A3, RNPS1, Y14, and MAGOH are involved in EJC formation, and the EJC–mRNA complex is then exported to the cytoplasm together with nuclear cap binding proteins CBP80/20 and nuclear poly(A) binding protein 2 (PABP2) [25–28]. The mRNA then recruits UPF2 and undergoes a so-called “pioneer” round of translation during mRNA export. NMD occurs when translation terminates more than 50–55 nt upstream of the last exon–exon junction. Transient SURF formation at the termination codon, which is composed of Smg1, UPF1, and translation termination factors eRF1–eRF3, is thought to interact with the downstream EJC so as to trigger phosphorylation of UPF1 and thereby elicit NMD [26–30]. In this study, we demonstrated that key NMD factors, UPF1 and UPF2, did not play an important role in the NMTR, which is evidenced by the fact that treatment of UPF1 and UPF2 siRNA did not produce truncated proteins in cell lines transfected with TGFBR2(−1)-splicing, even though both siRNAs drastically down-regulate endogenous UPF1 and UPF2. Moreover, down-regulating Y14 or EIF4A3, which are EJC components, using siRNA failed to restore translational repression of TGFBR2(−1)-splicing (unpublished data). Our results indicate that NMTR is at work on the NMD-escape PTC-containing TGFBR2 mRNA by some unknown surveillance mechanism. The involvement of another messenger ribonucleoprotein particle (mRNP) complex in this translational repression is essentially unknown. Future studies should be focused on the role of translational repression of the various RNA binding proteins in the PTC-containing mRNP complex.Several recent reports have demonstrated the importance of termination codon context, especially 3′ UTR length, in PTC recognition [16]. Therefore, we tested two TGFBR2(−1)-irrelevant vectors with short and extended 3′ UTR lengths. If the PTC-containing, 3′ UTR-extended construct failed to produce truncated protein, then unlike NMD, splicing and EJCs may not be involved in the mechanism of NMTR. In this experiment, we demonstrated a 65% reduction of truncated protein in the cell lines transfected with TGFBR2(−1)-irrelevant vector with an extended 3′ UTR length. These findings indicate that 3′ UTR length is an important factor; however, other important factors are involved in NMTR. Because NMTR depends on 3′ UTR length or a putative cis-element residing in the 3′ UTR, it is, in part, reminiscent of EJC-independent NMD. A PTC within the penultimate exon of the β-globin or TPI gene elicits NMD depending on the position of PTC relative to the last EJC [12,31]. However, for a PTC within the penultimate exon that normally elicits NMD, deleting the last intron fails to eliminate NMD, indicating that the last exon has a so-called “failsafe” sequence that allows for PTC recognition and triggers NMD in the absence of a downstream EJC. Intriguingly, this element requires that splicing occur upstream of the PTC because the PTC-containing mRNA that is derived from an intronless TPI or β-globin gene is immune to NMD [12]. It remains to be clarified whether NMTR also requires a splicing event upstream of the PTC. Recently, another case for EJC-independent NMD has been reported in immunoglobulin-μ mRNA [16]. Similar to NMTR, EJC-independent NMD of this mRNA depends on the 3′ UTR length. However, the mode of PTC recognition looks quite different between NMTR and EJC-independent NMD of this mRNA, in the sense that NMTR does not require the NMD factors, UPF1 and UPF2, as shown in our study. Important questions of whether mRNAs targeted by EJC-independent NMD are subject to NMTR should be addressed.Marked degradation of PTC-containing mRNAs and decreased protein synthesis from PTC-containing mRNAs by the NMD system have been reported in yeast [32]. Another important mRNA surveillance mechanism, nonsense-mediated altered splicing (NAS), has been reported in mammalian cells [33]. NAS induces alternative splicing in the PTC-containing mRNA, thus avoiding the production of toxic mutant proteins. Although the exact mechanism of NAS had not been reported, UPF1 plays an important role in NAS [34]. Together with our findings that UPF1 did not play a significant role in NMTR, all of these findings indicate that (1) NMD, NAS, and NMTR play important roles in the inhibition of deleterious mutant protein production, and (2) unlike NMD and NAS, UPF1 and UPF2 do not a play key role in NMTR, suggesting novel factor(s) or pathways exist in the NMTR.In conclusion, we demonstrated three different molecular pathways of PTC-containing mRNAs. We propose a novel mechanism of gene expression regulation for PTC-containing mRNAs, in which the deleterious transcripts are regulated either by NMD or NMTR. Future studies of the NMD and NMTR control mechanism will enable us to better understand the reason for specific protein expression among the numerous mRNA isoforms, as well as the selective cellular control mechanism of protein expression. The mutational inactivation of transforming growth factor β receptor type II (TGFBR2) occurs in ∼30% of colon cancers and promotes the formation of colon cancer by inhibiting the tumor suppressor activity of the TGFB signaling pathway. TGFBR2 mutations occur in >90% of microsatellite unstable (MSI) colon cancers and affect a polyadenine tract in exon 3 of TGFBR2, called BAT-RII, which is vulnerable to mutation in the setting of DNA mismatch repair (MMR) system deficiency. In light of the vulnerable nature of the BAT-RII tract in the setting of MMR inactivation and the favorable effects of TGFBR2 inactivation in colon cancer, analysis of TGFBR2 inactivation provides an opportunity to assess the roles of genomic instability vs. clonal selection in cells acquiring TGFBR2 BAT-RII tract mutations in MSI colon cancer formation. The contribution of genomic instability and/or clonal evolution to the mutational inactivation of TGBFR2 in MSI colon cancers has not been studied in a systematic way that would allow a determination of the relative contribution of these two mechanisms in the formation of MSI colon cancer. It has not been demonstrated whether the BAT-RII tract mutations are strictly a consequence of the BAT-RII region being hypermutable in the setting of MMR deficiency or if the mutations are rather a consequence of clonal selection pressure against the TGFB receptor. Through the use of defined cell line systems, we show that both genomic instability and clonal selection of TGFB resistant cells contribute to the high frequency of TGFBR2 mutations in MSI colon cancer. © 2007 Wiley-Liss, Inc. INTRODUCTION  Transforming growth factor β (TGFB) is a multifunctional cytokine and has been implicated in the pathogenesis of many types of cancer including colon cancer. In normal colon epithelial cells, TGFB signaling can induce cell cycle arrest, apoptosis, and differentiation, which would be predicted to suppress neoplasm formation in the colon (Massague et al.,1985; Sporn and Roberts,1985; Massague,1990; Chakrabarty,1992; Huang and Chakrabarty,1994). Transformed epithelial cells are often resistant to TGFB induced growth inhibition and apoptosis (Wang et al.,1995a; Grady et al.,1999). The mechanism for this resistance is most commonly mutational inactivation of an essential component of the TGFB receptor, TGFBR2 (Roberts, 1990). TGFBR2 is mutated in >90% of colon cancers that demonstrate microsatellite instability (MSI) and in 15% of microsatellite stable (MSS) colon cancers (Parsons et al.,1995). Thus, mutational inactivation of TGFBR2 is a relatively common event in colon cancers. The mutations typically seen in MSI colon cancers, which are deficient in DNA mismatch repair (MMR) function, are in a mutation vulnerable polyadenine microsatellite repeat in exon 3 called BAT-RII whereas the mutations observed in MSS colon cancers are usually missense mutations in the kinase domain of the receptor (Markowitz et al.,1995; Grady et al.,1998).  The high frequency of TGFBR2 BAT-RII mutations in MSI colon cancers suggests that the mutation of TGFBR2 is biologically important to the formation of these cancers. However, it has been shown that many nonfunctional, noncoding microsatellite repeats show increased mutation frequency in MSI colon cancers, and it is clear that many of these are mutated strictly as a result of MMR deficiency with no apparent biological contribution to the formation of the colon cancer (Zhang et al.,2001; Woerner et al.,2003; Hienonen et al.,2005). The observation of MSI in loci that are bystander loci as well as in potential tumor suppressor genes has helped fuel a debate regarding the nature of the contributory roles of genomic instability vs. clonal selection to cancer formation (Cahill et al.,1999; Duesberg et al.,1999; Tomlinson and Bodmer,1999).  In light of the substantial evidence that TGFB signaling inactivation is pathogenic in colon cancer, we postulated that the assessment of TGFBR2 mutations in MSI cancers might provide insight into the relative contributions of genomic instability vs. clonal selection during the formation of MSI colon cancer. Indeed, the difficulty in discerning the relative roles of these two processes to cancer formation has led to two competing models for the role of genomic instability in tumorigenesis (Gagos and Irminger-Finger,2005). One model argues that genomic instability is one of the earliest events in cancer formation and is required for the acquisition of a sufficient number of mutations in oncogenes and tumor suppressor genes to permit a normal cell to become a cancer cell (Loeb,1991; Nowak et al.,2002). The second model argues that genomic instability is a consequence of the process of tumorigenesis, which is primarily driven by the clonal outgrowth of cells that progressively acquire biological abilities that allow them to transform (Sieber et al.,2003). Experimental data and the identification of cancer family syndromes that are secondary to germline mutations in DNA repair genes has provided support for a primary pathogenic role of genomic instability in cancer formation. However, there is also compelling evidence that genomic instability is a less significant factor in the tumorigenesis process of many cancers, and instead that the impact of mutations on the natural selection of clonal outgrowths is the major factor that drives the development of tumors (Moolgavkar and Luebeck,2003; Sieber et al.,2003).  In the context of this controversy, Markowitz et al. (1995) identified mutations in a polyadenine tract in exon 3 of TGFBR2, termed BAT-RII, in >90% of colon cancers with the form of genomic instability termed MSI. MSI results from the inactivation of the DNA MMR system, which corrects errors that arise in microsatellite repeats during DNA replication. The discovery of TGFBR2 mutations in the BAT-RII tract provided compelling evidence that genomic instability could directly drive the pathogenesis of colon cancer formation through the induction of TGFB resistance in the evolving tumor cells. On the basis of the high frequency of frame shift mutations in the BAT-RII tract in MSI colon cancers compared with MSS colon cancers, it would appear that genomic instability is the predominant mechanism inducing these mutations; however, other tumor types with MSI display a much lower frequency of tumors with TGFBR2 BAT-RII mutations, demonstrating that MSI alone is not sufficient to lead to the high incidence of TGFBR2 mutations observed in MSI colorectal cancers (Myeroff et al.,1995). Furthermore, there are microsatellite repeats, such as in the BAT26 and SEMG1 loci, that are commonly mutated in MSI colon cancers that provide no apparent selective advantage to the tumor cells raising the question of whether the high frequency of the TGFBR2 BAT-RII mutations is, in fact, strictly a consequence of being in a hypermutable locus in the context of MMR inactivation (Zhang et al.,2001; Hienonen et al.,2005).  In light of these unique features of the TGFBR2 BAT-RII locus, we have investigated the contribution of genomic instability and clonal selection to the acquisition of TGFBR2 BAT-RII mutations in a defined cancer cell line system. To investigate the role of genomic instability and natural selection in the acquisition of TGFBR2 mutations in MSI colon cancer, we reconstituted TGFBR2 into the MSI colon cancer cell line HCT116 and assessed the status of the BAT-RII tract in the reconstituted TGFBR2 and the responsiveness of the cell line to TGFB mediated growth inhibition. We used four different TGFBR2 transgenes: (1) wild-type TGFBR2, (2) kinase deleted TGFBR2 (DN2R), (3) kinase inactive TGFBR2 (TGFBR2 D522N) and (4) TGFBR2-2+8G, which has a modified BAT-RII tract that is a 6 basepair polyadenine tract rather than a 10 basepair polyadenine tract and which is less susceptible to DNA replication errors, in an attempt to isolate the contribution of genomic instability and natural selection to the mutational inactivation of TGFBR2. We have observed that MSI colon cancer cell lines reconstituted with TGFBR2 rapidly become resistant to TGFB mediated growth inhibition and acquire mutations in the BAT-RII tract of the transgenic TGFBR2. Furthermore, we provide evidence that both genomic instability and the growth advantage of TGFB resistance contribute to the inactivation of TGFBR2 in these cells. MATERIALS AND METHODS  Cell Lines and Culture Condition  The human colon carcinoma cell line HCT116 was a generous gift from Michael Brattain (Roswell Park Medical Institute, Buffalo, NY) and was cultured at 37°C and 5% CO2 in DMEM (Gibco/Invitrogen, Carlsbad, CA) supplemented with 10% fetal bovine serum (Hyclone, Logan, UT). The cell lines, RKO, Vaco703, Vaco400, Vaco410, and LoVo were provided by J.K.V. Willson (UT Southwestern Medical School, Dallas, TX). The growth conditions for these cell lines has been previously described (Markowitz et al.,1995; Grady et al.,1999). Retroviral-Mediated Gene Reconstitution  The wild-type TGFBR2, TGFBR2 D522N, and TGFBR2 2+8 transgenes used for these studies were derived from the pRC-CMV-R2 plasmid, which was a gift from Michael Brattain (Wang et al.,1995b). The kinase null TGFBR2 (DN2R) construct was kindly provided by Hal Moses (Vanderbilt University, Nashville, TN). The constructs cloned into the pLXIN vector all have an HA tag inserted in the 5′ aspect of the transgene. Inserts containing either wild type TGFBR2, TGFBR2 2+8, and or a kinase deficient TGFBR2 (TGFBR2 D522N) were cloned into the pLXIN retroviral expression vector (Clonetech, Mountain View, CA). Retrovirus was then produced using the Phoenix packaging cell line following standard techniques. Sub-confluent cultures of HCT116 cell lines were infected with viral supernatant for 4–6 hr in the presence of polybrene (4 μg/ml). G418 (1600 μg/ml) was used for selection 96–144 hr after infection and resistant colonies were pooled after 7–10 days. Several polyclonal populations from independent rounds of infections were generated and used in the experiments described below. Colony Formation Assay  HCT116 cells were plated at clonogenic density (100–500 cells/well) in 6-well plates and treated with TGFB (10 ng/ml) (R&D, Minneapolis, MN) 3 hr after plating. The cells lines were grown until discrete microscopic colonies appeared, usually 7–10 days, and then the number of colonies in each well was manually counted. Means and standard errors of the means were calculated from experiments performed in triplicate wells and repeated at least in three to four independent experiments. The studies using the neutralizing antibody 2G7 (concentration 20 μg/ml) were conducted as described above with addition of the antibody approximately 12 hr after plating the cells. The antibody was kept in the media for the entire duration of the experiment. The control wells were treated with vehicle only. TGFB Signaling Analysis  TGFB mediated signaling was measured using the p3TP-lux luciferase reporter (a generous gift of J. Massague, Memorial Sloan-Kettering Cancer Center, New York, NY) and pRL-CMV luciferase reporter (Promega, Madison,WI). Briefly, 1 × 106 cells were plated in each well of a 6-well plate and transfected the following day using 2.5 μg of p3TP-lux and 0.5 μg of pCMV-RL per well in the presence of 9 μg of FuGene 6 (Roche Molecular Biochemicals, Mannheim, Germany), according to the manufacturer's protocol. TGFB (2 ng/ml) was added 15 min after the transfection. The luciferase activity in the cell lysates was measured after 48 hr using the Dual Luciferase Reporter Assay System (Promega, Madison, WI) according to the manufacturer's protocol in a Veritas luminometer (Turner Systems, Sunnyvale, CA). DNA Extraction, Mutational Analysis, and PCR  DNA was extracted from the cells as previously described (Grady et al.,1999). The DNA from the cell lines was amplified using the primers and reaction conditions listed below. Of note, the primers used for the sequencing studies were designed to recognize unique sequence present in the transgene that is not in the endogenous TGFBR2. This was accomplished by using vector sequence in part of the primer sequence. The PCR amplicons were gel extracted with the Qiagen Gel purification kit following the manufacturer's protocol and then cloned into a TA cloning vector (pGEM-T Easy Vector System, Promega, Madison, WI). Sequencing was performed using M13 primers and was performed in the forward and reverse direction to confirm the mutations were not secondary to a sequencing error. The sequencing products were analyzed using a 3700 Sequencer (ABI, Foster City, CA). The PCR products were generated using the following primer sets (primer set 1): 5′ TGFBR2 amplicon: forward primer: 5′-CGA CCCCGCCTCGATCCT-3′ and reverse primer 5′-GCCGTCAGGAACTGGAGTATGTTC-3′ was used to amplify 5′TGFBR2 and a 3′ TGFBR2 amplicon (primer set 2): forward primer 5′-TGCCAACAACATCAACCACAA-3′ and reverse primer 5′-GCTTCGGCCAGTAACGTT-3′was used to amplify 3′ TGFBR2. The reaction mix consisted of PCR buffer (Roche), 1.5 mM MgCl2 (Applied Biosystems), 200 pM dNTPs (Applied Biosystems), 500 nM of each primer and 1 unit Taq polymerase (Roche). The thermocycler reaction conditions were: 94°C × 10′, 35 × (94°C × 1′, 58.4°C (primer set 1) or 56.8°C (primer set 2) × 1 min, 72°C × 3 min), 72C × 10′. RNA Extraction, RT-PCR, and Quantitative RT-PCR  RNA was extracted using the RNEasy kit (Qiagen, Valencia, CA) following the manufacturer's protocol. The extracted RNA was resuspended in DNAse, RNAse free water (Gibco) and subjected to DNAse treatment (RQ1Dnase, Promega, Madison WI) followed by another round of chloroform extraction and ethanol-based precipitation. Reverse transcription was performed using Superscript II (Gibco/BRL) with oligo-dT priming following the manufacturer's instructions. The sequence of the primers used in the RT-PCR for TGFBR2 were 5′-TCAAGCTTCCACAGCTG TG-3′ and 5′-CCTTCATGACACACTTGGG-3′ and for GAPDH were 5′-CTGGCATGGCCTTC CGTG-3′ and 5′-GAAATGAGCTTGACAAAG-3′. Quantitative RT- PCR for the TGFBR2 transgene was performed using primers specific for the HA-tagged TGFBR2 transgene. (Primer sequences: 5′-CTGTGGCCGCTGCAC-3′ and 5′-ACATCATACGGTGGGATCGTG-3′. SYBR Green PCR Master Mix (ABI) was used and thermocycler conditions were 50°C × 2′, 95°C × 10′. 95°C × 0.15′, 40 cycles and 60°C × 1′. GAPDH expression was used to normalize for cDNA loading and was assessed as previously published (Forrester et al.,2005) Quantitative RT-PCR was performed using an ABI 7700. Western Blot  Western blotting was performed using anti-HA polyclonal rabbit anti-sera (Y-11), anti-p15/CDKN2B (sc-612), anti-p21/CDKN1A (sc-397) (Santa Cruz Biotechnology), and mouse anti-ß-actin antibody (A3845, Sigma). The protein lysates from cell lines were prepared using ice cold NP40 lysis buffer with protease and phosphatase inhibitors. Protein concentration was determined using the BCA commercial protein assay kit following the manufacturer's recommendations (Pierce). The samples were then incubated at 95°C × 5′ in Laemli loading buffer and 25–30 μg protein per sample was subjected to SDS-PAGE through a 4–15% gradient gel. The samples were transferred to 0.45 μm PVDF membrane using a Criterion System (BioRad). The membrane was incubated with 5% non-fat dry milk dissolved in TBS-1% Tween for 1 hr at room temperature, incubated overnight at 4°C using the primary antibody (1:100 dilution for Y-11 and sc-612; 1:200 for sc-397; and 1:5000 for A3845), followed by washing three times, 10 min each, using TBS-Tween, and a 1 hr incubation at room temperature with a HRP-conjugated secondary antibody (sc-2005 and sc-2004, Santa Cruz). The ECL detection system (Amersham) or SuperSignal ELISA Pico Chemiluminescent Substrate (Pierce) was then used following the manufacturer's protocol to assess the protein expression. Statistical Analysis  Fisher's exact test was used to assess the differences in proportions of mutant TGFBR2 BAT-RII alleles between cell lines and passage numbers. RESULTS  Resistance to TGFB Growth Inhibition Occurs Rapidly in MSI Colon Cancer Cell Lines Reconstituted with TGFBR2  To assess the effect of restoration of the TGFB signaling pathway on colon cancer cell lines with mutant TGFBR2, we reconstituted TGFBR2 in a series of MSI and MSS colon cancer cell lines. We reconstituted TGFBR2 in the MSI cell lines HCT116, RKO, Vaco703, and LoVo, which represent the spectrum of MSI cell lines with regards to the underlying mechanism for MMR inactivation [e.g. MLH1 mutation (HCT116 and Vaco703), MLH1 methylation (RKO), and MSH2 mutation (LoVo)] (Veigl et al.,1998; Duval et al.,1999). We initially observed restoration of TGFB mediated growth inhibition and TGFB mediated 3TP-Lux luciferase reporter activity. However, in the MSI colon cancer cell lines, we consistently observed the acquisition of resistance to TGFB growth inhibition between passage 0 and passage 2–6 after transduction with TGFBR2, suggesting there was selective pressure against maintaining the function of the TGFBR2 transgene in the MSI colon cancer cell lines (Fig. 1A and Table 1). Assessment of TGFB mediated transcription using a more sensitive measure of TGFB responsiveness, the 3TP-Lux luciferase reporter, demonstrated similar reductions in TGFB responsiveness in these cell lines that were not as dramatic as the effects on TGFB mediated growth inhibition (Fig. 1B). To determine the effect of autocrine TGFB on the HCT116 cell lines and the acquisition of TGFB resistance, we plated the HCT116+TGFRB2, HCT116+TGFRB2 D522N, and HCT116+TGFB2 2+8 cell lines at clonogenic densities and assessed the formation of colonies while being treated with a TGFB neutralizing antibody (Wang et al.,1996; Wang et al.,2004). The cell lines were grown with media supplemented with a TGFB neutralizing antibody (2G7 20 μg/ml, a kind gift from Dr Carlos Arteaga, Vanderbilt University) to inhibit autocrine TGFB. The neutralizing antibody was added after the cells were plated at clonogenic density and was kept in the media for the entire time that the cells were grown. We observed significantly more colonies in the HCT116+TGFBR2 cells at early passage (P3) when treated with the neutralizing antibody (colony number: 52 ± 4 vs. 69 ± 4, P = 0.005) but did not observe an effect at later passages (passage 6 and passage 10) (Table 2). Thus, responsiveness to autocrine TGFB signaling is present at early passage in the wild-type TGFBR2 transfected cells but not in the later passages, presumably because of more complete loss of TGFBR2 expression with serial passage of the cells. No effect was observed in the cell lines reconstituted with TGFBR2 D522N or TGFBR2 2+8. The lack of effect in the HCT116+TGFBR2 2+8 cell line may reflect higher expression levels of the TGFBR2 2+8 construct compared with the wild-type TGFBR2, which resulted in failure of the 2G7 antibody to neutralize enough TGFB to inhibit activation of the transgenic TGFBR2.  Figure 1.  Figure 1.  Open in figure viewer (A) TGFB mediated growth inhibition in MSI colorectal cancer cell lines, HCT116, RKO, and Vaco703 after reconstitution with wild-type TGFBR2. Growth inhibition was measured by the effect of TGFB on a two-dimensional colony formation assay. The results are displayed as the percent growth inhibition caused by the addition of TGFB to the tissue culture media at the designated passage numbers. (B) TGFB mediated SMAD signaling as measured by the 3TP-Lux luciferase reporter assay in HCT116 and RKO after reconstitution with TGFBR2. The fold induction caused by the addition of TGFB to the cells at each passage number is shown. Table 1. Alterations in TGFBR2 and TGFB Responsiveness in HCT116 Cell Lines after TGFBR2 Transfection  \\tHCT116 + TGFBR2\\tHCT116 + TGFBR2 2 + 8\\tHCT116 + DN2R or TGFBR2 D522N Early passagea\\tLate passagea\\tEarly passage\\tLate passage\\tEarly passage\\tLate passage aEarly Passage, passage 2-3 after transfection; Late passage, passage 6 for HCT116 + TGFBR2 and HCT116 + DN2R and passage 10 for HCT116 + TGFBR2 2 + 8. TGFB responsiveness\\tResponsive\\tResistant\\tResponsive\\tResistant\\tResistant\\tResistant TGFBR2 expression\\t+++\\t+\\t+++\\t+\\t++\\t++ BAT-RII mutation frequency (%)\\t29\\t40\\t0\\t0\\t10\\t20 Table 2. Colony Formation with Treatment with TGFB Neutralizing Antibody (2G7) Cell line\\tAverage Colony Number\\tP value No 2G7\\t2G7 aEarly Passage, passage 3 after transfection; Late passage, passage 6 for HCT116 + TGFBR2 and HCT116 + TGFBR2 D522N and passage 10 for HCT116 + TGFBR2 2 + 8. HCT116 + TGFBR2; Early passagea\\t52 ± 4\\t69 ± 2\\t0.005 HCT116 + TGFBR2; Late passagea\\t68 ± 3\\t70 ± 2\\t0.256 HCT116 + TGFBR2 2 + 8; Early passage\\t70 ± 4\\t72 ± 4\\t0.47 HCT116 + TGFBR2 2 + 8; Late passage\\t60 ± 4\\t61 ± 4\\t0.69 HCT116 + TGFBR2 D522N; Early passage\\t51 ± 6\\t49 ± 2\\t0.54 HCT116 + TGFBR2 D522N; Late passage\\t62 ± 4\\t64 ± 4\\t0.69 The induction of cdk inhibitors that have been implicated in TGFB mediated growth inhibition is presumably one of the anti-tumor mechanisms through which TGFBR2 exerts its effects (Alexandrow and Moses,1995). Cells with intact TGFBR2 would be predicted to show induction of these cdk inhibitors. Consequently, we assessed the expression of CDKN1A and CDKN2B in early and late passage cell lines. A modest reduction in CDKN2B and CDKN1A protein expression is found in the late passage line transfected with TGFBR2 but not TGFBR2 2+8 demonstrating that the late passage HCT116 reconstituted with TGFBR2 shows evidence of loss of TGFB mediated regulation of CDKN2B and CDKN1A as well as loss of TGFB mediated growth inhibition. The lack of difference between early and late passage HCT116 + TGFBR2 2+8 may reflect that these cells retain a low level of TGFB responsiveness that can be detected by Western blotting but that cannot be detected by the colony formation assay or that the growth inhibitory effect in these cells is mediated through mechanisms that are independent of CDKN1A and CDKN2B. Interestingly, CDKN1A expression is increased and CDKN2B expression is slightly decreased in the late passage HCT116 cell line reconstituted with TGFBR2 D522N. The reasons for these events are unclear but possible explanations include indirect effects related to the dominant negative function of the mutant TGFBR2 D522N protein or to indirect effects from other regulators of CDKN1A and CDKN2B (Fig. 2).  Figure 2.  Figure 2.  Open in figure viewer Expression of CDKN2B and CDKN1A assessed by immunoblotting in HCT116 cell lines transfected with TGFBR2, TGFBR2 2+8, or an inactive mutant of TGFBR2 (TGFBR2 D522N). A modest reduction in CDKN2B expression is apparent in the late passage cell line transfected with the TGFBR2 transgene, but CDKN2B expression is not substantially different between the early and late passage lines transfected with the TGFBR2 2+8 or TGFBR2 D522N constructs. The expression of CDKN1A is modestly reduced in the late passage HCT116+TGFBR2 cell line but not in the HCT116+ TGFBR2 2+8 cell line. Expression of CDKN1A is increased in the late passage HCT116 cell line reconstituted with TGFBR2 D552N. Densitometry of the bands is shown adjacent to the image of the immunoblot. This loss of TGB responsiveness was accompanied by a 10-fold decrease in mRNA expression of the TGFBR2 transgene, which may reflect effects on transcription of the transgenes or degradation of the transgene mRNA (Fig. 3A). Assessment of the expression of the TGFBR2 transgene with flow cytometry also demonstrated a substantial decrease in expression of the TGFBR2 and TGFBR2 2+8 transgenes in the late passage HCT116 lines that can be appreciated by a reduction in the percentage of cells showing high expression levels of the transgenic TGFBR2 (Fig. 3B). Of interest and in contrast to the MSI colon cancer cell lines, two MSS colon cancer cell lines that carry TGFBR2 mutations, Vaco400 and Vaco410, that were reconstituted with TGFBR2 have shown no loss of TGFB responsiveness after >50 passages in culture (data not shown) (Grady et al.,1999).  Figure 3.  Figure 3.  Open in figure viewer (A) Quantitative RT-PCR of mRNA expression of the TGFBR2 transgene in the HCT116 cell line at early passage (passage 2, P2) compared with late passage (Passage 15, P15). The studies were performed in triplicate and demonstrate a substantial decrease in TGFBR2 transgene expression. Specific assessment of the TGFBR2 transgene was conducted through the use of PCR primers that are specific to the sequence encoding the HA tag in the transgene. (B) Expression of the HA-tagged TGFBR2 transgenes in the HCT116 cell lines at early and late passage assessed by flow cytometry. Flow cytometry of the HA-tagged HCT116 cell lines reveals no significant change in expression between early and late passage in an HCT116 line transfected with an inactive mutant of TGFBR2 (TGFBR2 D522N), and a decrease in expression in the late passage HCT116 cell lines transfected with either TGFBR2 or TGFBR2 2+8. Early passage is passage 2 for all the cell lines and late passage is passage 17 for the HCT116 + TGFBR2 cell line and passage 19 for the HCT116 + TGFBR2 D522N and HCT116 + TGFBR2 2+8 cell lines. The percentage of cells expressing the HA-tagged TGFBR2 at levels clearly above the nontransfected control are indicated in each panel. TGFB Resistance Is the Consequence of Mutations in the TGFBR2 BAT-RII Tract of the Reconstituted TGFBR2 Transgene  In light of the loss of TGFB responsiveness in the MSI colon cancer cell lines, we assessed the mutation status of the TGFBR2 transgene in the reconstituted cell lines. We used a retroviral vector to reconstitute TGFBR2 in order to reconstitute each transduced cell with approximately one transgene. Our assay was developed to only detect the TGFBR2 transgene. Thus, the endogenous TGFBR2, which is homozygous mutant in the BAT-RII tract, does not confound our results. This assay design allows an assessment of the mutation status of the transgene allele populations in each of the reconstituted cell lines but does not allow us to determine the allele status on a cell-by-cell basis. However, because we used an MOI of 0.01, we believe that the vast majority of the transduced cells carried only one integrant/cell. We observed that 29% (N = 9/30 clones) of the reconstituted HCT116 cell line had acquired TGFBR2 BAT-RII tract mutations in the TGFBR2 transgene by passage 1 after transfection and that this percentage increased to 40% (N = 12/30) by passage 22 (P = 0.59, Fisher's exact test) (Table 1). To determine whether the TGFBR2 BAT-RII mutations were a consequence of the MSI background of the cell line vs. selective pressure against restoration of TGFB signaling, we reconstituted the HCT116 cell line with a modified version of TGFBR2 that lacks the intracellular kinase domain (HCT116/DN2R). We observed 10% (N = 2/20 clones) of the HCT116/DN2R had mutant TGFBR2 at passage 1 and that this percentage increased to 20% (N = 4/20 clones) TGFBR2 mutants at passage 19 after transduction with the DN2R transgene (P = 0.66, Fisher's exact test). Comparison of the proportion of mutant TGFBR2 BAT-RII alleles between the HCT116+TGFBR2 vs. HCT116+DN2R at early and late passages revealed no statistically significant differences between the groups P = 0.19 and P = 0.22, Fisher's exact test). These results suggest that the BAT-RII tract is vulnerable to mutation even in a setting in which no biological consequence would be presumed to result from the BAT-RII tract mutation. We next assessed the effects of clonal selection on TGFBR2 inactivation vs. mutational vulnerability of the 10 basepair polyadenine tract itself in the setting of MMR deficiency on the acquisition of mutations in the BAT-RII tract. We modified the TGFBR2 transgene so that the BAT-RII tract sequence was AGAAAAAGAA. This modified TGFBR2 transgene, termed TGFBR2 2+8, does not change the amino acid coding sequence of TGFBR2. Analysis of HCT116 reconstituted with TGFBR2 2+8 revealed no mutations in the BAT-RII tract at passages 1 and 16 after transfection (10 clones analyzed at the stated passage numbers). The difference between the proportion of mutant TGFBR2 BAT-RII alleles was statistically significant for the late passage HCT116+TGFBR2 vs. HCT116+TGFBR2 2+8 lines (P = 0.02, Fisher's exact test), but not for the HCT116+TGFBR2 2+8 vs. HCT116+DN2R (P = 0.27, Fisher's exact test). TGFB Resistance Occurs Regardless of Absence of Hyper-Mutable 10 Basepair Polyadenine BAT-RII Tract  To further assess the role of MSI vs. release from TGFB mediated growth inhibition on the acquisition of and selection for HCT116 cells that had inactivated TGFBR2 via BAT-RII mutations, we assessed the HCT116 cell lines reconstituted with the TGFBR2 2+8 construct, which carries the modified TGFBR2 BATRII tract (AGAAAAAGAA), for TGFB responsiveness at early and late passage. We observed that the HCT116 cell line reconstituted with TGFBR2 2+8 retained sensitivity to TGFB mediated growth inhibition consistently to passage 10 in comparison with HCT116 reconstituted with wild-type TGFBR2, in which resistance to TGFB growth inhibition occurred at passage 2–3 on average. These experiments were performed with five independently derived pools of clones with similar results for each pool of clones. Notably, despite the delay in the development of TGFB resistance, the HCT116 +TGFBR2 2+8 cells always eventually became resistant to TGFB mediated growth inhibition by passage 15. Similar results were observed in the MSI colon cancer cell lines RKO and Vaco703 (Fig. 4). To assess for the mechanism mediating the TGFB resistance in the HCT116+TGFBR2 2+8 cell line, TGFBR2 transgene expression was measured. TGFBR2 expression is reduced in late passage HCT116+TGFBR2 2+8, as it is in HCT116+TGFBR2, suggesting that reduction in TGFBR2 expression is the mechanism for the reduction in TGFB responsiveness in the HCT116 cells transfected with the TGFBR2 2+8 transgene (Fig. 5A). To determine if decreased TGFBR2 expression is the cause of TGFB resistance in the HCT116+TGFBR2 2+8 cell line, TGFBR2 was reconstituted in the late passage cell lines. Transduction of these late passage cell lines with wild-type TGFBR2 restored TGFB mediated transcription demonstrating that TGFBR2 inactivation is the event leading to TGFB resistance in the HCT116 TGFBR2 2+8 cells and that there is not an inactivating event in another element of the TGFB signaling pathway causing the TGFB resistance. The magnitude of the effect of TGFBR2 reconstitution in the HCT116 +TGFBR2 2+8 cells is similar to that seen in the HCT116+TGFBR2 cells and HCT116+TGFBR2 D522N cells.  Figure 4.  Figure 4.  Open in figure viewer TGFB mediated growth inhibition in MSI colorectal cancer cell lines, HCT116, RKO, and Vaco703 after reconstitution with the mutation resistant TGFBR2 2+8 transgene. Growth inhibition was measured by the effect of TGFB on a two-dimensional colony formation assay. The results are displayed as the percent growth inhibition caused by the addition of TGFB to the tissue culture media at the designated passage numbers. Figure 5.  Figure 5.  Open in figure viewer (A) Quantitative RT-PCR of mRNA expression of the TGFBR2 transgene in the HCT116 cell line at early passage (passage 3, P3) compared to late passage (passage 20, P20) after reconstitution with the TGFBR2 2+8 trangene. The studies were performed in triplicate and demonstrate a substantial decrease in TGFBR2 transgene expression in the late passage cell lines. Specific assessment of the TGFBR2 transgene was conducted through the use of PCR primers that hybridize to the sequence encoding the HA tag in the transgene. (B)Restoration of TGFB mediated SMAD signaling by reconstitution of TGFBR2 in late passage HCT116+TGFBR2, and HCT116+TGFBR2 2+8. The reconstituted HCT116 cell lines were assessed for TGFB responsiveness using the 3TP-Lux luciferase reporter. The HCT116 cell lines transiently reconstituted with TGFBR2 all demonstrate substantial increases in TGFB mediated 3TP-Lux luciferase activity (right side of graph). No effect is seen with reconstitution with TGFBR2 D522N (left side). DISCUSSION  The discovery of MSI and chromosome instability in colorectal cancer provided evidence that genomic instability occurs in most colon cancers. However, it was the discovery of germline mutations in the MMR genes as the cause of the colorectal cancer family syndrome Hereditary Nonpolyposis Colorectal Cancer syndrome that suggested that the genomic instability could play a causal role in the formation of these cancers (Bronner et al.,1994; Liu et al.,1994). Furthermore, the identification that TGFBR2 is a common mutation target in the setting of MSI colorectal cancer demonstrates that genomic instability can result in the mutational inactivation of genes that are biologically important in the pathogenesis of cancer (Markowitz et al.,1995). TGFBR2 BAT-RII mutations can be found in approximately 90% of MSI colorectal cancers and have been shown to impair TGFB mediated growth inhibition (Parsons et al.,1995; Grady et al.,1999). However, it has also been clearly demonstrated that microsatellite repeats can undergo mutation in the setting of MSI regardless of any apparent biological advantage to the mutations (Zhang et al.,2001; Hienonen et al.,2005). This observation has made it unclear whether TGFBR2 BAT-RII mutations, or mutations in other coding region microsatellite repeats, occur simply because they are particularly susceptible to mutation in the setting of MMR deficiency or because they provide a growth advantage with subsequent clonal selection (Zhang et al.,2001; Hienonen et al.,2005). Thus, we postulated that assessment of TGFBR2 inactivation would permit an assessment of the roles of genomic instability and clonal selection in colorectal cancer formation.  Our results provide additional insight into the ongoing controversy related to the relative significance of genomic instability vs. clonal selection and expansion in the pathogenesis of cancer (Rajagopalan et al.,2003; Sieber et al.,2003). In elegant studies of Barretts esophagus, which is a pre-neoplastic tissue alteration of the esophagus, Maley et al. (2004) have demonstrated that both clonal expansion and genomic instability contribute to the progression of Barretts epithelium to esophageal adenocarcinoma (Maley et al.,2004). We have now conducted a series of studies in colorectal cancer cell lines that have been reconstituted with TGFBR2 to define the roles of genomic instability and clonal evolution in the acquisition of cancer promoting mutations in colon cancer. We observed that MSI colon cancer cell lines reconstituted with TGFBR2 rapidly acquire resistance to TGFB mediated growth inhibition, which demonstrates the substantial biological negative selection pressure against TGFB signaling in this class of colon cancer. Wang et al. (1995b,2004) have also demonstrated that there is selective pressure against the expression of TGFBR2 in HCT116. These investigators were able to generate clones of HCT116 transfected with TGFBR2, but found that the receptors were expressed at low levels and were saturated by autocrine TGFB. Indeed, the acquisition of TGFB resistance in our studies is particularly notable in light of the fact that the transgene has an IRES driven neomycin resistance gene, which should have precluded the ability of the cells to acquire resistance to TGFB signaling by repressing the transcription of the TGFBR2 transgene. Consistent with the design of the TGFBR2 transgene, the TGFB resistance observed in the MSI colon cancer cell line was accompanied by the presence of inactivating mutations in the BAT-RII polyadenine tract of the transgenic TGFBR2. Further studies with an inactivated TGFBR2 transgene and with a “mutation resistant” version of TGFB2 demonstrated that both genomic instability and clonal selection pressure for the loss of TGFB signaling drive the loss of TGFBR2 that results from mutations in the TGFBR2 BAT-RII tract.  Thus, reconstitution of TGFBR2 in MSI colon cancer cell lines has provided an opportunity to study the contribution of clonal selection and genomic instability in the inactivation of this particular gene because of the potent selection pressure against TGFB mediated growth inhibition in MSI colorectal cancer cell lines. For undetermined reasons these cell lines more rapidly inactivate the reconstituted TGFB signaling pathway compared with MSS cell lines. The TGFB signaling pathway appears to be a target for mutational inactivation in both MSS and MSI tumors. Indeed, studies of primary tumors demonstrate the presence of TGFBR2, SMAD2, and SMAD4 mutations and deletions in MSS colorectal cancers providing evidence that inactivation of the TGFB signaling pathway favors tumor formation in MSS tumors as well as in MSI tumors (Macias-Silva et al.,1996; Takagi et al.,1996; Grady et al.,1998). A possible explanation for the difference we have observed between MSS and MSI cell lines is that the selection pressure for TGFBR2 inactivation may involve mechanisms that are germane to in vitro growth in the MSI cancer cell lines, but that are less important in the MSS cancer cell lines. An alternative explanation is that the increased hypermutability of MSI cell lines results in a more rapid loss of TGFB responsiveness compared with the MSS cell lines and that longer time in culture would eventually reveal that some clones of MSS cell lines also will become TGFB resistant by mutating the TGFBR2 transgene.  The potent selection pressure against TGFB mediated growth inhibition permitted the measurement of the rate of loss of TGFB responsiveness in these cell lines, in relatively short time frames (<20 passages of the cell lines). Our studies with the HCT116, Vaco703, LoVo, and RKO cell lines demonstrate that inactivation of TGFBR2 is the preferential mechanism through which the TGFB signaling pathway is inhibited in MSI colorectal cancer. Indeed, the ability of TGFBR2 reconstitution in these cells to restore TGFB mediated SMAD signaling provides clear evidence there were no post-receptor alterations that were impairing the TGFB pathway in these cells. The studies of the MSI colon cancer cell lines reconstituted with the TGFBR2 2+8 transgene provided evidence that clonal selection pressure plays a significant role in the inactivation of TGFBR2 in MSI colorectal cancer and that the inactivation of the receptor is not strictly a consequence of the susceptibility of the TGFBR2 BAT-RII tract to mutate in the setting of MSI.  Furthermore, we observed decreased mRNA expression of the TGFBR2 transgenes in the HCT116 lines transfected with TGFBR2 and TGFBR2 2+8. Previous studies have suggested the decreased mRNA levels observed in colorectal cancers with TGFBR2 BAT-RII mutations are secondary to nonsense mediated decay (Markowitz et al.,1995). With regards to the TGFBR2 2+8 construct, our experimental design does not permit us to determine whether the repression of TGFBR2 in the cell lines reconstituted with the mutation resistant TGFBR2 construct is reflective of a process that is occurring in primary tumors because of the use of a retrovirus expression vector for reconstitution of the TGFBR2 transgene. As we did not reconstitute TGFBR2 through the use of homologous recombination with an expression vector that was driven off of the endogenous promoter, we cannot address whether the decreased mRNA levels of TGFBR2 2+8 reflect a process that may affect the endogenous TGFBR2 in primary colorectal cancer. Nonetheless, our results do clearly show that there is biological pressure to inactivate TGFB responsiveness in these cells, which we demonstrated was through repressing the expression of the TGFBR2 transgene. Restoration of TGFB responsiveness with reconstitution of TGFBR2 in the TGFB resistant late passage lines demonstrates that the loss of TGFB sensitivity is through inactivation of TGFBR2 and not through a different mechanism, such as mutational inactivation of SMAD4, etc.  With regards to genomic instability, the evaluation of the TGFBR2 BAT-RII mutation frequency in early and late passage HCT116 revealed that the BAT-RII tract is susceptible to mutation even in the setting of an inactive receptor (HCT116+DN2R). The frequency of mutations in the DN2R transgene was approximately half that seen in wild-type TGFBR2 suggesting that the BAT-RII locus is susceptible to mutation in part because it is particularly prone to DNA replication errors. These errors appear to accumulate with serial passage number although they are more common in the wild-type TGFBR2 transgene, which is consistent with clonal selection pressure affecting the mutation frequency. In fact, it appears that approximately half of the mutations can be accounted for by clonal selection pressure.  In summary, we have used a defined cell line system to investigate the cooperation between clonal selection pressure and genomic instability in mediating mutational inactivation of TGFBR2 in MSI colorectal cancer. We have demonstrated that TGFB signaling restoration is rapidly selected against in a panel of MSI colon cancer cell lines and that this inactivation is a consequence of both loss of MMR proficiency and selection against the growth inhibitory effects of TGFB signaling. These data are consistent with that of Maley et al. (2004) who demonstrated that clonal expansion and genomic instability cooperate in the progression of Barretts esophagus to esophageal adenocarcinoma (Hornick et al.,2005). It is of note that our studies do not address the role of genomic instability in the initiation of cancer but they do demonstrate the plausibility that genomic instability arising in early neoplastic lesions could contribute to the pathogenesis of colorectal cancer through the induction of mutations in susceptible genes that then promote the clonal evolution of cells that have acquired the mutations.  We previously demonstrated that mutational inactivation of transforming growth factor b type II receptors (RIIs) is very common among the 13% of human colon cancers with microsatellite instability. These mutations principally cluster in the BAT-RII polyadenine sequence repeat. Among microsatellite stable (MSS) colon cancers, we now find that nonBAT-RII point mutations inactivate RII in another 15% of cases, thus doubling the known number of colon cancers in which RII mutations are pathogenetic. Functional analysis confirms that these mutations inactivate RII signaling. Moreover, another 55% of MSS colon cancers demonstrate a transforming growth factor b signaling blockade distal to RII. The transforming growth factor b pathway and RII in particular are major targets for inactivation in MSS colon cancers as well as in colon cancers with microsatellite instability. Introduction TGF-b4 inhibits the growth of epithelial cells in general (1, 2) and can inhibit growth and/or induce apoptosis in nontransformed colon epithelial cells (3, 4). TGF-b signaling is transduced by a heteromeric receptor complex composed of type I and type II components, both of which are serine/threonine-directed kinases (5). A role for RII as a human colon cancer tumor suppressor gene was demonstrated by the discovery of inactivating RII mutations in colorectal cancers that show MSI due to defects in DNA mismatch repair (6–8). These cancers with MSI account for 13% of all colon cancers (6, 7, 9). Furthermore, the restoration of wild-type RII in cell lines from colon cancers with MSI abolishes their tumorigenicity in athymic mice (10). RII mutations in colon cancers with MSI usually result in frameshifts clustered in a naturally occurring 10-bp microsatellite-like polyadenine tract in the 59 coding half of the gene (BAT-RII; Refs. 7 and 8). In a few colon cancers with MSI, inactivation of one of the RII alleles occurs via non-BAT-RII mutations that alter the RII kinase domain (7, 9, 11), demonstrating an underlying selective advantage for RII inactivation, irrespective of whether this occurs via BAT-RII or non-BAT-RII mutational events. We hypothesized that the RII mutation should function similarly as a tumor suppressor gene in MSS colon cancers. Accordingly, we examined the RII sequence in 19 MSS colorectal cancer cell lines. These Vaco cell lines have been matched to antecedent tumor and normal tissues and have been extensively characterized as a model for human colon cancer (7, 12–14). Three of the 19 cell lines were shown to express mutant-only RII transcripts. Functional analysis showed that the RII mutations inactivated RII signaling in each case. Moreover, an additional 11 of these cell lines demonstrated a loss of TGF-b responsiveness, apparently through post-RII signaling defects. Therefore, TGF-b signaling in general and RII in particular are targets for inactivation in MSS colon cancers as well as in colon cancers with MSI. Materials and Methods Cell Lines and Primary Tumors. The establishment of the panel of the Vaco colon cancer cell lines has been described previously (7, 12). The Vaco8-2 cell line was established from a stage IV cecal colon adenocarcinoma from a 56-year-old male. Vaco400 was established from a liver metastasis from a 54-year-old man. Vaco410 was derived from a stage IV colon adenocarcinoma metastasis in the right lobe of the liver of a 35-year-old woman. These cell lines were maintained as described previously (12). Genomic DNA from the original paraffin-embedded formalin-fixed tumor blocks was extracted as described previously (15). DNA from matched normal tissue for each colon cancer was obtained when available. TGF-b Growth Inhibition Assay. The colon cancer cell lines were plated at clonogenic density (100–500 cells/well in 24-well plates for adherent cell lines and 1000–5000 cells/well in 6-well plates for collagen-dependent adherent cell lines) and treated with TGF-b1 (10 ng/ml) 3 h after plating. The cell lines were grown until discrete microscopic colonies appeared (usually for 7–10 days), and then the number of colonies in each well was manually counted. Mean and SEs of the means were calculated from experiments performed in triplicate wells and repeated in at least nine independent determinations. RT-PCR Amplification and Sequencing of RII. RNA from the cell lines was prepared by extraction with guanidine isothiocyanate (7). RT-PCR and cloning of the reaction products and manual sequencing of the cloned DNA were performed as described previously (7), or the reaction products were sequenced by automated sequencing using a ABI 377 DNA Sequencer. For amplification of RII from genomic DNA, sense primer 1871 (59-GGTGTGTGAGACGTTGACTGAGTG-39) was paired with antisense primer 2001 (59- AATCTTCTCCTCCGAGCAGCTC-39) for 30 cycles of 95°C for 30 s, 58°C for 1 min, and 70°C for 1 min. All RII mutations were confirmed to be present in at least two independently amplified PCR reactions compared with the reference sequence (GenBank accession number M85079). TGF-b Signaling Analysis. The cell lines were transiently transfected following the manufacturers’ protocols using Lipofectin (Life Technologies, Inc., Gaithersburg, MD), Superfect (Qiagen), or FuGENE (Boehringer Mannheim) with a TGF-b-responsive firefly luciferase reporter plasmid (p3TP-Lux; Ref. 16) and an internal control reporter plasmid containing cDNA encoding R. reniformis luciferase under a thymidine kinase promoter (pRL-TK) or a Received 11/6/98; accepted 12/8/98. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked advertisement in accordance with 18 U.S.C. Section 1734 solely to indicate this fact. 1 Supported by NIH Grants RO1 CA67409 and RO1 CA 72160 (to S. M.), by an Advanced Research Training Award from the American Digestive Health Foundation and NIH Grant KO8 CA77676-01 (to W. M. G.), and by NIH Grants P30 CA43703 and T32 CA 59366 (to Case Western Reserve University). S. M. is an investigator in the Howard Hughes Medical Institute. 2 These authors contributed equally to this study. 3 To whom requests for reprints should be addressed, at UCRC #2, Room 200, Ireland Cancer Center, 11001 Cedar Road, Cleveland, OH 44106. 4 The abbreviations used are: TGF-b, transforming growth factor b; RII, TGF-b receptor type II; MSI, microsatellite instability; MSS, microsatellite stable; RT-PCR, reverse transcription-PCR. 320 Downloaded from cancerres.aacrjournals.org on May 28, 2017. © 1999 American Association for Cancer Research.  cytomegalovirus promoter (pRL-CMV; Promega, Madison, WI). The p3TPLux plasmid was kindly provided by Dr. Joan Massague´ (Memorial SloanKettering Cancer Center and Howard Hughes Medical Center, New York, NY). After transfection, the cell lines were exposed to TGF-b1 (10 ng/ml) for 72 h and then assessed for reporter activity. The luciferase activity was measured using the Dual-Luciferase Reporter Assay System following the protocol included with this kit (Promega). The samples were assayed on a Turner Dual Injector Luminometer (Promega) or a MLX Multiplate Luminometer (Dynex Technologies). Retroviral Transduction. The cell lines were infected with the MFG-RII or MFG-CAT retroviruses (17). Cells were plated in 6-well plates at a density of 200,000 or 400,000 cells/well; on the following day, they were incubated in MFG-RII or MFG-CAT viral supernatant produced by a virus producer cell line and polybrene (4 mg/ml) for 4 h. The cells then were rinsed three times in PBS. G418 (600 mg/ml) was added 2 days after the viral infection. TGF-b1 (10 ng/ml) was added to designated subsets of the infected cell lines 3 h after infection to assess for reconstitution of TGF-b-induced growth inhibition. The growth inhibition assays were performed as described above. All of these experiments were performed in triplicate and repeated at least three times. Results and Discussion MSS Colon Cancers Are Commonly TGF-b Resistant. We initially determined whether TGF-b growth-inhibitory responses were intact in 19 colon cancer cell lines that had been previously determined to be MSS (7). TGF-b growth inhibition was assayed by determining the ability of TGF-b1 to inhibit the colony formation of cells plated at clonogenic density. Cell lines in which TGF-b1 (10 ng/ml) reduced colony formation at 7–10 days after plating by .25% were considered to be responsive to TGF-b-mediated growth inhibition. Five cell lines (26%) were determined to demonstrate sensitivity to TGF-b-mediated growth inhibition, with TGF-b-mediated suppression of colony formation ranging from 42–79%. However, 74% (14 of 19) of the MSS cell lines were found to be resistant to the growthinhibitory effects of TGF-b. RII Mutations Detected in MSS Colon Cancers. Ninety percent of colon carcinoma cell lines with MSI demonstrate both TGF-b resistance and BAT-RII mutations that inactivate the RII receptor (7, 9). Accordingly, we initially examined the MSS colon cancers for BAT-RII mutations using our previously described assay (8, 9). Consistent with our prior results, no BAT RII mutations were detected in any of these cell lines (7). Therefore, we determined the complete RII cDNA sequences in RT-PCR cDNA pools amplified from each of the 14 MSS cell lines that demonstrated TGF-b resistance. In three cell lines (15%), only mutant RII sequences were obtained, and no wildtype RII was expressed. In the remaining cell lines, RII was expressed and was wild-type in sequence. RII mutations were previously demonstrated to be ubiquitous among the approximately 13% of colon cancers that show MSI (7, 9, 18). These new findings therefore double the number of colon cancers in which RII mutations are pathogenetic to approximately 28% of all colon adenocarcinomas and show that such RII mutant cases occur in both the MSI and MSS subtypes. The mutations detected in MSS cell lines Vaco8-2, Vaco400, and Vaco410 are located within conserved regions of the kinase domain of the RII gene (Vaco8-2, Vaco410, and Vaco400 cells) or in the 59 extracytoplasmic portion of the receptor (Vaco400 cells). In Vaco8-2 cells, a missense mutation (GAC3AAC) changed an aspartic acid to an asparagine at codon 522 (Fig. 1; Ref. 19), altering the charge of a residue within subdomain XI of the serine/threonine kinase domain of the receptor. In Vaco410 cells, a missense mutation (CGT3CAT) changed an arginine to a histidine at codon 528 (Fig. 1; Ref. 19). This arginine, which is also within kinase subdomain XI, is strictly conserved among all serine/threonine protein kinases (20). In Vaco400 cells, two mutations were observed in the pooled cDNA products; both were apparently heterozygous (Fig. 1). Sequencing individual RT-PCR cDNA clones revealed that this was due to separate mutations present on each of the two expressed Vaco400 alleles. One Vaco400 allele carried a missense mutation (TAT3GAT) at codon 470 that changed a tyrosine to an aspartic acid (19) and altered the charge of an amino acid in subdomain IX of the receptor kinase domain. The second Vaco400 allele carried a missense mutation (AAA3ACA) at codon 52 that changed a lysine to a threonine in the extracytoplasmic region of the receptor. To confirm that these base changes did not merely arise during the establishment of the cell lines, we analyzed genomic DNA from the tumors from which Vaco8-2, Vaco400, and Vaco410 were established. In each case, the RII mutations were demonstrated to be present in the antecedent tumors from which the cell lines were derived. Normal tissue was also available from individuals matched to Vaco410 and Vaco400. In both instances, only wild-type RII sequences were demonstrated in the normal tissues, confirming that in these tumors, RII mutations arose somatically and were selected for during carcinogenesis. Kinase domain mutations in RII have previously been observed in occasional colon cancers with MSI both by us and by other investigators (9, 11) and have also been noted in two squamous cell carcinomas of the oropharynx (21) and in one T-cell lymphoma (22). The selection of these Vaco colon cancer mutations during tumorigenesis, their location within RII, and the nature of the amino acid changes produced suggested that these mutations likely inactivated the TGF-b receptor and induced the TGF-b resistance observed in the Vaco cell lines. Of note, RII mutations were not observed in an investigation of Fig. 1. Point mutations of TGF-b RII in colon cancer cell lines Vaco410, Vaco8-2, and Vaco400. RII cDNA was PCR-amplified in two overlapping pieces, cloned into the pCRII vector (Invitrogen), and then sequenced manually or by ABI automated sequencing. Vaco410 has a homozygous missense mutation in exon 7 at bp 1918 (G3A) that changes codon 528 from Arg to His. Vaco8-2 has a homozygous missense mutation at bp 1899 (G3A) that changes codon 522 from Asp to Asn. Vaco400 has two heterozygous missense mutations that are on opposite alleles. One mutation is located at bp 490 and changes codon 52 from Lys to Thr, and the other mutation is at bp 1743 and changes codon 470 from Tyr to Asp. The mutations in Vaco400 (at codon 470), Vaco410, and Vaco8-2 all affect the kinase domain of the receptor. The mutation at codon 52 in Vaco400 affects the extracytoplasmic portion of RII. 321 TGF-b RECEPTOR MUTATIONS AND MSS COLON CANCER Downloaded from cancerres.aacrjournals.org on May 28, 2017. © 1999 American Association for Cancer Research.  a different set of colon cancers studied by single-strand conformational polymorphism; this contrasting result is most likely a consequence of the lower sensitivity of single-strand conformational polymorphism compared to sequencing (11). RII Mutations Inactivate TGF-b Receptor Signaling. To confirm that the RII mutations detected in the MSS colon cancers inactivated the receptor, we determined the ability of wild-type RII to restore TGF-b-mediated responses in these cell lines. We initially assayed the ability of wild-type RII to restore TGF-b-mediated transcriptional responses as assayed by the TGF-b-responsive firefly luciferase activity encoded by the reporter construct p3TP-Lux (16). p3TP-Lux was transiently transfected into these cell lines accompanied by either RII expression vector, pRC/CMV-TGF-bRII, or control DNA, and p3TP-Lux-mediated luminescence was determined in the presence and absence of exogenous TGF-b1. Each determination was corrected for transfection efficiency by assaying for the luminescence from a cotransfected control construct, pRL-TK or pRL-CMV (Promega), a plasmid that expresses R. reniformis luciferase activity that is easily discriminated from firefly luciferase activity. Consistent with the findings that Vaco8-2, Vaco400, and Vaco410 were resistant to TGF-b-mediated growth inhibition, each of these cell lines demonstrated low basal activity of p3TP-Lux and no further response of the reporter to added TGF-b1 (Fig. 2). In Vaco410 and Vaco400 cells, cotransfection of wild-type RII augmented the basal p3TP-Lux activity. p3TP-Lux activity increased further after the addition of exogenous TGF-b1, such that total the 3TP-Lux output reached 5–10-fold over baseline (Fig. 2). The increase in p3TP-Lux activity due to the introduction of wild-type RII likely reflects the activation of the restored TGF-b signaling pathway by the autocrine TGF-b produced by these cell lines.5 The responsiveness of p3TP-Lux to exogenous TGF-b1 clearly establishes that introducing wild-type RII restores TGF-b-mediated signaling in these cell lines. Thus, the RII mutations in Vaco410 and Vaco400 directly account for the TGF-b resistance of these cell lines. As discussed below, we hypothesized that the Vaco8-2 RII mutation also inactivated receptor signaling, but that additional progression events in this cell line interfered with the ability to reconstitute TGF-b signaling by a single gene alone. Tumor Suppressor Activity of Wild-Type RII in MSS Colon Cancer. To assess the potency of the TGF-b tumor suppressor pathway inactivated in RII mutant MSS colon cancers, we determined the ability of wild-type RII to inhibit colony formation in RII mutant cancer cells. Colon cancer cells were plated at a concentration selected to achieve clonogenic density after G418 selection and transduced with MFG-RII, a replication-incompetent retrovirus encoding wildtype RII, or an equal titer of a control MFG-CAT retrovirus, and then colonies arising in the presence or absence of TGF-b1 were counted and compared. Noninfected cells were removed from the assay by selection of the cells in G418. As expected, TGF-b1 had no colony suppression activity in any of the cell lines infected with the control MFG-CAT virus (Fig. 3). In contrast, 90% of colonies were suppressed in both Vaco400 and Vaco410 cells that were transduced with wild-type RII and treated with TGF-b1 (Fig. 3). Thus, RII is revealed as a potent suppressor gene in these cell lines, and the RII mutations in these tumors are again confirmed as the cause of their resistance to TGF-b growth inhibition. Similar to the inability of wild-type RII in Vaco8-2 to restore TGF-b-mediated transcriptional responses, we observed that wild-type RII also did not restore TGF-b-mediated growth inhibition in Vaco8-2. RT-PCR and Western blot analysis confirmed that after MFG-RII infection in Vaco8-2, wild-type RII cDNA was expressed, and RII protein expression was increased to levels similar to those obtained in MFG-RII-infected Vaco400 and Vaco410. Parenthetically, we noted that because G418-resistant Vaco8-2 colony numbers were the same in MFG-CAT- and MFGRII-transduced Vaco8-2, there was no apparent nonspecific toxicity of the MFG-RII virus relative to the MFG-CAT control. Thus, the partial reduction of Vaco400 and Vaco410 colony numbers by the MFG-RII retrovirus alone is likely due to the autocrine TGF-b produced by these cell lines. Mutation of Both RII and Smad4 in Vaco8-2. Because wild-type RII did not restore TGF-b-mediated signaling in the Vaco8-2 cell line, the functional properties of the Vaco8-2 RII mutant were studied by 5 S. Markowitz and M. G. Brattain, unpublished data. Fig. 2. Restoration by RII of TGF-b-stimulated transcriptional responses. Relative luciferase activity from TGF-b-responsive reporter plasmid 3TPLux in cell lines Vaco410, Vaco400, and Vaco8-2 after transient transfection with pRC/CMV RII is shown. The cells were grown for 2 days after transfection with or without TGF-b1 (10 ng/ml). Vaco410 and Vaco400 demonstrate both a basal and TGF-b-inducible increase in 3TPLux activity after transfection with RII. The basal induction in Vaco410 and Vaco400 is presumably secondary to autocrine TGF-b produced by these cell lines. Vaco8-2 shows no increase in 3TPLux activity after RII transfection with or without the addition of TGF-b1. The luciferase activity was normalized to the activity of a cotransfected control vector, pRL-TK or pRL-CMV, that contains the distinguishable R. reniformis luciferase. Fig. 3. Suppression of colony-forming activity by wild-type RII. The cell lines were plated at a concentration to yield clonogenic density after G418 selection, infected with a RII retrovirus, and then assessed for colony formation 7–10 days after being grown in the presence or absence of exogenous TGF-b1 (10 ng/ml). Reconstitution of RII suppresses the number of colonies formed in Vaco400 and Vaco410. This suppression is significantly augmented by exogenous TGF-b. Vaco8-2 shows no suppression of colony formation after RII reconstitution. MFG-CAT, a MFG retrovirus encoding CAT and a neomycin resistance gene, was used to control for nonspecific effects of the retrovirus. 322 TGF-b RECEPTOR MUTATIONS AND MSS COLON CANCER Downloaded from cancerres.aacrjournals.org on May 28, 2017. © 1999 American Association for Cancer Research.  cloning this mutation into a cDNA expression vector and transiently transfecting the Vaco8-2 RII into the RII-deficient cell line Hct116 (7). As shown in Fig. 4A, transient transfection of wild-type RII restored TGF-b signaling in Hct116 as assayed by the p3TP-Lux reporter. In contrast, the Vaco8-2 RII was inactive in restoring RII function in Hct116. Hence, the Vaco8-2 RII mutation that was selected for during Vaco8-2 tumorigenesis is sufficient to inactivate the TGF-b receptor and to abolish TGF-b signaling. We initially considered the hypothesis that the Vaco8-2 RII mutation might have dominant negative activity, rendering the cell line resistant to the addition of a wild-type allele. However, cotransfecting the Vaco8-2RII and wild-type RII expression vectors revealed, at best, a borderline inhibition of wild-type RII by the Vaco8-2 mutant. (Fig. 4A); accordingly, we hypothesized that in Vaco8-2, a separate event had further inactivated the TGF-b signaling pathway. The binding of TGF-b to its receptor activates a heteromeric complex of Smad2-Smad3-Smad4 transcription factors to translocate to the nucleus (23, 24). Previously, we and others have reported that mutations of Smad2 and Smad4 are present at a low frequency in some human colon cancers (13, 14, 25, 26). Accordingly, we examined the possibility of Smad inactivation as a separate event in the Vaco8-2 TGF-b signaling pathway. Whereas neither Smad4 nor RII transfected singly into Vaco8-2 triggered the 3TP-Lux reporter, cotransfection of RII and Smad4 triggered a 5–10-fold p3TP-Lux response, thus suggesting that the inactivation of RII and Smad4 accounted for two separate defects in the Vaco8-2 TGF-b signaling pathway (Fig. 4B). Examination of the Vaco8-2 Smad4 gene revealed that neither Smad4 exon 1 nor exon 2 could be amplified from Vaco8-2 genomic DNA. Thus, biallelic deletion of Smad4 was indeed a second genetic event in the Vaco8-2 cell line TGF-b pathway. Due to the small amounts of residual normal tissue remaining in even the microdissected tumor antecedent to the Vaco8-2 cell line, we could not establish whether Smad4 loss occurred in the predecessor Vaco8-2 tumor or only after establishment of the Vaco8-2 cell line. Nonetheless, it is intriguing to speculate that the presence of mutations in both the RII and Smad4 genes in the Vaco 8-2 cell line, although rare, seems to suggest that RII and Smad4 may have some functions that are distinct from one another. In summary, multiple lines of evidence now suggest that the TGF-b pathway is a potent tumor suppressor of human colorectal carcinogenesis. We have previously determined that among the 13% of human colon cancers of the MSI subtype, nearly all have inactivating RII frameshift mutations clustering at the BAT-RII tract (7, 9). Data from the Vaco colon cancer cell lines now suggest that among MSS human colon cancers, an additional 15% of tumors also bear inactivating RII mutations, and that restoring wild-type RII to these tumors is potently growth suppressive. Hence, RII mutations play a pathophysiological role in approximately 30% of human colon cancers, and RII gene therapy may have a future role in the treatment of these tumors. Analysis of TGF-b sensitivity in the Vaco colon cancer cell line panel further suggests that an additional 55% of human colon cancers demonstrate functional inactivation of TGF-b-mediated growth inhibition. Furthermore, these cancer lines have lost TGF-bmediated transcriptional responses as well.6 Presumptively, these additional cancers bear defects in the TGF-b signaling pathway at points distal to RII. We have previously shown that Smad2 and Smad4 mutations are present in only a small percentage of human colon cancers (13, 14, 27). However, the tumor-promoting activity of such downstream inactivation of TGF-b signaling is suggested by recent studies in murine models showing that heterozygous Smad4 mutation promotes colon adenoma to carcinoma progression (28), and homozygous Smad3 knockout leads directly to colon cancer (29). Study of additional mechanisms leading to the inactivation of TGF-b signaling in the remaining set of wild-type RII MSS colon cancers is currently ongoing.  Adjuvant chemotherapy improves survival among patients with stage III colon cancer, but no reliable molecular predictors of outcome have been identified.  Methods  We evaluated loss of chromosomal material (also called loss of heterozygosity or allelic loss) from chromosomes 18q, 17p, and 8p; cellular levels of p53 and p21WAF1/CIP1 proteins; and microsatellite instability as molecular markers. We analyzed tumor tissue from 460 patients with stage III and high-risk stage II colon cancer who had been treated with various combinations of adjuvant fluorouracil, leucovorin, and levamisole to determine the ability of these markers to predict survival.  Results  Loss of heterozygosity at 18q was present in 155 of 319 cancers (49 percent). High levels of microsatellite instability were found in 62 of 298 tumors (21 percent), and 38 of these 62 tumors (61 percent) had a mutation of the gene for the type II receptor for transforming growth factor β1 (TGF-β1). Among patients with microsatellite-stable stage III cancer, five-year overall survival after fluorouracil-based chemotherapy was 74 percent in those whose cancer retained 18q alleles and 50 percent in those with loss of 18q alleles (relative risk of death with loss at 18q, 2.75; 95 percent confidence interval, 1.34 to 5.65; P=0.006). The five-year survival rate among patients whose cancer had high levels of microsatellite instability was 74 percent in the presence of a mutated gene for the type II receptor for TGF-β1 and 46 percent if the tumor did not have this mutation (relative risk of death, 2.90; 95 percent confidence interval, 1.14 to 7.35; P=0.03).  Conclusions  Retention of 18q alleles in microsatellite-stable cancers and mutation of the gene for the type II receptor for TGF-β1 in cancers with high levels of microsatellite instability point to a favorable outcome after adjuvant chemotherapy with fluorouracil-based regimens for stage III colon cancer.  Colorectal cancer is the second most common cause of death from cancer in the United States.1 Postoperative adjuvant chemotherapy improves the outcome in stage III (Dukes’ stage C) colon cancer and is now widely accepted as standard therapy.2,3 Many patients with stage II (Dukes’ stage B) disease are considered to be at high risk for recurrence and receive adjuvant therapy, although its benefit in such cases is uncertain. Markers that reliably predict survival are needed.2,4,5  The sequence of genetic alterations leading to colorectal cancer usually begins with the inactivation of the pathway involving the adenomatous polyposis coli tumor-suppressor gene and β-catenin.6,7 Subsequent changes often include the loss of portions of chromosomes, termed loss of heterozygosity or allelic loss, or the loss of whole chromosomes.8 In about 15 percent of cases of sporadic colorectal cancers, there are insertions or deletions of nucleotides within repeated sequences of DNA, termed microsatellite instability, due to defective repair of mismatched nucleotides.9–11 Tumors with high levels of microsatellite instability are characteristic of the hereditary non-polyposis colorectal cancer syndrome, but in most cases such tumors are sporadic.11–13 Neoplasms with high levels of microsatellite instability accumulate mutations in microsatellites within the coding region of certain genes,10,11,14 but loss of chromosomes is rare.8  Some of these genetic alterations are prognostic markers in colorectal cancer.15 Loss of heterozygosity at chromosome 18q indicates a poor prognosis.15–24 Other alterations that have been found to have prognostic value are allelic loss at chromosomes 17p, 1p, 3p, 4p, 5q, or 8p; changes in the levels of certain gene products, including the DCC (deleted in colorectal cancer) protein, p53, and p27Kip1; mutation of the ras gene; and increased expression of genes involved in fluoropyrimidine metabolism.15,25 In addition, colorectal cancers with high levels of microsatellite instability metastasize less often and have a better prognosis than microsatellite-stable cancers.15,26–33  Molecular alterations also have the potential to predict survival after chemotherapy.34–38 We examined a panel of molecular markers (listed in the Glossary) in specimens of colon cancer from patients enrolled by the Eastern Cooperative Oncology Group in two National Cancer Institute Gastrointestinal Intergroup clinical trials of adjuvant chemotherapy with fluorouracil-based regimens.  Glossary of Molecular Markers.  Marker\\tAbnormality or Abnormal Gene*\\tFunctions of Wild-Type Gene Product\\tReported Prognostic or Predictive Value in Colorectal Cancer Loss of heterozygosity at chromosome 18q\\tDCC\\tNetrin-1 receptor; caspase substrate in apoptosis; cell adhesion molecule; tumor suppression\\tAdverse prognostic marker Smad4 (DPC4, MADH4)\\tNuclear transcription factor in TGF-β1 signaling; regulation of angiogenesis; activator of WAF1 promoter; downstream mediator of Smad2; tumor suppression\\tAdverse prognostic marker Smad2 (MADH2, JV18)\\tEndodermal differentiation; interacts with SKI protein\\tNone Loss of heterozygosity at chromosome 17p\\tp53 (TP53)\\tTranscription factor; regulator of cell-cycle progression after cellular stress, of apoptosis, of gene expression, and of DNA repair; tumor suppression\\tAdverse prognostic marker Loss of heterozygosity at chromosome 8p\\tUnknown\\tUnknown\\tAdverse prognostic marker High labeling index for p53 protein\\tp53 (TP53)\\tSame as for loss of heterozygosity at chromosome 17p\\tAdverse prognostic marker, adverse predictive marker Increased labeling index for p21WAF1/CIP1 protein\\tWAF1 (CIP1, SDI1)\\tCyclin-dependent kinase inhibitor; controller of cell-cycle progression\\tFavorable predictive marker Microsatellite instability\\tConsequence of abnormal genes in mismatch-repair family\\tRepair of nucleotide mismatches\\tFavorable prognostic marker, favorable predictive marker Mutation in gene for type II receptor for TGF-β1\\tTGF-β1 RII\\tReceptor for signaling in TGF-β1 pathway; inhibitor of colonic epithelial proliferation\\tNone Mutation in BAX gene\\tBAX\\tProapoptosis\\tAdverse prognostic marker *Alternative terms are given in parentheses. Go to: METHODS Patients  Specimens from 516 eligible patients enrolled in two randomized trials of adjuvant chemotherapy for colon carcinoma were studied. These patients had stage III cancer (Dukes’ stage C, with lymph-node metastasis) or high-risk stage II cancer (Dukes’ stage B2, with colonic obstruction, adherence to or invasion of adjacent organs, or tumor perforation and with en bloc resection of all visible disease, including regional peritoneal metastases). In one trial (Eastern Cooperative Oncology Group protocol E2284 [National Cancer Institute Gastrointestinal Intergroup INT 0035]), three treatments — fluorouracil plus levamisole, levamisole alone, and surgery alone — were compared.39 Patients in this trial were enrolled from February 1985 to October 1987, and the median duration of follow-up in surviving patients was 9.0 years. In the other trial (protocol E2288 [INT 0089]), four treatments — low-dose leucovorin plus fluorouracil, high-dose leucovorin plus fluorouracil, levamisole plus fluorouracil, and low-dose leucovorin plus fluorouracil plus levamisole — were compared.40 Patients were enrolled from August 1988 to July 1992, and the median duration of follow-up in surviving patients was 4.8 years.  Thus, of the seven cohorts in these two trials, five received fluorouracil-based chemotherapy. We studied survival in relation to the presence or absence of molecular markers in available tumor specimens from 460 patients in the five cohorts that received fluorouracil. There were no statistically significant differences in outcome among the patients in these five cohorts. The study population was representative of all the patients in the two clinical trials; the only statistically significant difference between patients included in this study and those not included was the frequency of regional metastases (6 percent vs. 9 percent, P=0.04) (Table 1).  Table 1 Table 1 Principal Characteristics of the Patients Enrolled in the Two Clinical Trials, According to Whether the Tumor Was Analyzed in the Current Study.* Analysis of Tumor Specimens  Formalin-fixed, paraffin-embedded specimens were obtained through the Eastern Cooperative Oncology Group Pathology Coordinating Office. A data base of information on the patients was maintained at the Eastern Cooperative Oncology Group Statistical Center. Laboratory analysis of tumor specimens was performed without knowledge of the patients’ clinical data. Microdissection of tumor tissue and of non-neoplastic control tissue, when available, and preparation of DNA were performed as previously described.14,16,32,41  Microsatellite Markers The microsatellite analysis depended on the type of tissue available (Fig. 1). In the case of 298 tumor specimens for which non-neoplastic control tissue was also available, allelic losses from chromosomes 18q, 17p, and 8p were evaluated with polymorphic markers, and microsatellite instability was determined with eight dinucleotide and two mononucleotide markers (additional information is available with the full text of the article at www.nejm.org).14,16,32,42 The 218 tumor specimens for which insufficient control tissue was available were tested for microsatellite instability with two mononucleotide markers that are rarely polymorphic and that do not require control tissue for evaluation. The mononucleotide repeat in the BAX gene14 was analyzed in cancers with high levels of microsatellite instability.  Figure 1 Figure 1 Autoradiographs of Molecular Markers Allelic Loss at Chromosomes 18q, 17p, and 8p Allelic loss was determined by examining autoradiographs of DNA amplified by the polymerase chain reaction (PCR). Allelic loss was defined as a reduction in the intensity of the autoradiograph of one of the two alleles in the amplified tumor DNA to at least 50 percent of the level of DNA in non-neoplastic control tissue.14,16,42 Of the 298 cases for which control DNA was available, the results for 18q were interpretable in 279 (94 percent), for 17p in 223 (75 percent), and for 8p in 201 (67 percent). Tumors with high levels of microsatellite instability infrequently lose chromosomes, including 18q, 17p, and 8p,8,14 and were therefore categorized as having no allelic loss; these consisted of 40 additional tumors with no loss at 18q, 102 with no loss at 17p, and 102 with no loss at 8p.  The p53 and p21WAF1/CIP1 Proteins Immunohistochemical analysis was performed as previously described.43,44 Three categories of p53 staining were defined for statistical analysis with the use of labeling-index cutoff points of 40 percent and 5 percent after quantitation by computer-assisted image analysis.43 After initial evaluation of immunohistochemical results for p21WAF1/CIP1, the labeling index was estimated and categorized as greater than 30 percent, 20 to 30 percent, 10 to 19 percent, 5 to 9 percent, and less than 5 percent of nuclei. Of the 516 analyzed tumors, results were interpretable for 445 (86 percent).  Classification of Microsatellite Instability Changes in the electrophoretic mobility of DNA amplified by PCR were used to assess microsatellite instability.14,16,32,42 The number of markers with altered allelic sizes and the number of technically satisfactory markers were recorded for each tumor. In evaluating the 298 carcinomas for which control DNA was available, we ascertained microsatellite instability with the use of the interpretable markers among eight dinucleotide markers and two polyadenine tracts. Tumors with a shift in at least two markers and at least 30 percent of the interpretable markers were classified as having high levels of microsatellite instability, in accordance with international criteria.11 A low level of microsatellite instability was defined as a shift in only one dinucleotide marker. In this study, tumors with low levels of microsatellite instability were categorized as microsatellite-stable tumors.11  All the tumors with a shift in a mononucleotide marker had high levels of microsatellite instability when examined with the complete panel of markers, as reported previously.45,46 Therefore, in the 218 cases without control DNA that were evaluated with two mono-nucleotide markers, a shift in a marker was considered to indicate high levels of microsatellite instability.  Statistical Analysis  Cases with missing results were included in all analyses that did not involve the missing data. The Cochran–Mantel–Haenszel (stratum-adjusted Pearson’s chi-square) test and Pearson’s chi-square test were used to analyze associations among categorical variables.47 Analysis of variance was applied to data on age and tumor size. Survival curves were estimated by the method of Kaplan and Meier,48 and differences were assessed by means of the stratified log-rank test.49 Proportional-hazards regression models were used for multivariable comparisons of time-to-event end points.50 All computations were performed with SAS software (version 6.12, SAS Institute, Cary, N.C.). All P values were calculated with two-sided tests of significance.  Go to: RESULTS We first present the genetic abnormalities in the colon cancers we studied and then relate these molecular findings to the survival of the patients after adjuvant chemotherapy.  Genetic Alterations  Loss of heterozygosity at chromosome 18q was observed in 155 of 319 cancers (49 percent). Of these 155 specimens, 143 (92 percent) had loss of all the analyzed 18q markers. Loss of heterozygosity at 17p was found in 166 of 325 tumors (51 percent). Of 309 tumors, 254 (82 percent) had allelic loss from both 18q and 17p or from neither (P<0.001 for concordance between the status of 18q and 17p). Loss of 8p alleles was found in 95 of 303 tumors (31 percent).  Of 445 cancers, 205 (46 percent) had a high labeling index for p53 protein, a finding consistent with a mutation of the p53 gene.51 In 204 of 288 tumors (71 percent), a high p53 labeling index was associated with allelic loss from 17p and a low index was associated with retention of 17p (P<0.001 for the concordance between p53 labeling index and 17p status). The p21WAF1/CIP1 protein was detected in 211 of 445 tumors (47 percent). In 276 of 445 cancers (62 percent), there was an inverse relation between p53 and p21WAF1/CIP1 labeling (P<0.001).  Of the 298 tumor specimens we evaluated for microsatellite instability, 62 (21 percent) had high levels of microsatellite instability. Low levels of microsatellite instability were found in 28 of 298 cancers (9 percent), which were categorized as the microsatellite-stable tumors.  Of the 218 tumor specimens for which control DNA was not available, high levels of microsatellite instability were found in 40 specimens (18 percent). Of the 516 specimens in the entire study that could be analyzed, 102 (20 percent) were classified as having high levels of microsatellite instability and 227 (44 percent) as having microsatellite stability; in 187 specimens (36 percent) the results of evaluation were indeterminate or unsatisfactory.  High levels of microsatellite instability and a high labeling index for p53 protein were found in 24 of 90 tumor specimens (27 percent), whereas 106 of 202 tumor specimens (52 percent) with microsatellite stability had a high p53 labeling index (P<0.001). Among the tumor specimens that had high levels of microsatellite instability and that were evaluated with dinucleotide and mononucleotide markers, mutation of the gene for the type II receptor for transforming growth factor β1 (TGF-β1) was present in 38 of 62 specimens (61 percent). None of the tumor specimens with low levels of microsatellite instability or with microsatellite stability had such a mutation. Mutation of the BAX gene was present in 22 of 60 cancers (37 percent) with high levels of microsatellite instability.  Survival Analysis  Among the 460 patients who were treated with fluorouracil-based chemotherapy, female sex, less advanced stage of disease, and the absence of regional metastases were significant favorable predictors of five-year disease-free survival and five-year overall survival after chemotherapy (Table 2). Younger age (≤65 years) and the presence of a well-differentiated adenocarcinoma, as compared with a moderately or poorly differentiated tumor, were also favorable predictors of five-year overall survival.  Table 2 Table 2 Disease-free and Overall Survival at Five Years in Relation to Clinical and Pathological Characteristics of Patients with Stage II or III Colon Cancer Treated with Fluorouracil-Based Adjuvant Chemotherapy.* Because the efficacy of adjuvant chemotherapy is more firmly established for patients with stage III cancer than for those with stage II cancer, survival among those with stage II disease was analyzed separately. Of the molecular markers we tested, the status of 18q was significantly associated with both five-year disease-free survival and five-year overall survival after chemotherapy among patients with stage III cancer (Table 3). Patients with tumors that retained 18q had a five-year disease-free survival rate of 64 percent, as compared with 44 percent among those with loss of heterozygosity at 18q (P=0.002) (Fig. 2 and Table 3). The corresponding five-year overall survival rates were 69 percent with retention of 18q alleles and 50 percent with allelic loss at 18q (P=0.005) (Fig. 3 and Table 3). The 18q status also had predictive value in an analysis of the subgroup of patients with microsatellite-stable stage III carcinoma.  Figure 2 Figure 2 Disease-free Survival According to the Analysis of Molecular Markers in Patients with Stage III Colon Cancer Treated with Postoperative Adjuvant Chemotherapy with Fluorouracil-Based Regimens Figure 3 Figure 3 Overall Survival According to the Analysis of Molecular Markers in Patients with Stage III Colon Cancer Treated with Postoperative Adjuvant Chemotherapy with Fluorouracil-Based Regimens Table 3 Table 3 Disease-free and Overall Survival at Five Years in Relation to Molecular Markers in Patients with Stage III Colon Cancer Treated with Fluorouracil-Based Adjuvant Chemotherapy.* Mutation of the gene for the type II receptor for TGF-β1, a specific indicator of high levels of micro-satellite instability, was marginally associated with improved five-year overall survival (Table 3). High levels of microsatellite instability and alteration of the BAT-26 marker were also moderately associated with improved disease-free survival at five years (P=0.02 for both associations). Among the patients who had stage III cancer with both high levels of microsatellite instability and mutation of the gene for the type II receptor for TGF-β1, the rate of disease-free survival at five years was 79 percent, as compared with 40 percent among those whose tumors had high levels of micro-satellite instability and no mutation of this gene (P= 0.007) (Fig. 2 and Table 3). The corresponding rates of overall survival at five years were 74 percent and 46 percent (P=0.04) (Fig. 3 and Table 3).  There was no relation between survival after treatment with a particular regimen and the presence of any of the molecular markers. No marker had predictive value in the analysis of 121 patients with stage II cancer, possibly because of the small sample.  In proportional-hazards regression models that were adjusted for sex, age, the extent of spread, the presence or absence of regional metastases, and the presence or absence of obstruction, several variables —allelic loss at 18q, microsatellite stability, absence of mutation of the gene for the type II receptor for TGF-β1, and absence of an allelic shift in BAT-26 —were each independently associated with an increased risk of recurrence (Table 4). In models in which multiple markers were analyzed, microsatellite-stable cancers had a higher relative risk of recurrence than cancers with high levels of microsatellite instability and mutation of the gene for the type II receptor for TGF-β1 (relative risk, 2.60; 95 percent confidence interval, 1.36 to 4.95; P=0.004). After adjustment for multiple markers, loss of 18q alleles remained an indicator of recurrence and death (Table 4); in contrast, allelic loss at 17p was not predictive either in univariate analysis or after such adjustments (data not shown).  Table 4 Table 4 Relative Risks of Recurrence and Death in Patients with Stage III Colon Cancer Treated with Fluorouracil-Based Postoperative Adjuvant Chemotherapy, According to Proportional-Hazards Regression Models. Go to: DISCUSSION In this study, we showed that the status of chromosome 18q in tumors with microsatellite stability and of the gene for the type II receptor for TGF-β1 in tumors with high levels of microsatellite instability could be used to predict the likelihood of survival in patients with stage III colon cancer who received fluorouracil-based adjuvant chemotherapy. We do not know whether these markers reflect resistance or sensitivity to fluorouracil or inherent differences in the biologic characteristics of the tumors.  In several15–24 but not all15,29,52,53 previous studies, loss of heterozygosity at 18q was an indicator of a poor prognosis in patients with stage II cancer, patients with stage III cancer, or both groups. This loss usually involves the DCC gene, but there are numerous other genes in the deleted region. The product of the DCC gene is the netrin-1 receptor, which guides the migration of neuronal axons.54–57 In colon cancer, loss of DCC is associated with metastasis and an adverse prognosis.58–61 If it does not bind to netrin-1, the DCC protein triggers apoptosis.62 For this reason, loss of DCC as a result of loss of 18q could impair apoptosis, thereby conferring resistance to chemotherapy. The absence of an association between survival and loss of heterozygosity at 8p or 17p suggests that loss of heterozygosity at 18q is a specific marker for survival and not simply a reflection of generalized chromosomal instability.  Our study confirmed the concordance between allelic loss at chromosome 18q and allelic loss at 17p.16 Although alteration of p53 is a plausible predictive marker,15,33,34,63–69 we found no significant relation between survival and the status of the p53 gene or p53 protein. Another study, however, found a higher rate of seven-year survival after adjuvant therapy with fluorouracil and levamisole in patients who had cancer without increased levels of p53 protein than in those who had cancer with increased p53 levels.36 The explanation for these discrepant results in patients in the same clinical trial is not apparent.  The p21WAF1/CIP1 protein is a downstream effector of the p53 protein,70,71 and we found an inverse relation between p53 and p21WAF1/CIP1 in colon cancer, as has been reported previously.72,73 Despite the importance of p21WAF1/CIP1 for in vitro responses to chemotherapeutic agents35 and the report that increased levels of p21WAF1/CIP1 were associated with chemosensitivity of metastatic colorectal cancer,74 pretreatment levels of this protein were not related to survival in our study.  A mutation of the gene that encodes the type II receptor for TGF-β1 in cancers with high levels of microsatellite instability was associated with a favorable outcome, but the mechanism of this effect is uncertain. High levels of microsatellite instability improve the prognosis15,26–33 and may also increase the likelihood of survival after chemotherapy.37,38 Because cancers with high levels of microsatellite instability usually retain 18q alleles, loss of heterozygosity in such tumors is unlikely to be a determinant of outcome after adjuvant chemotherapy. The TGF-β1 pathway inhibits tumor proliferation by blocking the cell cycle late in the G1 (gap 1) phase,75,76 so continued proliferation due to mutation of the gene for the type II receptor for TGF-β1 could increase susceptibility to chemotherapy. However, colon-cancer cell lines that are deficient in mismatch-repair activity and that have high levels of microsatellite instability are relatively resistant to fluorouracil in vitro.77  We find that specific molecular markers in resected stage III colon cancer can be used to predict survival after adjuvant fluorouracil-based regimens. Prospective studies are needed to determine whether newer chemotherapeutic agents, such as irinotecan78 and oxaliplatin, would benefit patients with stage III cancer whose tumors have molecular markers associated with a reduced efficacy of fluorouracil-based regimens. Our study is a first step toward the goal of individualized cancer treatment based on the molecular characteristics of the tumor.  Abstract The TGF type H receptor (RI!) was found to be mutated within a polyadenine tract in 100 of 111 (90%) colorectal cancers with microsatellite instability. Other polyadenine tracts of similar length were mutated in these samples but not as frequently as RI!. In most cases, the polyadeninetract mutations affected both allelesofRI!, and in four tumors heterozygous for the polyadenine mutations, three had additional mutations that were expected to inactivate the other RH allele. These genetic data support the idea that RU behaves like a tumor suppressor during CR cancer development and is a critical target of inactivation in mismatch repair-deficient tumors. Introduction Of the 160,000 new CR3 cancers diagnosed this year, approxi mately 21,000 will have widespread mutations in microsatellite se quences characteristic of the RER phenotype (1â€”4).An inherited form of CR cancer, HNPCC, accounts for approximately one-fourth of these 21,000 cases, and is due principally to germline mutations of the MMR genes, hMSH2 and hMLHI, hPMSJ, or hPMS2(5â€”8).Somatic mutations of MMR genes also account for a significant fraction of the sporadic CR cancers exhibiting the RER phenotype (9). We have recently shown that RI! was mutated in 8 of 11 RER tumors (10). TGF-f3 inhibits the growth of colon epithelial cells and is the primary ligand for RI! ( 11). The TGF-j3-RI! complex interacts with the TGF-p type I receptor, which in turn activates parallel pathways that inhibit cellular proliferation ( 12). Genetic analysis of these pathways has shown that both type I and type II receptors are absolutely required for growth inhibition (13). Materials and Methods Primary Tumors. Serial 6-@xmparaffin sections of HNPCC tumors were microscopically dissected to separate neoplastic from nonneoplastic elements. Whenever possible, normal and tumor components were prepared from the same paraffin block (14). DNA was prepared from cryostat sections of other CR tumorsas described( I5). In these cases, DNA was also obtainedfrom separate pieces of uninvolved colonic mucosa. Of the primary tumors in this study, DNA was prepared from paraffin sections in 69 cases and from frozen sections in 15 cases. Cell Lines and Xenografts. CR cancercell lines (n = 94) were obtained from ATCC or established as described (16). Xenografts (n = 55) were Received I0/6/95; accepted I0/25/95. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked advertisement in accordance with 18 U.S.C. Section 1734 solely to indicate this fact. @ This work was supported by NIH Grants CA09320, CA35494, CA5 1183, CA57208, and CA62924, American Cancer Society Grant FRA-45l, and the Clayton Fund. B. V. is an investigator of the Howard Hughes Medical Institute. 2 To whom requests for reprints should be addressed, at Ireland Cancer Center, University Circle Research Center 2, Room 200, 11001 Cedar Road, Cleveland, OH 44106. 3 The abbreviations used are: CR, colorectal: HNPCC. hereditary nonpolyposis cola rectal cancer; RER. replication error; MMR. mismatch repair; TGF-@3, transforming growth factor @3; RIl, TGF@ type II receptor; pA, polyadenine; RT-PCR, reverse tran scription-PCR. generated from primary CR cancers and either blood or colonic mucosa samples were used to purify corresponding normal DNA (10). RER status was assessed as described previously using the microsatellite markers BAT-26, BAT-40, D2S123, and D18S58 (17). In some cell lines, RER status could not be assessed directly because of the absence of corresponding normal DNA, and the presence of MMR gene mutations or microsatellite instability was used as a surrogate to indicate RER status. PCR and Sequencing Analysis. Markerloci were PCRamplifiedfor 30 cycles using one 32P-labeled primer and one unlabeled primer. Reactions were resolved on urea-formamide polyacrylamide gels and exposed to film. The primers that were used for the amplification of BAT-Ru were 5'-CTT TAT TCT GGA AGA TGC TGC-3' and 5'-GAA GAA AGT CTC ACC AGG C-3'. BAT-13 and BAT-26 were identified serendipitously while sequencing introns 1 and 5 of hMSH2, respectively. The primers for BAT-13 were 5'-ATC AGC AGC ATG AAG CCA G-3' and 5'-TCT liT ACA AAA GAT TCA AAA TFC-3'; for BAT-26, 5'-TGA CTA Cli' Tl'G ACTTCA GCC-3' and 5'-AAC CAT TCA ACA 1TF TFA ACC C-3'. BAT-bA, BAT-lOB, BAT-25, and BAT-40 tracts were derived from introns of the dystrophin gene, APP gene, c-kit oncogene, and 3-f3-hydroxysteroid dehydrogenase gene, respectively, and were identified by searching the Genbank human data base with polyadenine tracts. The primers for BAT-bA were 5'-GA TAA TAT AGC ATF ATA ACA CTG-3' and 5'-GAA CAC AAA GGA AGT GTC TG-3'; for BAT-lOB, 5'-ATG TCG TAT TAT GAC CAT CAC-3' and 5'-GGT CAA GGC TAC AGT AAG C-3'; for BAT 25, 5'-TCG CCT CCA AGA ATG TAA GT-3' and 5'-TCT GCA m TAA CTA TGG CTC-3'; and for BAT-4O,5'-ACA ACC CTG CU UG TFC CT-3' and 5'-GTA GAG CAA GAC CAC CTF G-3'. The search for lO-bp pA tracts in the human genome led to the identification of six cDNA clones (RMSA-l, human glucose transporter 2, interleukin 5, human mitochondrial transcription factor 1, DNA polymerase accessory pro tein Al, and ISL-l. The search also identified the cDNA for Ril. To screen for mutation within RI!, the entire coding region was amplified using RT-PCR in two overlapping segments. Products were directly sequenced with 32P-labeled primers using SequiTherm polymerase. The primers used for amplification and sequencing of Ril are available from the authors upon request. Results and Discussion RI! has a lO-bp pA tract at codons 125â€”128of its 565-codon open reading frame. Of the 8 CR cancers with RI! mutations demonstrated previously, 7 were caused by small insertions or deletions within this pA tract (BAT-RI!). To examine the generality of these results, we designed a PCR-based assay for analyzing BAT-RI! at the genomic level. A 73-bp genomic DNA fragment encompassing BAT-Ril was amplified from 84 primary tumors known to exhibit the RER pheno type (Fig. 1A). Of these, 78 were from HNPCC patients and 6 were sporadic tumors. Seventy-five of these tumors (89%) had at least one altered BAT-RI! allele (Fig. 1B). All of the alterations were due to lâ€”2-bpinsertions or deletions that would result in a frameshift in the encoded protein. These results are in contrast to the results found in non-RER tumors, in which 0 of 40 were mutated. Inactivation of a single RI! allele by a BAT-RI! frameshift mutation would not be predicted to abrogate RI! function in the presence of an unaltered, second RI! allele. It was, therefore, of interest to determine how often both RI! alleles were mutated in RER tumors. In most of the 5548 Downloaded from cancerres.aacrjournals.org on June 1, 2017. © 1995 American Association for Cancer Research. MICROSATELLITEINSTABILITY AND MUTATIONS A @ 663 4 737 GAC â€˜ITt'AU CTG GAA GAl OCT GCI@id CCA AAG TGC A1@ ATG AAG dAA AAA AAA A/ID CCI GOT GAG ACT TIC lit Asp Phe lIe Leu GIu Asp Ala Ala 5cr Pro Lys Cys 1k Met Lys GIu Lys Lys Lys Pro GIy GIu Thr Pbs Phe 110 134 C D abcde abcdefghij klmn B ab c d e f g h i j k -@ Fig. 1. pA tract of RI!. A, sequence of RI! cDNA encompassing BAT-RH and including amino acids 110â€”134.Arrows, primers used for genomic amplification. The stretch of 10 A residues is boxed. B, analysis of primary tumors. Genomic DNA from sporadic primary RER CR cancers and paired normal colon samples. Lanes a, c, e, g, i, and Ac73-bp PCR products from normal DNA resolved on a sequencing gel. Lanes b, d, f@h, j, and 1, products from tumor samples. Deletions of pA tracts are present in all tumor samples except that in Lane b. C, analysis of xenografts. Genomic amplification of 5 xenografted CRCs. Lanes a, c, and e, amplified products of three non-RER xenografts; Lanes b and d, products for two RER samples. Deletions can be seen in Lanes b and d. D, analysis of cell lines. Lanes a and b, products from normal DNA controls. The products of RER cell lines are shown in Lanes c (SW48), d (LoVo), e (LS-I80),f(DLD-l), g (HCT15), h (VACO5), i (VACO6), k (HCTI 16), 1 (C), and n (KM12). Products of non-RER cell lines are shown in Lanes j (VACO8)andm (VACO4IO).Deletionscanbe seenin cell linesexceptLanesa, b.j. andm. Arrowheads.positionof the wild-type73-bpPCRproduct. primary tumors described above, the band representing the wild-type allele was significantly decreased in intensity compared to the mutant BAT-RI! alleles (Fig. lB). This suggested that both alleles were mutant, and that residual wild-type signal arose from contaminat ing nonneoplastic cells within the tumor specimens. However, the presence of nonneoplastic cells made it difficult to unambiguously interpret these data. To answer the question more definitively, we analyzed 27 RER colorectal cancer cell lines, passaged in vitro or as xenografts in nude mice, for mutations in BAT-RI! sequences. At least 1 BAT-RI! alteration was found in 25 of these samples. In 19 of the 25, both alleles were altered, whereas only 1 allele was altered in 6. As in the primary tumors, lâ€”2-bp insertions or deletions causing frameshifts were always observed. The combined frequency of BAT-RI! mutations in the primary and cultured CR cancer was 90% (100 of 111). In contrast, BAT-RI! was not altered in any of 122 samples from non-RER CR cancer lines (75 cell lines and 47 xenografts). Alterations of numerous pA sequences, distributed widely and randomly throughout the genome, was the characteristic first used to define tumors with microsatellite instability (1, 18). In the context of such widespread changes, it is difficult to ascribe significance to alterations of any single pA tract, such as those in BAT-RI!. To further address this issue, we studied several pA tracts in the tumors described above. There was a gradual decrease in the fraction of Fig. 2. Frequency of alterations in pA tracts of different lengths. RER CR cancers were tested for alterations in BAT-Ru, BAT-bA, BAT-lOB, BAT-I3, BAT-25, BAT 26, and BAT-40. BAT-Ril, BAT-lOA, and BAT-lOB have l0-bp pA tracts; BAT-13 has a 13-bp PA tract; and BAT 25, BAT-26, and BAT-40 have pA tracts of approximately 25, 26, and 40 bp. respectively. Each marker was tested with the same panel of 27 RER CR lines (see text). tumors exhibiting pA tract length alterations as the length of the tract decreased from 40 to 13 bp (Fig. 2). This is consistent with other studies showing that the probability of maintaining a microsatellite in MMR-deficient cells is inversely proportional to the length of the microsatellite (4). To determine whether smaller pA tracts, like those in RI!, were often mutated in RER cancers, we searched Genbank (Version R86.0; December, 1994) for l0-bp pA tracts within genes. No 1O-bp pA tracts were identified within coding exons of genomic DNA entries, but six were identified within coding regions of cDNA entries (See â€oeMaterialsand Methodsâ€•).Of these, four were examined, using multiple primer combinations by PCR, and each failed to yield the expected amplification product. Presumably, this was because introns were present within or closely adjacent to the pA tracts. This suggested that long pA tracts are rare in coding sequences. A search for 1O-bp pA tracts within noncoding DNA revealed numerous tracts in intronic and intergenic genomic DNA, as well as in untranslated portions of cDNAs. PCR-based amplification was then performed on two randomly chosen intronic l0-bp pA tracts (BAT-bA and BAT lOB) using genomic DNA as template (see â€oeMateriaalsnd Methodsâ€•). BAT-1OA and BAT-lOB were altered less frequently (70 and 41%, respectively) than were longer pA tracts. Moreover, BAT-lOA and BAT-lOB were altered less frequently than BAT-RIl in the same tumors (P < 0.03 for each by @. Additionally, we found that pA tracts rarely became smaller than 10 nucleotides in RER tumor cells, BAT-RH BAT-bA BAT.IOB BAT.l3 BAT-25 5549 MICROSATELLITEINSTABILITY AND MUTATIONS mutations, it will be of interest to determine whether other components of the TGF-j3 pathway are responsible for this insensitivity.\""
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# looking at sample entry\n",
    "data_text['Text'][100]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "64ca6392",
   "metadata": {},
   "outputs": [],
   "source": [
    "def preprocess(total_text, index, column):\n",
    "    if type(total_text) is not int:\n",
    "        string = \"\"\n",
    "        # Replace multiple spaces with single space\n",
    "        total_text = re.sub('\\s+',' ', total_text)\n",
    "        # Converting all the chars into lower-case\n",
    "        total_text = total_text.lower()\n",
    "        \n",
    "        data_text[column][index] = total_text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "be0a19c4",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/2f/3wddnk1d66930qz4f367j7040000gn/T/ipykernel_2802/521996428.py:9: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  data_text[column][index] = total_text\n"
     ]
    }
   ],
   "source": [
    "no_text = []\n",
    "for index, row in data_text.iterrows():\n",
    "    if type(row['Text']) is str:\n",
    "        preprocess(row['Text'], index, 'Text')\n",
    "    else:\n",
    "        no_text.append(index)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "88d72899",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Class</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>FAM58A</td>\n",
       "      <td>Truncating Mutations</td>\n",
       "      <td>1</td>\n",
       "      <td>cyclin-dependent kinases (cdks) regulate a var...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>CBL</td>\n",
       "      <td>W802*</td>\n",
       "      <td>2</td>\n",
       "      <td>abstract background non-small cell lung cance...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>CBL</td>\n",
       "      <td>Q249E</td>\n",
       "      <td>2</td>\n",
       "      <td>abstract background non-small cell lung cance...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>CBL</td>\n",
       "      <td>N454D</td>\n",
       "      <td>3</td>\n",
       "      <td>recent evidence has demonstrated that acquired...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>CBL</td>\n",
       "      <td>L399V</td>\n",
       "      <td>4</td>\n",
       "      <td>oncogenic mutations in the monomeric casitas b...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   ID    Gene             Variation  Class  \\\n",
       "0   0  FAM58A  Truncating Mutations      1   \n",
       "1   1     CBL                 W802*      2   \n",
       "2   2     CBL                 Q249E      2   \n",
       "3   3     CBL                 N454D      3   \n",
       "4   4     CBL                 L399V      4   \n",
       "\n",
       "                                                Text  \n",
       "0  cyclin-dependent kinases (cdks) regulate a var...  \n",
       "1   abstract background non-small cell lung cance...  \n",
       "2   abstract background non-small cell lung cance...  \n",
       "3  recent evidence has demonstrated that acquired...  \n",
       "4  oncogenic mutations in the monomeric casitas b...  "
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# merge gene_var and text based on ID\n",
    "result = pd.merge(data, data_text, on='ID', how = 'left')\n",
    "result.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "5f10cdde",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Class</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>FAM58A</td>\n",
       "      <td>Truncating Mutations</td>\n",
       "      <td>1</td>\n",
       "      <td>cyclin-dependent kinases (cdks) regulate a var...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>CBL</td>\n",
       "      <td>W802*</td>\n",
       "      <td>2</td>\n",
       "      <td>abstract background non-small cell lung cance...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>CBL</td>\n",
       "      <td>Q249E</td>\n",
       "      <td>2</td>\n",
       "      <td>abstract background non-small cell lung cance...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>CBL</td>\n",
       "      <td>N454D</td>\n",
       "      <td>3</td>\n",
       "      <td>recent evidence has demonstrated that acquired...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>CBL</td>\n",
       "      <td>L399V</td>\n",
       "      <td>4</td>\n",
       "      <td>oncogenic mutations in the monomeric casitas b...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3316</th>\n",
       "      <td>3316</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>D171N</td>\n",
       "      <td>4</td>\n",
       "      <td>introduction myelodysplastic syndromes (mds) a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3317</th>\n",
       "      <td>3317</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>A122*</td>\n",
       "      <td>1</td>\n",
       "      <td>introduction myelodysplastic syndromes (mds) a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3318</th>\n",
       "      <td>3318</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>Fusions</td>\n",
       "      <td>1</td>\n",
       "      <td>the runt-related transcription factor 1 gene (...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3319</th>\n",
       "      <td>3319</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>R80C</td>\n",
       "      <td>4</td>\n",
       "      <td>the runx1/aml1 gene is the most frequent targe...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3320</th>\n",
       "      <td>3320</td>\n",
       "      <td>RUNX1</td>\n",
       "      <td>K83E</td>\n",
       "      <td>4</td>\n",
       "      <td>the most frequent mutations associated with le...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3316 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        ID    Gene             Variation  Class  \\\n",
       "0        0  FAM58A  Truncating Mutations      1   \n",
       "1        1     CBL                 W802*      2   \n",
       "2        2     CBL                 Q249E      2   \n",
       "3        3     CBL                 N454D      3   \n",
       "4        4     CBL                 L399V      4   \n",
       "...    ...     ...                   ...    ...   \n",
       "3316  3316   RUNX1                 D171N      4   \n",
       "3317  3317   RUNX1                 A122*      1   \n",
       "3318  3318   RUNX1               Fusions      1   \n",
       "3319  3319   RUNX1                  R80C      4   \n",
       "3320  3320   RUNX1                  K83E      4   \n",
       "\n",
       "                                                   Text  \n",
       "0     cyclin-dependent kinases (cdks) regulate a var...  \n",
       "1      abstract background non-small cell lung cance...  \n",
       "2      abstract background non-small cell lung cance...  \n",
       "3     recent evidence has demonstrated that acquired...  \n",
       "4     oncogenic mutations in the monomeric casitas b...  \n",
       "...                                                 ...  \n",
       "3316  introduction myelodysplastic syndromes (mds) a...  \n",
       "3317  introduction myelodysplastic syndromes (mds) a...  \n",
       "3318  the runt-related transcription factor 1 gene (...  \n",
       "3319  the runx1/aml1 gene is the most frequent targe...  \n",
       "3320  the most frequent mutations associated with le...  \n",
       "\n",
       "[3316 rows x 5 columns]"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "result.dropna()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "07dd97ba",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "a4473e5f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Text(0.5, 1.0, 'Comparing number of gene classes')"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAhAAAAIDCAYAAABVQie8AAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjQuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8rg+JYAAAACXBIWXMAAAsTAAALEwEAmpwYAABLGUlEQVR4nO3deXxN1/7/8XcSOYYMiCEqptCGmkPojSFKE0NV1VyCoC1VRagqvYqWNkSLG1Prmsca2iqipTRV0dsBrVKqWhGzVgyRmDLs3x9+OV+nCbI1ck7i9Xw8PG6ts846n3VOrvPO2mvv7WQYhiEAAAATnO1dAAAAyHsIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIE7CYpKUkLFixQx44dVb9+fdWtW1edO3fWqlWrlJ6ebu/y7psWLVqoV69e9i7DbvLC/A3D0JQpU/TYY4+pbt26Wr58ub1LylW9evVSixYt7F0GHFwBexeAB9ORI0c0cOBAnTx5Uu3atVOnTp1048YNbd26VWPHjtUPP/ygKVOmyMnJyd6l5rjXX39dhQsXtncZuIOvvvpK8+bN0+OPP67g4GDVr1/f3iUBDocAgVx3/fp1vfTSS7p48aLWrl2ratWqWR/r27ev3nzzTa1YsUK1a9dW79697Vjp/REcHGzvEnAXhw4dkiQNHz5cVatWtXM1gGPiEAZy3YoVKxQXF6fRo0fbhIcMr732mooWLaoPP/zQDtUBUkpKiiTJzc3NzpUAjosVCOS66OhoFSlSRG3bts3y8UKFCmn16tUqW7asTfuuXbs0c+ZM7d27V5JUq1YtDR48WA0aNLD2adGihR5//HE9+uijmjdvnk6fPq1HHnlE48aN00MPPaSJEyfq66+/lru7uzp06KDw8HA5O9/M0VWrVtXQoUPl4uKipUuXKjk5Wf7+/nr11Vf16KOPWl8jJSVFCxYsUHR0tOLj42UYhipVqqTevXurc+fO1n5Vq1bVwIED9euvvyo2NlYVK1bUp59+qpYtW8rHx0dLly611tykSRPVr19fc+fO1bFjx/TQQw8pLCxMoaGhNu/B9u3bNXPmTB0+fFglSpRQv379dODAAf3vf//Tl19+edv3vFevXrJYLAoLC9P06dN1+PBheXl5qXPnzho0aJD1PejVq5dOnjyZaay/t/fq1UtFihRRly5dFBUVpbi4OFWoUEEjR45U/fr1FRkZqc8++0wFChRQSEiIXn/9dRUqVMhmzDVr1uj999/Xn3/+qWrVqmnIkCFq2rSpTZ8ff/xRUVFR+umnnyRJ/v7+Cg8PV+3atW0+80aNGik9PV0bN25UsWLFtG7dOnl5eWX5Xtzt56hFixY6efKkJOmJJ56Qj4/PHd/b7H4mv//+u6ZNm6bvvvtOKSkpevTRRzVo0CCbOWf3c8rueLfzxx9/KCoqSt9++61SU1NVvXp1DR06VAEBAbd9zmeffably5fr4MGDun79ukqXLq3WrVsrPDxcFotFknTjxg1NmTJFX375pc6ePasSJUqoRYsWCg8PV9GiRSXd3F8ya9YsbdiwQadOnZKHh4caN26s4cOH66GHHrK+3qVLlxQVFaUtW7bowoULKl++vJ599ln17t3b5tDmypUrtWLFCh07dkyFChVSQECAwsPD9cgjj9z1fcA/48S9MJCbDMNQzZo1Va9ePesXaHZs27ZNL7/8sipUqKBOnTpJuvkFdOrUKUVFRemJJ56QdPMf/9TUVKWnpyssLEyGYWjOnDny9PSUh4eHHnnkEf3rX//Sli1bFBsbq0mTJqlDhw6Sbn7hly1bVklJSQoLC5Orq6sWL16sa9euae3atapcubIkacSIEfrss8/UvXt3Va1aVRcuXNDq1at1/PhxzZ07V82aNbOOV6RIEdWrV08hISG6du2a+vTpoxYtWmQKEIZhKDk5WT179lTJkiW1atUq/frrrzbjxcTEaNCgQfLz81OHDh109uxZLV26VEWKFJGbm9tdA8TRo0eVnJysbt26qVKlStq4caO+//57jR071hpUzASIuLg43bhxQ71795aHh4c++OADXb16VY8++qgKFy6s4OBg7dq1Sxs3btTgwYP18ssvW+d74cIFpaenq3fv3vLy8tKHH36o48ePa968eWrUqJEkaefOnRowYICqVaump556Sjdu3NDHH3+skydPauHChdYvu4zxKleurC5duujcuXPW17qXn6OtW7dq3bp1+uKLLzR69GiVK1futoedsvuZHDp0SD169FDJkiXVuXNnubq6auPGjfrll1/03nvv6cknnzT1OWV3vKwcPXpUnTp1UoECBdSzZ0/r+3/06FEtX75ctWvXzvR5r1mzRmPGjFGLFi0UFBSklJQUffHFF/r+++/13HPPaeTIkZKkMWPGaOPGjerdu7fKly+vw4cPa/ny5Xrssce0YMECSdKcOXMUFRWl0NBQVa1aVSdOnNCSJUtUpkwZbdy4US4uLrpy5Yq6deum06dPq0ePHipTpoy+/fZbbd68WT169NC4ceMkSevXr9err76qZ555RvXr19f58+e1ePFiGYahL774Qh4eHrd9H5ADDCAXJSQkGH5+fsawYcOy/ZyUlBQjKCjIaNasmXH58mVr+6VLl4ymTZsaTZs2NW7cuGEYhmE0b97cqFq1qvHrr79a+02ePNnw8/MzwsPDrW3JyclGjRo1jOHDh1vb/Pz8jGrVqhn79++3tv3+++9G9erVrfX++eefRtWqVY13333XpsY//vjD8PPzMyZMmGAzXkBAgHH16lWbvs2bNzd69uxp8/eqVasaBw8etLZlvM6t9QUHBxstW7a0Ge+LL74w/Pz8jObNm9/xPezZs6fh5+dnbNu2zdp27do1o0GDBka3bt1s+mU11t/bM8b78ssvrW3Lli0z/Pz8jK5du1rb0tPTjaCgIJvXaN68ueHn52d89dVX1rYLFy4YDRs2NDp06GAYhmGkpaUZTzzxhPHss88aqamp1n7JyclGSEiI0b59e5vxqlWrZpw5c+aO74GZn6OoqCjDz8/POH78+B3HzO5n0rNnTyM4ONhITk62qadHjx5Go0aNjOvXr1v7Zfdzys54WRk6dKhRu3Zt4+jRo9a28+fPG/Xr1zeGDBliHf/W+lu3bm1069bNSE9Pt3m9oKAg46mnnrK21a5d23jzzTdtXm/atGlGx44djaSkJMMwDKNNmzZG//79bfqsXLnSePrpp434+HjDMG6+/zVq1LD5/7FhGMZ7771n+Pn5Wf+/8vzzzxtt27a16fPVV18ZTz75pLFr167bvgfIGeyBQK7KWIJNS0vL9nMOHDigM2fOKDQ0VO7u7tZ2T09P9ezZU2fPntX+/fut7RUqVLDZ+Obr6ytJCgkJsbYVKVJEJUqU0F9//WXzWo0bN1aNGjWsf69SpYqaNm2qr776Sunp6SpVqpR2796tl156ydrHMAylpqZKkpKTk23Gq127dqal+6z4+vra7AcpVaqUSpYsqXPnzkmSfv31Vx07dkzPPvuszXjBwcHWlZG7KVy4sB5//HHr3wsWLChfX1/ra5hVsGBBm+XyjPc5YzVIkpycnOTj45Ppffbz87OurEhSsWLF1K5dO/3yyy/666+/dODAAR0/flzBwcG6dOmSzp8/r/Pnz+vatWtq3ry5Dh48qLNnz1qfX6FCBXl7e9+xXrM/R3eT3c/kwoUL+v7779WsWTNdu3bNOpfExESFhITo3Llz2rdvn7X/3T4ns+PdKj09Xdu3b1ezZs1UsWJFa3vx4sW1YsUKjRkzJsvnrV+/XnPnzrU5dJCQkCBPT09duXLF2lamTBlt2rRJH3/8sRITEyVJ4eHh+uijj6z7ScqUKaPvvvtOixcvts7p2Wef1aeffqoKFSpIkrZs2SI/Pz+VKlXKOr/z589bV4JiYmKsYx05ckQzZ87UiRMnJEnNmjVTdHQ0Z87kAvZAIFcVLVpUrq6uOn/+fLafk/EPQ8YX1K0y/qE+deqU/P39JUklSpSw6ePi4iJJmY6Ju7i4yPjbEbyHH34402tUqlRJMTExunjxory8vGSxWLR+/XrFxsbq6NGjio+PtwaHv493u+Pwf5dVP4vFYr0eRnx8vCTZ/KOfoXLlyjp48OBdX6NYsWI2x9D//hpmFStWTAUK/N8/IRnvc1bv/9/fl6w+y4wvj5MnT+rUqVOSpMjISEVGRmb5+qdOnbKGhr+/ZlbM/hzdTXY/k+PHj0uSli5detvDdqdPn7b+990+J7Pj3erixYu6cuVKljX7+fll+RxJcnV11Q8//KCNGzfqyJEjOnbsmBISEiRJPj4+1n7jx49XeHi4Ro8erTfeeEN169ZVSEiIOnXqZD2cMHLkSA0cOFDvvPOOIiIiVKNGDbVo0UJdu3ZVqVKlJEnHjh3TtWvXFBgYeMf5DRo0SD/99JNmzJihGTNm6OGHH1aLFi3UpUsX688T7h8CBHKVk5OT/P39tX//fqWmptp8Ad1q2rRpOn78uEaPHp3py+dWGY+5urpa2243ZnauKXHrOBkyVkucnZ11/fp19ejRQwcPHtRjjz2mwMBA9enTRw0bNrT5rTFDxpfq3fz9C+PvMlY4Mjar3apgwYI58hp3ktWK0T95n7OS8Vk6OztbvyyHDh2qunXrZtn/1t/ys/M+m/05upvsfiYZ711oaOht91LcGlzv9jmZHS+r55r9jCZMmKBly5apevXqqlu3rtq3by9/f39NmDDBJqwEBgYqJibG+mfnzp2KiIjQokWL9PHHH8vLy0vVqlXT5s2btWPHDsXExGjHjh2KiorSwoULtWrVKlWpUkVpaWmqX7/+bfeylC5dWtLNFYhPP/1U3333nbZt26YdO3Zo7ty5WrhwoRYsWKCGDRuamifMIUAg14WEhOj7779XdHS02rdvn+nxjE2LaWlpKlasmPU3nCNHjmTqGxcXJ+nmPyQ54dixY5na4uPjVaxYMevu/v379+vtt9+2OePi1uX0+6F8+fKSbm6Aa9Kkic1jR48ezbHXcXZ21o0bNzK13+thjtvJOMvhVhnzKF++vPWLrkiRItZNlRl+/vlnXbp0KVuHhm6V0z9H2f1MMl7XxcUl01x+//13nThxwtSFxf7JeMWLF1ehQoWy/DmfP3++/vrrL40aNcqm/eTJk1q2bJnat2+faTXo1p+LGzdu6ODBgypTpozatm2rtm3bKj09XQsXLlRkZKSio6PVo0cP/frrr3J3d9cTTzxhPdy1adMmDRs2TGvWrNGoUaPk4+Oj5OTkTPO7dOmS/ve//1lXUDKu1xEYGGhdrdi9e7fCwsK0dOlSAsR9xh4I5Lpu3brJx8dHkZGR+u2332weS0tL0/jx43Xu3Dm98MILcnV1VY0aNVSqVCmtXLlSSUlJ1r5JSUlasWKFSpUqpZo1a+ZIbV9++aXNl9tvv/2m2NhYtWzZUtLNJWAp8294S5YskfR/v5XmtJo1a+qhhx7S2rVrbb7gf/rpJx04cCDHXqdkyZJKSEiwCUT79++3LtfnlF9++cWm7nPnzmn9+vUKCAhQ8eLFVbNmTZUqVcp6Om2GpKQk6xJ5dld3MuT0z1F2P5PSpUurZs2a+uSTT2ze15SUFL3++usaMmSIqZ+bfzJegQIF1LhxY23fvt1m5eDSpUuaP3++9fDIrS5duiQp88/89u3bdfToUetrXbhwQd26ddMHH3xg7ePs7KxatWpZ/zstLU29e/fWO++8YzNWnTp1rH2km2fW/Prrr9q+fbtNvzlz5mjo0KE6fPiwpJsrVCNHjrRZIatevbpcXV3/0YobsocVCOS6ggULaubMmerXr586d+6sdu3aqVatWrp48aI+//xzHTx4UK1bt1bfvn0l3VxWHjNmjIYNG6ZOnTpZf/Nfu3at/vzzT0VFReXYPxZOTk7q3r27evXqpZSUFC1evFheXl4aPHiwJKlRo0YqUKCARo4cqdDQUBUoUEAxMTGKjY2Vq6trpk2UOcXZ2VmjRo1SeHi4nn32WbVv317nz5/XkiVLslxCv1dPPfWUNm7cqBdeeEHdu3dXQkKCli5dqkqVKlkvrpQTihYtqueee059+/aVi4uLli9frtTUVI0ePVqS7WfesWNHde7cWQULFrSecvnuu+/e9hDK7eT0z5GZz2TMmDEKCwtTp06d1L17dxUrVkzR0dHau3evXnnlFRUvXtzUXP7JeK+88oq6dOmiLl26WDeUrl69WleuXFF4eHim/g8//LDKli2r999/X9evX1eZMmX0888/65NPPlHBggWtP/Pe3t5q166dVqxYoatXr8rf318XL17UsmXLVLJkSbVp00YWi0W9evXSnDlzrNesuHbtmlatWqXChQtbT60dMGCAtmzZokGDBunZZ5/VI488ot27d+vTTz9VUFCQgoKCJEnPPfecxowZoz59+qh169YyDEOffvqp9VAj7i8CBOyievXq+vTTT7Vo0SJ9/fXX2rRpkwzDUNWqVfXOO++oY8eONsdpW7duraJFi2r27NmaNWuWChQooDp16ujtt9++48VvzGrTpo3Kly+vefPmKT09XY0bN9arr75qPebq5+enqKgozZw5U1OnTpWbm5seeeQRLVy4UCtWrND333+vlJQUU8fSs6t169aaNm2a5syZoylTpsjb21ujR4/WunXrTG1KvZPmzZtr7NixWrJkid5++235+vpq/Pjx+uGHH/TVV1/lyGtIUtOmTVWrVi3Nnz9fFy9eVJ06dTR9+nSbFYCMz3zOnDmaPXu2nJ2d9cgjj2jOnDlq3rz5Pb1uTv8cZfcz8ff318qVKzVjxgwtXLhQqamp8vX1tbkOiRn/ZLwqVapo1apVmjp1qubNmydnZ2fVrl1bkydPzvLiSxaLRXPnztWkSZO0ZMkSGYahChUq6PXXX1dqaqrefvtt7d+/XzVr1tSECRNUvnx5RUdHKzo6WoULF1ZgYKCGDRtm3Sg8ZMgQFStWTB999JEmT54sFxcX1atXT1OmTFGVKlUk3dxIumrVKkVFRenzzz/XqlWrVLZsWb300kvq37+/Neh16dJFrq6uWrJkiaZOnar09HTVrFlT//3vf/XYY4+Zfl9hjsNcSOrgwYPq3Lmztm3bZnMcMjY2VtOmTdPvv/+uEiVKqGfPnurXr5/Nc/ft26fIyEjt379fbm5u6tixowYPHmzzj/jRo0c1adIk7dq1Sy4uLmrdurVeffVVm9O58GCrWrWqOnTooEmTJtm7lEzS0tJ06dKlLM/WaNeunTw9PR+4O0baG58JHnQOcZDojz/+0IABAzIdt9uzZ49efPFFVa5cWTNmzFC7du0UGRmp+fPnW/vEx8erT58+KliwoKZPn65+/fpp4cKFioiIsPa5dOmSwsLCdO7cOU2ePFmvvPKKNm3apFdeeSXX5gj8E2lpaQoKCtLYsWNt2g8dOqTDhw/bXNoZuYPPBA86ux7CSE1N1apVq/Tee+9lueQbFRWl6tWra8qUKZKkoKAgpaam6v3337deM37u3Lny8PDQ7NmzZbFY1KxZMxUqVEgTJ07UgAED5O3treXLlysxMVHr1q2zHhv09vZW//79tXfvXusGHsBRWSwWtW7dWmvXrpWTk5Nq1qypP//8UytXrlTx4sWt+0WQe/hM8KCz6wrE7t279e6776pfv34aMWKEzWPXr1/Xrl27rLvfM7Rq1UqJiYnas2ePpJvXy2/evLnNpqXWrVsrLS1NsbGx1j4NGjSw2VjUpEkTubm5ZdrlCziqiRMnatCgQfr22281YcIErVixQv/617+0Zs0a6x4N5C4+EzzI7LoCUaVKFW3dulUlSpTQxx9/bPPY8ePHlZKSkumqcRnn/8bFxalOnTo6ffp0pj5eXl5yd3e3ntt95MgRPf300zZ9XFxcVK5cOWufu7l27Zr279+vUqVKmT59DHnDtm3bJP3fFQsdUfv27bO8doYj15zf8ZkgP0tLS9Nff/2lmjVrZrr2il0DRMmSJW/72OXLlyUp0ybHjOupJyUl3bZPRr+Mc70vX7581z63SkxMtF7HPcO+ffuyPMUJAID8bvny5ZnOVHLY0zjvdnKIs7NztvrcTVZ9Fi9erJkzZ2bZf/ny5Tl21UMAABxZxg3oMu5TciuHDRAZN175+4V5MlYMPDw8rKsKWV28JykpyTqGu7v7bfuULVs2U3tYWFimc6kz3sQyZcqoXLly9zAjAADypqwO3TtsgKhQoYJcXFwyXbM94+++vr5yc3OTt7d3psvsJiQkKDk52bo3wtfXN1OftLQ0nThxQq1atcr02p6envL09MzJ6QAAkK84xHUgslKwYEEFBARoy5YtNocqNm/eLA8PD+sV6xo3bqyYmBiba9Fv3rxZLi4u1hupNG7cWN999531PgbSzQtUXblyJdPNWgAAwN05bICQpIEDB2rPnj0aNmyYtm/frunTp2v+/PkaMGCA9W5zzz//vP766y/1799fMTEx1otIde3a1Xp4okePHrJYLOrTp4+++OILrVmzRq+++qqCgoJUr149e04RAIA8yaEDRGBgoGbMmKE//vhDgwYN0oYNGzRy5Ei98MIL1j5VqlTRggULdOXKFQ0ZMkQLFy5U37599e9//9vax8vLS0uWLFGxYsU0YsQITZs2zXoNewAAYJ7D3AvD0Z04cUJPPPGEtm3bxiZKAMAD4U7ffQ69AgEAABwTAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAHmDXUtLyxWsg9xWwdwEAAPsp5OqiSqOi7+trHJ3U9r6OD/tgBQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpeSJArFy5Um3atFHdunXVrl07rV+/3ubx2NhYderUSXXq1FGLFi20YMGCTGPs27dPvXr1kr+/v5o0aaKpU6cqJSUlt6YAAEC+4vABYtWqVRo/frwef/xxzZ49W40aNdKrr76qzz77TJK0Z88evfjii6pcubJmzJihdu3aKTIyUvPnz7eOER8frz59+qhgwYKaPn26+vXrp4ULFyoiIsJe0wIAIE8rYO8C7uaTTz7RY489ptdee02S1KhRI+3fv18rVqxQmzZtFBUVperVq2vKlCmSpKCgIKWmpur9999Xr169ZLFYNHfuXHl4eGj27NmyWCxq1qyZChUqpIkTJ2rAgAHy9va25xQBAMhzHH4F4vr163Jzc7NpK1asmC5evKjr169r165datmypc3jrVq1UmJiovbs2SNJ2rlzp5o3by6LxWLt07p1a6WlpSk2Nvb+TwIAgHzG4QNE7969tWPHDn322WdKSkrS559/rq+++krt27fX8ePHlZKSIl9fX5vnVKxYUZIUFxenq1ev6vTp05n6eHl5yd3dXXFxcbk2FwAA8guHP4TRtm1bffvttwoPD7e2dejQQc8//7x+/PFHSZK7u7vNczJWLJKSknT58uUs+2T0S0pKytSemJioxMREm7YzZ878o3kAAJCfOHyAGDhwoH788UeNHj1a1atX1969ezV79my5u7vrySefvONznZ2dZRjGXfv83eLFizVz5sx/VDcAAPmZQweIPXv2KDY2VhEREerYsaMkqWHDhvL09NTYsWPVuXNnSVJycrLN8zJWFTw8PKwrD3/vk9HPw8MjU3tYWJg6dOhg03bmzBmFhob+80kBAJAPOHSAOHXqlCSpXr16Nu0BAQGSpIMHD8rFxUXHjh2zeTzj776+vnJzc5O3t7fi4+Nt+iQkJCg5OTnT3ghJ8vT0lKenZ47NAwCA/MahN1FmfLnv3r3bpv2nn36SJFWuXFkBAQHasmWLzaGKzZs3y8PDQzVr1pQkNW7cWDExMbpx44ZNHxcXFzVs2PA+zwIAgPzHoVcgatSooeDgYL399tu6fPmyHn30Ue3fv1+zZs1SUFCQ6tSpo4EDB6pv374aNmyYOnTooB9//FHz58/XK6+8osKFC0uSnn/+eUVHR6t///4KCwvT0aNHNXXqVHXt2lVly5a18ywBAMh7HDpASNK0adM0c+ZMLVq0SAkJCfLx8VG/fv3Uv39/SVJgYKBmzJihqKgoDRo0SN7e3ho5cqT69etnHaNKlSpasGCBIiMjNWTIEBUvXlx9+/bV4MGD7TUtAADyNIcPEBaLRcOHD9fw4cNv2yckJEQhISF3HCcgIECrV6/O6fIAAHggOfQeCAAA4JgIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEkIdcS0nLF68BIO8rYO8CAGRfIVcXVRoVfV9f4+iktvd1fAD5AysQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADAtTwSIH374Qd27d1edOnXUpEkTTZgwQcnJydbHY2Nj1alTJ9WpU0ctWrTQggULMo2xb98+9erVS/7+/mrSpImmTp2qlJSU3JwGAAD5hsMHiJ9++kl9+/ZVqVKlNGfOHA0aNEjr16/XmDFjJEl79uzRiy++qMqVK2vGjBlq166dIiMjNX/+fOsY8fHx6tOnjwoWLKjp06erX79+WrhwoSIiIuw1LQAA8rQC9i7gbt59913VrVtX//nPf+Tk5KRGjRopPT1dCxcu1NWrVxUVFaXq1atrypQpkqSgoCClpqbq/fffV69evWSxWDR37lx5eHho9uzZslgsatasmQoVKqSJEydqwIAB8vb2tvMsAQDIWxx6BeL8+fPatWuXunfvLicnJ2t7aGiotm7dKmdnZ+3atUstW7a0eV6rVq2UmJioPXv2SJJ27typ5s2by2KxWPu0bt1aaWlpio2NzZ3JAACQjzh0gPjtt99kGIaKFi2q8PBw1a1bV/Xr19e4ceN07do1HT9+XCkpKfL19bV5XsWKFSVJcXFxunr1qk6fPp2pj5eXl9zd3RUXF5dr8wEAIL9w6EMY58+flySNGjVKISEhmjNnjg4dOqTp06fr+vXr6tatmyTJ3d3d5nlubm6SpKSkJF2+fDnLPhn9kpKSMrUnJiYqMTHRpu3MmTP/fEIAAOQTDh0gMs6SqFevnsaNGydJCgwMlGEYmjx5srp27XrH5zs7O8swjLv2+bvFixdr5syZ91g1AAD5n0MHiIyVhKCgIJv2Jk2aaNKkSdq3b58k2ZzSKcm6quDh4WFdefh7n4x+Hh4emdrDwsLUoUMHm7YzZ84oNDT0HmcCAED+4tABolKlSpKkGzdu2LRnrEyUK1dOLi4uOnbsmM3jGX/39fWVm5ubvL29FR8fb9MnISFBycnJmfZGSJKnp6c8PT1zahoAAOQ7Dr2JskqVKvLx8dGmTZts2mNiYlSgQAH5+/srICBAW7ZssTlUsXnzZnl4eKhmzZqSpMaNGysmJsYmiGzevFkuLi5q2LBh7kwGAIB8xKEDhJOTk0aMGKFdu3ZpxIgR+uabbzR37lzNmTNHPXv2lJeXlwYOHKg9e/Zo2LBh2r59u6ZPn6758+drwIABKly4sCTp+eef119//aX+/fsrJibGehGprl27qmzZsnaeJQAAeY9DH8KQpCeffFIWi0WzZs3SgAEDVKJECQ0aNEgDBgyQdHNT5YwZMxQVFaVBgwbJ29tbI0eOVL9+/axjVKlSRQsWLFBkZKSGDBmi4sWLq2/fvho8eLC9pgUAQJ7m8AFCkoKDgxUcHHzbx0NCQhQSEnLHMQICArR69eqcLg0AgAeSQx/CAAAAjokAAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0DcZ9dS0vL0+AAAZCVPXAciLyvk6qJKo6Lv2/hHJ7W9b2MDAHA7rEAAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMM1UgOjdu7f+97//3fbxL7/8Um3bcl0CAADyuzteSOrq1au6cOGC9e/ff/+9QkJCVLFixUx909PT9fXXX+vEiRM5XyUAAHAodw0QzzzzjC5fvixJcnJy0jvvvKN33nkny/6GYahx48Y5XyUAAHAodwwQXl5emjJlivbt2yfDMDRr1iyFhISoatWqmfo6OzvLy8uLQxgAADwA7novjGbNmqlZs2aSpFOnTunZZ59VnTp17nthAADAcZm6mVZERMT9qgMAAOQhpu/G+fXXX2vDhg06d+6c0tIy30rayclJixcvzpHiAACAYzIVIJYvX66JEydKkkqUKCGLxXJfigIAAI7NVIBYsmSJqlWrpv/+978qWbLk/aoJAAA4OFMXkjp9+rS6detGeAAA4AFnKkBUqFBB586du1+1AACAPMJUgOjfv7+WLl2qw4cP3696AABAHmBqD8Tu3bvl5uam9u3by9fXV15eXnJycrLpw1kYAADkf6YCxI4dOyRJZcqU0dWrV3Xy5Mn7UhQAAHBspgLEl19+eb/qAAAAeYipPRAAAACSyRWI3r17Z6vfkiVL7qkYAACQN5gKECdOnMjUlp6ergsXLuj69evy8fHRI488kmPFAQAAx5QjeyDS0tK0bds2jRkzRs8991yOFAYAABxXjuyBcHFxUcuWLdWlSxe9++67OTEkAABwYDm6ibJSpUr69ddfc3JIAADggHIsQNy4cUPr169XiRIlcmpIAADgoHLkLIwbN24oLi5OiYmJGjx4cI4UBgAAHNc/PgtDurkHonLlynrqqafUo0ePHCkMAAA4Lq5ECQAATDMVIDKkpaVp//79OnnypCwWix566CHVqFEjp2sDAAAOynSAiImJ0ZtvvqmzZ8/KMAxJN+/AWbp0aY0bN04tWrTI8SIBAIBjMRUgdu3apcGDB6tEiRIaNmyYqlSpIsMwdOTIEa1YsUJDhgzRkiVLVK9evftVLwAAcACmAsSMGTPk4+OjtWvXysPDw+axHj16qFOnTpozZ47++9//5miRAADAsZi6DsTPP/+sLl26ZAoPkuTu7q7OnTtr7969OVYcAABwTDl6JUonJyelpKTk5JAAAMABmQoQderU0dq1a3XlypVMjyUlJWnNmjWqVatWjhUHAAAck6k9EC+//LJ69+6tp556Sj179lSlSpUkybqJ8uzZs3rzzTfvR50AAMCBmAoQAQEBmjFjht566y1FRkbKycnJeipnqVKlNHXqVP3rX/+6L4UCAADHYfo6EE888YQef/xx/fLLL9ZLW3t7e6tOnToqUOCerksFAADymGztgVi2bJnatWun1NRUSTfvfVG7dm09+eST2rZtmwYPHqxly5bd10IBAIDjuGOAMAxDI0eO1MSJE/Xnn3/q1KlTmfqUK1dOzs7Omjx5soYPH37fCgUAAI7jjgFizZo1Wr9+vXr06KGvv/5aFSpUyNRn2LBh2rZtm9q3b6/PPvtM69atu1+1AgAAB3HXANGgQQONHTtWBQsWvG2/ggUL6p133lG1atX04Ycf5niRAADAsdwxQPz+++964oknsjeQs7NatWqlQ4cO5UhhAADAcd0xQLi4uMhisWR7sOLFi8vZOUcvbgkAABzQHb/tK1asqP3792d7sH379qls2bL/uCgAAODY7hgg2rZtqw0bNujw4cN3Hejw4cPasGGDgoKCcqw4OIZrKWn54jUAADnnjld+6tatm1atWqVevXrp9ddfV9u2beXi4mLTJz09XdHR0Zo8ebLc3NwUFhZ2XwtG7ivk6qJKo6Lv62scndT2vo4PAMhZdwwQbm5umjNnjl566SW99tprevPNN1WjRg2VKlVK6enpSkhI0C+//KIrV67ooYce0qxZs1S6dOncqh0AANjJXa89XblyZa1fv17Lly9XdHS09uzZY70ipaurq+rWrauWLVuqW7dupjZcAgCAvCtbN6+wWCzq27ev+vbtK0k6f/68XFxcVLRo0ftaHAAAcEz3dPcrLy+vnK4DAADkIVy0AQAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGl5LkC8/PLLCgkJsWmLjY1Vp06dVKdOHbVo0UILFizI9Lx9+/apV69e8vf3V5MmTTR16lSlpKTkVtkAAOQreSpAfPrpp/riiy9s2vbs2aMXX3xRlStX1owZM9SuXTtFRkZq/vz51j7x8fHq06ePChYsqOnTp6tfv35auHChIiIicnsKAADkC/d0N057OHv2rN5++22VKVPGpj0qKkrVq1fXlClTJElBQUFKTU3V+++/r169eslisWju3Lny8PDQ7NmzZbFY1KxZMxUqVEgTJ07UgAED5O3tbY8pAQCQZ+WZFYgxY8aocePGCgwMtLZdv35du3btUsuWLW36tmrVSomJidqzZ48kaefOnWrevLksFou1T+vWrZWWlqbY2NjcmQAAAPlInggQa9as0S+//KI33njDpv348eNKSUmRr6+vTXvFihUlSXFxcbp69apOnz6dqY+Xl5fc3d0VFxeX6fUSExN14sQJmz9nzpzJ4VkBAJB3OfwhjJMnTyoiIkIRERHy8vKyeezy5cuSJHd3d5t2Nzc3SVJSUtJt+2T0S0pKytS+ePFizZw5M0fqBwAgP3LoAGEYhl5//XU1a9ZMrVq1yvLxO3F2ds5Wn78LCwtThw4dbNrOnDmj0NDQbFQNAED+59ABYvny5Tp06JA2bNig1NRUSf8XGlJTU+Xh4SFJSk5OtnlexqqCh4eHdeXh730y+mWMcStPT095enrm3EQAAMhnHDpAbN68WRcuXFCTJk0yPVajRg2NHz9eLi4uOnbsmM1jGX/39fWVm5ubvL29FR8fb9MnISFBycnJmfZGAACAu3PoAPHmm29mWjmYNWuWDh48qJkzZ6pcuXL67LPPtGXLFoWFhcnJyUnSzeDh4eGhmjVrSpIaN26smJgYjRw50nomxubNm+Xi4qKGDRvm7qQAAMgHHDpAVK5cOVNbsWLFZLFYVKtWLUnSwIED1bdvXw0bNkwdOnTQjz/+qPnz5+uVV15R4cKFJUnPP/+8oqOj1b9/f4WFheno0aOaOnWqunbtqrJly+bqnAAAyA/yxGmcdxIYGKgZM2bojz/+0KBBg7RhwwaNHDlSL7zwgrVPlSpVtGDBAl25ckVDhgzRwoUL1bdvX/373/+2Y+UAAORdDr0CkZVJkyZlagsJCcl0f4y/CwgI0OrVq+9XWQAAPFDy/AoEAADIfQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAIB7dC0lLU+PD/wTBexdAADkVYVcXVRpVPR9G//opLb3bWzgn2IFAgAAmEaAAAAAphEgAACAaQQIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAkCuy427THInS+D+4m6cAHLd/b6LpcSdLIH7jRUIAABgGgECAACYRoAAAACmESAAAIBpBAgAAGAaAQIAAJhGgAAAAKYRIAAAgGkECAAAYBoBAgAAmEaAAAAApjl8gEhPT9fKlSvVrl07+fv7Kzg4WBEREUpKSrL22bdvn3r16iV/f381adJEU6dOVUpKis04R48e1YsvvqiAgAA99thjGjdunM0YAAAg+xz+Zlrz5s3T9OnT9dxzzykwMFBxcXGKiorS77//rvnz5ys+Pl59+vSRv7+/pk+frj/++EPTpk1TUlKSxo4dK0m6dOmSwsLCVKpUKU2ePFkJCQmaMmWKzpw5ow8++MDOMwQAIO9x6ABhGIbmzZunbt266ZVXXpEkNWrUSMWLF9ewYcN08OBBLVu2TB4eHpo9e7YsFouaNWumQoUKaeLEiRowYIC8vb21fPlyJSYmat26dSpevLgkydvbW/3799fevXtVp04de04TAIA8x6EPYSQnJ+vpp5/WU089ZdNeuXJlSdKxY8e0c+dONW/eXBaLxfp469atlZaWptjYWEnSzp071aBBA2t4kKQmTZrIzc1N27dvz4WZAACQvzj0CoS7u7vGjBmTqX3r1q2SpCpVquj06dPy9fW1edzLy0vu7u6Ki4uTJB05ckRPP/20TR8XFxeVK1fO2udWiYmJSkxMtGk7c+bMP5oLAAD5iUMHiKzs3btXc+fOVXBwsDw9PSXdDBp/5+bmZt0kefny5bv2udXixYs1c+bMHK4cAID8I08FiN27d+vFF19UuXLlNHHiRN24ceOO/Z2d736EJqs+YWFh6tChg03bmTNnFBoaaq5gAADyqTwTIDZt2qRRo0apUqVKmjdvnooXL67k5GRJsv7vrZKSkuTh4SHp5grF7fqULVs2U7unp6d1dQMAAGTm0JsoMyxcuFDDhw9X3bp1tXz5cpUuXVrSzUMQ3t7eio+Pt+mfkJCg5ORk694IX1/fTH3S0tJ04sSJTPsnAADA3Tl8gFizZo0mTZqkNm3aaN68edZVhQyNGzdWTEyMzeGMzZs3y8XFRQ0bNrT2+e6773Tx4kVrn9jYWF25ckWNGjXKlXkAAJCfOPQhjISEBL399tvy8fFRaGioDhw4YPN4hQoV9Pzzzys6Olr9+/dXWFiYjh49qqlTp6pr167WwxM9evTQsmXL1KdPHw0aNEgXL17UlClTFBQUpHr16tljagAA5GkOHSB27Nihq1ev6uTJk1luYIyMjFT79u21YMECRUZGasiQISpevLj69u2rwYMHW/t5eXlpyZIleueddzRixAi5ubmpdevWGjlyZG5OBwCAfMOhA8QzzzyjZ5555q79AgICtHr16jv28fPz06JFi3KmMAAAHnAOvwcCAAA4HgIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQeGNdS0vLFawCAIyhg7wKA3FLI1UWVRkXf19c4OqntfR0fABwFKxAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAAA4gGspaXnqNQrk2EgAAOCeFXJ1UaVR0ff1NY5OaptjY7ECAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABMI0AAAADTCBAAgDwvr11DIT/gOhAAgDwvr11DIT9gBQIAAJj2QAWIjRs3qm3btqpdu7batGmjdevW2bskAADypAcmQGzatEkjRoxQkyZNNGvWLDVs2FCvvfaaPv/8c3uXBgBAnvPA7IGYNm2a2rRpo9GjR0uSmjZtqkuXLuk///mPWrdubefqAADIWx6IFYjjx4/r2LFjatmypU17q1atdOTIER0/ftxOlQEAkDc9ECsQR44ckST5+vratFesWFGSFBcXp/Lly1vbExMTlZiYaNP35MmTkqQzZ86YLyD5vPnnZNOJEyfu29g27uMcJOZhSn6Yg8Q8siE/zEFiHqY42BwyvvPS0jKfwupkGIaRI1U5sI0bN+qVV17Rtm3bVK5cOWt7fHy8WrZsqWnTpunJJ5+0ts+YMUMzZ860R6kAADic5cuXKyAgwKbtgViBuFtGcna2PZITFhamDh062LTduHFDx48fV6VKleTi4pLjNUo3k15oaKiWL1+uMmXK3JfXyA35YR75YQ5S/phHfpiDxDwcSX6Yg5Q780hLS9Nff/2lmjVrZnrsgQgQHh4ekqTk5GSb9qSkJJvHM3h6esrT0zPTOJUrV75PFdoqU6aMzUpJXpUf5pEf5iDlj3nkhzlIzMOR5Ic5SPd/HhmH+//ugdhEmbH34dixYzbt8fHxNo8DAIDseSACRMWKFVWuXLlM13zYsmWLKlWqpLJly9qpMgAA8qYH4hCGJA0aNEijR49W0aJF9fjjj2vbtm367LPPNG3aNHuXBgBAnvPABIiOHTvqxo0bWrBggdasWaPy5ctr8uTJNmdf2Junp6defvnlLPdf5CX5YR75YQ5S/phHfpiDxDwcSX6Yg2T/eTwQp3ECAICc9UDsgQAAADmLAAEAAEwjQAAAANMIEA7m4MGDqlGjxr3dc8PO0tPTtXLlSrVr107+/v4KDg5WRESE9YJdeYFhGFq0aJFatWql2rVr6+mnn9aGDRvsXdY/8vLLLyskJMTeZdyT1NRU1a5dW1WrVrX54+/vb+/STPnhhx/UvXt31alTR02aNNGECRMyXdjOkX333XeZPoNb/3zyySf2LjHbVq5cqTZt2qhu3bpq166d1q9fb++STLt27ZomT56sJk2aqE6dOurWrZu2b9+e63U8MGdh5AV//PGHBgwYoNTUVHuXck/mzZun6dOn67nnnlNgYKDi4uIUFRWl33//XfPnz7d3ednywQcfKCoqSoMHD1bdunX19ddfa8SIEXJxcXGoM3ay69NPP9UXX3yhChUq2LuUexIXF6fr169r8uTJqlSpkrX975efd2Q//fST+vbtqxYtWmjOnDmKj4/X1KlTdf78+TxzGnmNGjW0atUqmzbDMPTvf/9bV65cUbNmzexUmTmrVq3S+PHj1a9fPzVt2lTbt2/Xq6++KldXV7Vp08be5WXb0KFDtXPnTvXv318BAQHatWuXXn75Zb377rtq1apV7hViwO5SUlKMZcuWGf7+/kbDhg0NPz8/4/Tp0/Yuy5T09HSjQYMGxvjx423ao6OjDT8/P+PAgQN2qiz7bty4YTRo0MB46623bNp79uxpdO/e3U5V3bszZ84YDRo0MIKCgozg4GB7l3NP1q9fb1SrVs24cuWKvUu5Z6GhoUZoaKiRnp5ubVu2bJnxxBNP5Ol5LVq0yKhWrZrx008/2buUbOvWrZvRq1cvm7YePXoYPXv2tFNF5u3fv9/w8/Mz/vvf/9q0R0ZGGkFBQUZaWlqu1ZJ3Ynw+tnv3br377rvq16+fRowYYe9y7klycrKefvppPfXUUzbtGfcP+ftlxB2Ri4uLli5dqv79+9u0u7q66vr163aq6t6NGTNGjRs3VmBgoL1LuWcHDx5UhQoVVLhwYXuXck/Onz+vXbt2qXv37nJycrK2h4aGauvWrXl2Xn/99Zf+85//WA/L5BXXr1+Xm5ubTVuxYsV08eJF+xR0D+Li4iRJzZs3t2lv0KCBzpw5o0OHDuVaLQQIB1ClShVt3bpVL7/88n270+f95u7urjFjxqh+/fo27Vu3bpUkPfzww/YoyxRnZ2dVrVpV3t7eMgxD586d09y5c/XNN9+oW7du9i7PlDVr1uiXX37RG2+8Ye9S/pFDhw7JYrHoueeek7+/vxo0aKCxY8fmmX01v/32mwzDUNGiRRUeHq66deuqfv36GjdunK5du2bv8u7ZjBkz5OzsrPDwcHuXYkrv3r21Y8cOffbZZ0pKStLnn3+ur776Su3bt7d3adn20EMPSZJOnjxp0378+HGb/80N7IFwACVLlrR3CffF3r17NXfuXAUHB6tKlSr2LseULVu2aMiQIZKkxx9/XE8//bSdK8q+kydPKiIiQhEREfLy8rJ3Of/Ir7/+qqSkJHXp0kUvvvii9u/frxkzZiguLk5Lliyx+a3eEZ0/f16SNGrUKIWEhGjOnDk6dOiQpk+fruvXr2vSpEl2rtC8hIQErVu3Tv369ctzV3Js27atvv32W5vg06FDBz3//PP2K8qkWrVq6eGHH9aECRP0zjvv6NFHH9WePXus+8yuXLmSa7UQIHBf7N69Wy+++KLKlSuniRMn2rsc06pXr65ly5bp0KFD+s9//qP+/ftr8eLFDv+FZRiGXn/9dTVr1ix3N1PdJ9OmTVPRokVVtWpVSTeXaUuUKKFXX31V33zzjRo3bmznCu8sJSVFklSvXj2NGzdOkhQYGCjDMDR58mQNGjRI5cuXt2eJpq1Zs0bp6enq3bu3vUsxbeDAgfrxxx81evRoVa9eXXv37tXs2bOtK6h5gcVi0cyZM/Xaa6+pZ8+ekqRy5copPDxcr732Wq4eFiNAIMdt2rRJo0aNUqVKlTRv3jwVL17c3iWZVr58eZUvX14NGjSQu7u7XnvtNf3444+qV6+evUu7o+XLl+vQoUPasGGD9Wwe4/9frT41NVUuLi4OH4Ju1bBhw0xtjz/+uKSbqxOOHiAyjrcHBQXZtDdp0kSTJk3SoUOH8lyA2Lx5s5o2bZrnVrf27Nmj2NhYRUREqGPHjpJu/nx5enpq7Nix6tq1q/z8/OxcZfb4+vpq9erV+vPPP5WUlKRKlSpp9+7dkqSiRYvmWh3sgUCOWrhwoYYPH666detq+fLlKl26tL1LyraLFy9q3bp1Onv2rE179erVJUl//vmnPcoyZfPmzbpw4YKaNGmiGjVqqEaNGlq3bp2OHTumGjVq5Knz9RMSErRmzZpMx3Qz9g7khWCacerpjRs3bNozVibyUpiTpLNnz+rAgQN56pTHDKdOnZKkTL8EBAQESJJ+//33XK/pXly7dk2ffvqpTp48qdKlS6ty5cpydnbWL7/8IicnJz366KO5VgsBAjlmzZo1mjRpktq0aaN58+bJw8PD3iWZkp6erlGjRmU6333nzp2SlCd+O3nzzTe1du1amz/NmzdXmTJlrP+dVzg5OWns2LFatmyZTfumTZvk4uKSacOuI6pSpYp8fHy0adMmm/aYmBgVKFAgz10Qa+/evZKUJ977v/P19ZUk62/qGX766SdJko+PT26XdE9cXV311ltv6aOPPrK2Xbt2TatWrVKDBg1ydQWCQxjIEQkJCXr77bfl4+Oj0NBQHThwwObxChUqOPySp5eXl3r06KG5c+eqUKFCqlWrlnbv3q0PPvhAXbp0sZ6S6siyqrFYsWKyWCyqVauWHSq6d15eXgoNDdXSpUvl7u6ugIAA7d69W++//75CQ0NVsWJFe5d4V05OThoxYoSGDx+uESNGqGPHjtq/f7/mzJmjnj17Ovz/J/7ut99+U+HChfPMl+2tatSooeDgYL399tu6fPmyHn30Ue3fv1+zZs1SUFBQnjkd1cXFRc8++6wWLlyo0qVLq1y5cpo3b55OnTqlyZMn52otBAjkiB07dujq1as6efKkQkNDMz0eGRmZJ06VGj16tB566CGtXbtWM2bMUJkyZTRkyBA999xz9i7tgfTaa6/J29tbH330kebOnStvb28NGTIkT+2af/LJJ2WxWDRr1iwNGDBAJUqU0KBBgzRgwAB7l2bauXPn8tyZF7eaNm2aZs6cqUWLFikhIUE+Pj7q169fpmu/OLqhQ4fK2dlZs2fPVlJSkmrVqqVFixapdu3auVqHk5GxwwoAACCb2AMBAABMI0AAAADTCBAAAMA0AgQAADCNAAEAAEwjQAAAANO4DgTwAEpPT9fnn3+ujz76SL///rsSEhJUrFgx1a9fX3369MkTV0hMSkrS6tWrtXHjRsXHxystLU0PP/ywunTpoi5dusjZ+f9+P2rRooV8fHy0dOlSO1YM5C8ECOABc/nyZQ0bNkw7duxQw4YN1atXLxUrVkynTp3SunXr9Oyzz+qNN96w3unPER05ckQDBw7UyZMn1a5dO3Xq1Ek3btzQ1q1bNXbsWP3www+aMmVKnrvXBJCXECCAB8y4ceMUGxurSZMmqUOHDjaPDRgwQC+++KImTZqkxo0bW+8f4EiuX7+ul156SRcvXtTatWtVrVo162N9+/bVm2++qRUrVqh27dp58pbTQF7BHgjgAbJ7925FR0erffv2mcKDJBUsWFDjx49XSkqKPv74YztUeHcrVqxQXFycRo8ebRMeMrz22msqWrSoPvzwQztUBzw4WIEAHiAbNmyQpDte+79ixYpatGhRppsL/fjjj4qKirLevdDf31/h4eE2199v0aKFmjRpovr162vu3Lk6duyYHnroIYWFhWW6R0p2xstKdHS0ihQporZt22b5eKFChbR69WqVLVv2tmMYhqEPP/xQH330kf744w+lpqbKx8dHHTt21AsvvGA99HHp0iVFRETo22+/1blz51SmTBm1adNGL7/8sgoWLCjp5q26p0yZoi+//FJnz55ViRIl1KJFC4WHh+fqnRGB3EaAAB4g33//vUqVKqUqVarcsV9gYKDN33fu3KkBAwaoWrVqGjp0qG7cuKGPP/5YoaGhWrhwoQICAqx9d+zYoc8//1w9e/ZUyZIltWrVKr311lsqV66cmjVrZnq8WxmGoYMHD6pevXpydXW9bf2VKlW64/ymT5+u999/Xx06dFDXrl2VnJysdevW6b333pObm5s17ISHh+vAgQPq3bu3SpcurR9//FFz587VxYsXNWHCBEnSW2+9pY0bN6p3794qX768Dh8+rOXLlys+Pl4LFiy4Yx1AXkaAAB4gZ86cyfKW31evXtXVq1dt2lxcXFS0aFGlp6dr3LhxqlWrlpYtWyYXFxdJUs+ePfXMM89o4sSJWrdunfV5p0+f1rp166yHF0JCQtS0aVOtX79ezZo1Mz3erS5cuKDU1FSVKlXqnt+DlJQULVu2TG3bttWkSZOs7V26dFFgYKB27Nih0NBQJSQk6JtvvtHIkSOtd2Pt0qWLDMPQ8ePHrc/bsGGDOnXqpOHDh1vbihQpoh07dig5OVlubm73XCvgyAgQwAPEMAxldQPeqKioTL8t+/j46Msvv9SBAwd0/Phxde/eXZcuXbLp07x5cy1atEhnz56Vt7e3JMnX19dmb0KpUqVUsmRJnTt3TpJMj3erjFMz09LS7mH2N7m6uuqbb75RSkqKTfuFCxfk7u6uK1euSJI8PDxUpEgRrVixQuXKlVPTpk1VpEgRRURE2DyvTJky2rRpk2rWrKng4GB5enoqPDxc4eHh91wjkBcQIIAHSOnSpa1f5Lfq1q2bmjZtav37pEmTlJSUJEk6duyYJCkyMlKRkZFZjnvq1CnrF76Xl1emxy0Wi9LT0+9pvFsVLVpUrq6uOn/+/G3nmB2urq766quvtG3bNsXFxSk+Pt4aZjIClsVi0VtvvaU33nhDQ4YMkcViUcOGDdWyZUs988wz1j0Q48ePV3h4uEaPHq033nhDdevWVUhIiDp16iQPD49/VCfgyAgQwAPE399fn3zyieLj41WxYkVre6VKlWz2DRQtWtQaIDK++IcOHaq6detmOe6th0VuvYBTVsyOdysnJyf5+/tr//79Sk1NVYECWf8TNm3aNB0/flyjR4/OdLjDMAy99NJLiomJUf369eXv769u3bqpQYMGCgsLs+nbrl07NW3aVFu3btX27dv1zTffKDY2VitWrNCaNWtksVgUGBiomJgY65+dO3cqIiJCixYt0scff5xloALyAwIE8AB5+umn9cknn2jRokUaN25ctp7j4+Mj6eZx/UaNGtk89vPPP+vSpUsqVKhQtmv4p+OFhITo+++/t56O+nfXrl3T2rVrlZaWpmLFimV6fNeuXYqJidFLL72koUOHWttTU1N18eJFlS9fXpKUnJysgwcP6pFHHlHnzp3VuXNn6xkXS5YsUWxsrJo0aaKDBw+qTJkyatu2rdq2bav09HQtXLhQkZGRio6OVq9evbL93gB5CdeBAB4gjRo10pNPPqmVK1dqxYoVWfZZv3699u/fb/17zZo1VapUKS1dulTJycnW9qSkJOvSfcZGyOz4p+N169ZNPj4+ioyM1G+//WbzWFpamsaPH69z587phRdeyPJMjYsXL0qSHn74YZv21atX6+rVq0pNTZUkHT58WKGhoVq7dq21j8ViUfXq1SXd3GR64cIFdevWTR988IG1j7Ozs2rVqmX9byC/YgUCeMBMmDBBaWlpevPNN7V27VoFBwerVKlSOnv2rLZs2aJDhw6pZMmSGjVqlKSb+wXGjBmjYcOGqWPHjurcubMKFiyoNWvW6NSpU3r33XdveyghK/90vIIFC2rmzJnq16+fOnfurHbt2qlWrVq6ePGiPv/8cx08eFCtW7dW3759s3y+v7+/3N3dFRERoZMnT6po0aL67rvvtGnTJhUsWNAaaurUqaOAgABNmzZNp0+fVtWqVXX69GktW7ZMlStXVmBgoCwWi9q1a6cVK1bo6tWr8vf318WLF7Vs2TKVLFlSbdq0MfHJAHmLk5HVlmwA+V5MTIw++ugj/fLLL/rrr7/k7u6uRx99VMHBwerYsaMKFy5s0/9///uf5syZo3379snZ2VmPPPKIBgwYoObNm1v73O6mVVm1Z2e8Ozl79qwWLVqkr7/+WqdOnZJhGKpataq6du2qjh072twH4++vv3v3br377rv69ddfZbFY5Ovrq969e+vnn3/WkiVL9PXXX6tkyZK6ePGiZs6cqZiYGP35558qWrSoHn/8cQ0dOtS6t+LatWuaO3euoqOjdfr0aRUuXFiBgYEaNmyYzT4TIL8hQAAAANM4QAcAAEwjQAAAANMIEAAAwDQCBAAAMI0AAQAATCNAAAAA0wgQAADANAIEAAAwjQABAABM+38iHkWWpAEn4AAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 576x576 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "gene_class = result.groupby(['Class']).size()\n",
    "gene_class.plot.bar(figsize=(8,8),rot = 0)\n",
    "plt.xlabel(\"Gene Class\")\n",
    "plt.ylabel(\"Count\")\n",
    "plt.title(\"Comparing number of gene classes\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "9b21afff",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Gene\n",
      "KLF4        1\n",
      "FGF19       1\n",
      "FANCC       1\n",
      "FAM58A      1\n",
      "PAK1        1\n",
      "         ... \n",
      "BRCA2     125\n",
      "PTEN      126\n",
      "EGFR      141\n",
      "TP53      163\n",
      "BRCA1     264\n",
      "Length: 264, dtype: int64\n"
     ]
    }
   ],
   "source": [
    "gene=result.groupby(['Gene']).size().sort_values()\n",
    "print(gene)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "6f2246c4",
   "metadata": {},
   "outputs": [],
   "source": [
    "result.to_csv(datadir + '/processed_text.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "15762ce9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
